{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Week 1 -- Testing protein encodings for similarity detection with sourmash"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This notebook requires the following libraries: `pandas`, `requests`. Most of these libraries come with python, but you will get an error message if they are not installed: `ModuleNotFoundError: No module named 'pandas'`. To install something in your `rotation` environment, run:\n",
    "\n",
    "`$ conda activate rotation`  \n",
    "`(rotation) $ conda install pandas`"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Download data\n",
    "\n",
    "First download some data that we can use as a test set. We will download genomes, \"transcriptomes\" (computationally predicted RNA sequences from DNA), and amino acid sequences. We will use sequences that are related to species *Treponema denticola*, *Bacteroides thetaiotaomicron*, and *Porphyromonas gingivalis*. In the csv which we use to cue downloads, we have recorded the taxonomic relatedness of each bug that we download relative to our species of interest. We will use these levels to infer whether sourmash recapitulates known taxonomic relationships between sequences."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "!git clone https://github.com/bluegenes/2018-test_datasets.git\n",
    "\n",
    "!python 2018-test_datasets/download_genbank_datasets.py --genbank --protein --rna --subfolder 2018-test_datasets/denticola.csv\n",
    "!python 2018-test_datasets/download_genbank_datasets.py --genbank --protein --rna --subfolder 2018-test_datasets/bacteroides.csv\n",
    "!python 2018-test_datasets/download_genbank_datasets.py --genbank --protein --rna --subfolder 2018-test_datasets/gingivalis.csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "fatal: destination path '2018-test_datasets' already exists and is not an empty directory.\r\n"
     ]
    }
   ],
   "source": [
    "!git clone https://github.com/bluegenes/2018-test_datasets.git"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/343/205/GCA_000343205.1_CA_glsol121/GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/343/205/GCA_000343205.1_CA_glsol121/GCA_000343205.1_CA_glsol121_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/343/205/GCA_000343205.1_CA_glsol121/GCA_000343205.1_CA_glsol121_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/568/735/GCA_000568735.2_ASM56873v2/GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/568/735/GCA_000568735.2_ASM56873v2/GCA_000568735.2_ASM56873v2_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/568/735/GCA_000568735.2_ASM56873v2/GCA_000568735.2_ASM56873v2_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/818/865/GCA_000818865.1_ASM81886v1/GCA_000818865.1_ASM81886v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/818/865/GCA_000818865.1_ASM81886v1/GCA_000818865.1_ASM81886v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/818/865/GCA_000818865.1_ASM81886v1/GCA_000818865.1_ASM81886v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/465/015/GCA_000465015.1_Treponema_pedis_T_M1/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/465/015/GCA_000465015.1_Treponema_pedis_T_M1/GCA_000465015.1_Treponema_pedis_T_M1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/465/015/GCA_000465015.1_Treponema_pedis_T_M1/GCA_000465015.1_Treponema_pedis_T_M1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/712/955/GCA_001712955.1_ASM171295v1/GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/712/955/GCA_001712955.1_ASM171295v1/GCA_001712955.1_ASM171295v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/712/955/GCA_001712955.1_ASM171295v1/GCA_001712955.1_ASM171295v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/896/735/GCA_001896735.1_ASM189673v1/GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/896/735/GCA_001896735.1_ASM189673v1/GCA_001896735.1_ASM189673v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/896/735/GCA_001896735.1_ASM189673v1/GCA_001896735.1_ASM189673v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/125/GCA_001829125.1_ASM182912v1/GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/125/GCA_001829125.1_ASM182912v1/GCA_001829125.1_ASM182912v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/125/GCA_001829125.1_ASM182912v1/GCA_001829125.1_ASM182912v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/156/105/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/156/105/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/156/105/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/775/995/GCA_000775995.1_ASM77599v1/GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/775/995/GCA_000775995.1_ASM77599v1/GCA_000775995.1_ASM77599v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/775/995/GCA_000775995.1_ASM77599v1/GCA_000775995.1_ASM77599v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/405/175/GCA_001405175.1_13470_2_54/GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/405/175/GCA_001405175.1_13470_2_54/GCA_001405175.1_13470_2_54_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/405/175/GCA_001405175.1_13470_2_54/GCA_001405175.1_13470_2_54_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/567/655/GCA_001567655.1_ASM156765v1/GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/567/655/GCA_001567655.1_ASM156765v1/GCA_001567655.1_ASM156765v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/567/655/GCA_001567655.1_ASM156765v1/GCA_001567655.1_ASM156765v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/301/975/GCA_000301975.1_ASM30197v1/GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/301/975/GCA_000301975.1_ASM30197v1/GCA_000301975.1_ASM30197v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/301/975/GCA_000301975.1_ASM30197v1/GCA_000301975.1_ASM30197v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/338/595/GCA_000338595.1_Trep_dent_ATCC_33521_V1/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/338/595/GCA_000338595.1_Trep_dent_ATCC_33521_V1/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/338/595/GCA_000338595.1_Trep_dent_ATCC_33521_V1/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/281/815/GCA_001281815.1_ASM128181v1/GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/281/815/GCA_001281815.1_ASM128181v1/GCA_001281815.1_ASM128181v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/281/815/GCA_001281815.1_ASM128181v1/GCA_001281815.1_ASM128181v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/505/GCA_001829505.1_ASM182950v1/GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/505/GCA_001829505.1_ASM182950v1/GCA_001829505.1_ASM182950v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/505/GCA_001829505.1_ASM182950v1/GCA_001829505.1_ASM182950v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/217/655/GCA_000217655.1_ASM21765v1/GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/217/655/GCA_000217655.1_ASM21765v1/GCA_000217655.1_ASM21765v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/217/655/GCA_000217655.1_ASM21765v1/GCA_000217655.1_ASM21765v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/725/GCA_000340725.1_Trep_dent_AL-2_V1/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/725/GCA_000340725.1_Trep_dent_AL-2_V1/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/725/GCA_000340725.1_Trep_dent_AL-2_V1/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/830/635/GCA_001830635.1_ASM183063v1/GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/830/635/GCA_001830635.1_ASM183063v1/GCA_001830635.1_ASM183063v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/830/635/GCA_001830635.1_ASM183063v1/GCA_001830635.1_ASM183063v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/382/565/GCA_000382565.1_ASM38256v1/GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/382/565/GCA_000382565.1_ASM38256v1/GCA_000382565.1_ASM38256v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/382/565/GCA_000382565.1_ASM38256v1/GCA_000382565.1_ASM38256v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/164/975/GCA_900164975.1_16852_2_85/GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/164/975/GCA_900164975.1_16852_2_85/GCA_900164975.1_16852_2_85_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/164/975/GCA_900164975.1_16852_2_85/GCA_900164975.1_16852_2_85_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/758/165/GCA_000758165.1_Spiroch1.0/GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/758/165/GCA_000758165.1_Spiroch1.0/GCA_000758165.1_Spiroch1.0_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/758/165/GCA_000758165.1_Spiroch1.0/GCA_000758165.1_Spiroch1.0_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/501/115/GCA_000501115.1_BorGarIPT101/GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/501/115/GCA_000501115.1_BorGarIPT101/GCA_000501115.1_BorGarIPT101_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/501/115/GCA_000501115.1_BorGarIPT101/GCA_000501115.1_BorGarIPT101_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/002/069/965/GCA_002069965.1_ASM206996v1/GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/002/069/965/GCA_002069965.1_ASM206996v1/GCA_002069965.1_ASM206996v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/002/069/965/GCA_002069965.1_ASM206996v1/GCA_002069965.1_ASM206996v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/002/069/655/GCA_002069655.1_ASM206965v1/GCA_002069655.1_ASM206965v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/002/069/655/GCA_002069655.1_ASM206965v1/GCA_002069655.1_ASM206965v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/002/069/655/GCA_002069655.1_ASM206965v1/GCA_002069655.1_ASM206965v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/413/095/GCA_000413095.1_Trep_dent_SP32_F0464_V1/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/413/095/GCA_000413095.1_Trep_dent_SP32_F0464_V1/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/413/095/GCA_000413095.1_Trep_dent_SP32_F0464_V1/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/413/115/GCA_000413115.1_Trep_dent_SP23_F0463_V1/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/413/115/GCA_000413115.1_Trep_dent_SP23_F0463_V1/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/413/115/GCA_000413115.1_Trep_dent_SP23_F0463_V1/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/941/035/GCA_000941035.1_ASM94103v1/GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/941/035/GCA_000941035.1_ASM94103v1/GCA_000941035.1_ASM94103v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/941/035/GCA_000941035.1_ASM94103v1/GCA_000941035.1_ASM94103v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/338/635/GCA_000338635.1_Trep_dent_ASLM_V1/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/338/635/GCA_000338635.1_Trep_dent_ASLM_V1/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/338/635/GCA_000338635.1_Trep_dent_ASLM_V1/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/713/255/GCA_001713255.1_ASM171325v1/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/713/255/GCA_001713255.1_ASM171325v1/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/713/255/GCA_001713255.1_ASM171325v1/GCA_001713255.1_ASM171325v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/604/335/GCA_001604335.1_ASM160433v1/GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/604/335/GCA_001604335.1_ASM160433v1/GCA_001604335.1_ASM160433v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/604/335/GCA_001604335.1_ASM160433v1/GCA_001604335.1_ASM160433v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/018/035/GCA_900018035.1_7521_5_26/GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/018/035/GCA_900018035.1_7521_5_26/GCA_900018035.1_7521_5_26_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/018/035/GCA_900018035.1_7521_5_26/GCA_900018035.1_7521_5_26_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/217/015/GCA_000217015.3_CLC_glsol119/GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/217/015/GCA_000217015.3_CLC_glsol119/GCA_000217015.3_CLC_glsol119_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/217/015/GCA_000217015.3_CLC_glsol119/GCA_000217015.3_CLC_glsol119_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/830/585/GCA_001830585.1_ASM183058v1/GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/830/585/GCA_001830585.1_ASM183058v1/GCA_001830585.1_ASM183058v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/830/585/GCA_001830585.1_ASM183058v1/GCA_001830585.1_ASM183058v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/940/825/GCA_001940825.1_ASM194082v1/GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/940/825/GCA_001940825.1_ASM194082v1/GCA_001940825.1_ASM194082v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/940/825/GCA_001940825.1_ASM194082v1/GCA_001940825.1_ASM194082v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/502/195/GCA_000502195.1_BorAfzIPT138/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/502/195/GCA_000502195.1_BorAfzIPT138/GCA_000502195.1_BorAfzIPT138_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/502/195/GCA_000502195.1_BorAfzIPT138/GCA_000502195.1_BorAfzIPT138_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/746/215/GCA_001746215.1_ASM174621v1/GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/746/215/GCA_001746215.1_ASM174621v1/GCA_001746215.1_ASM174621v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/746/215/GCA_001746215.1_ASM174621v1/GCA_001746215.1_ASM174621v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/236/685/GCA_000236685.1_ASM23668v1/GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/236/685/GCA_000236685.1_ASM23668v1/GCA_000236685.1_ASM23668v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/236/685/GCA_000236685.1_ASM23668v1/GCA_000236685.1_ASM23668v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/475/GCA_001829475.1_ASM182947v1/GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/475/GCA_001829475.1_ASM182947v1/GCA_001829475.1_ASM182947v1_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/475/GCA_001829475.1_ASM182947v1/GCA_001829475.1_ASM182947v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/443/305/GCA_001443305.1_ASM144330v1/GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/443/305/GCA_001443305.1_ASM144330v1/GCA_001443305.1_ASM144330v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/443/305/GCA_001443305.1_ASM144330v1/GCA_001443305.1_ASM144330v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/147/745/GCA_900147745.1_138_1/GCA_900147745.1_138_1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/147/745/GCA_900147745.1_138_1/GCA_900147745.1_138_1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/147/745/GCA_900147745.1_138_1/GCA_900147745.1_138_1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/745/GCA_000340745.1_Trep_dent_US-Trep_V1/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/745/GCA_000340745.1_Trep_dent_US-Trep_V1/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/745/GCA_000340745.1_Trep_dent_US-Trep_V1/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/002/009/835/GCA_002009835.1_ASM200983v1/GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/002/009/835/GCA_002009835.1_ASM200983v1/GCA_002009835.1_ASM200983v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/002/009/835/GCA_002009835.1_ASM200983v1/GCA_002009835.1_ASM200983v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/415/GCA_001829415.1_ASM182941v1/GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/415/GCA_001829415.1_ASM182941v1/GCA_001829415.1_ASM182941v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/415/GCA_001829415.1_ASM182941v1/GCA_001829415.1_ASM182941v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/260/795/GCA_000260795.1_ASM26079v1/GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/260/795/GCA_000260795.1_ASM26079v1/GCA_000260795.1_ASM26079v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/260/795/GCA_000260795.1_ASM26079v1/GCA_000260795.1_ASM26079v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/426/705/GCA_000426705.1_ASM42670v1/GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/426/705/GCA_000426705.1_ASM42670v1/GCA_000426705.1_ASM42670v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/426/705/GCA_000426705.1_ASM42670v1/GCA_000426705.1_ASM42670v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/239/475/GCA_000239475.1_ASM23947v1/GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/239/475/GCA_000239475.1_ASM23947v1/GCA_000239475.1_ASM23947v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/239/475/GCA_000239475.1_ASM23947v1/GCA_000239475.1_ASM23947v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/167/145/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/167/145/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/900/167/145/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/768/055/GCA_000768055.1_ASM76805v1/GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/768/055/GCA_000768055.1_ASM76805v1/GCA_000768055.1_ASM76805v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/768/055/GCA_000768055.1_ASM76805v1/GCA_000768055.1_ASM76805v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/605/GCA_000340605.1_Trep_dent_H1-T_V1/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/605/GCA_000340605.1_Trep_dent_H1-T_V1/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/605/GCA_000340605.1_Trep_dent_H1-T_V1/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/295/GCA_001829295.1_ASM182929v1/GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/295/GCA_001829295.1_ASM182929v1/GCA_001829295.1_ASM182929v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/295/GCA_001829295.1_ASM182929v1/GCA_001829295.1_ASM182929v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/537/795/GCA_000537795.1_Stap_aure_SMMC6007_V1/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/537/795/GCA_000537795.1_Stap_aure_SMMC6007_V1/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/537/795/GCA_000537795.1_Stap_aure_SMMC6007_V1/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/802/305/GCA_000802305.1_ASM80230v1/GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/802/305/GCA_000802305.1_ASM80230v1/GCA_000802305.1_ASM80230v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/802/305/GCA_000802305.1_ASM80230v1/GCA_000802305.1_ASM80230v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/222/305/GCA_000222305.1_ASM22230v1/GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/222/305/GCA_000222305.1_ASM22230v1/GCA_000222305.1_ASM22230v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/222/305/GCA_000222305.1_ASM22230v1/GCA_000222305.1_ASM22230v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/815/GCA_000246815.1_ASM24681v1/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/507/245/GCA_000507245.1_ASM50724v1/GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/507/245/GCA_000507245.1_ASM50724v1/GCA_000507245.1_ASM50724v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/507/245/GCA_000507245.1_ASM50724v1/GCA_000507245.1_ASM50724v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/713/255/GCA_001713255.1_ASM171325v1/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/713/255/GCA_001713255.1_ASM171325v1/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/713/255/GCA_001713255.1_ASM171325v1/GCA_001713255.1_ASM171325v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/645/GCA_000340645.1_Trep_dent_H-22_V1/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/645/GCA_000340645.1_Trep_dent_H-22_V1/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/340/645/GCA_000340645.1_Trep_dent_H-22_V1/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/338/615/GCA_000338615.1_Trep_dent_ATCC_33520_V1/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/338/615/GCA_000338615.1_Trep_dent_ATCC_33520_V1/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/338/615/GCA_000338615.1_Trep_dent_ATCC_33520_V1/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/165/GCA_001829165.1_ASM182916v1/GCA_001829165.1_ASM182916v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/165/GCA_001829165.1_ASM182916v1/GCA_001829165.1_ASM182916v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/829/165/GCA_001829165.1_ASM182916v1/GCA_001829165.1_ASM182916v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/501/555/GCA_000501555.1_BorAfzIPT109/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/501/555/GCA_000501555.1_BorAfzIPT109/GCA_000501555.1_BorAfzIPT109_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/501/555/GCA_000501555.1_BorAfzIPT109/GCA_000501555.1_BorAfzIPT109_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/216/715/GCA_000216715.2_CLC_glsol140/GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/216/715/GCA_000216715.2_CLC_glsol140/GCA_000216715.2_CLC_glsol140_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/216/715/GCA_000216715.2_CLC_glsol140/GCA_000216715.2_CLC_glsol140_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/147/075/GCA_000147075.1_ASM14707v1/GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/147/075/GCA_000147075.1_ASM14707v1/GCA_000147075.1_ASM14707v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/147/075/GCA_000147075.1_ASM14707v1/GCA_000147075.1_ASM14707v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/242/595/GCA_000242595.3_ASM24259v3/GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/242/595/GCA_000242595.3_ASM24259v3/GCA_000242595.3_ASM24259v3_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/242/595/GCA_000242595.3_ASM24259v3/GCA_000242595.3_ASM24259v3_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/894/145/GCA_001894145.1_ASM189414v1/GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/894/145/GCA_001894145.1_ASM189414v1/GCA_001894145.1_ASM189414v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/001/894/145/GCA_001894145.1_ASM189414v1/GCA_001894145.1_ASM189414v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/500/065/GCA_000500065.1_AB2/GCA_000500065.1_AB2_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/500/065/GCA_000500065.1_AB2/GCA_000500065.1_AB2_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/500/065/GCA_000500065.1_AB2/GCA_000500065.1_AB2_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/413/075/GCA_000413075.1_Trep_dent_SP44_F0466_V1/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/413/075/GCA_000413075.1_Trep_dent_SP44_F0466_V1/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/413/075/GCA_000413075.1_Trep_dent_SP44_F0466_V1/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/415/GCA_000246415.2_CLC_glsol068/GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/415/GCA_000246415.2_CLC_glsol068/GCA_000246415.2_CLC_glsol068_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/246/415/GCA_000246415.2_CLC_glsol068/GCA_000246415.2_CLC_glsol068_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/342/705/GCA_000342705.1_CA_glsol153/GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/342/705/GCA_000342705.1_CA_glsol153/GCA_000342705.1_CA_glsol153_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/342/705/GCA_000342705.1_CA_glsol153/GCA_000342705.1_CA_glsol153_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/468/055/GCA_000468055.1_ASM46805v1/GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/468/055/GCA_000468055.1_ASM46805v1/GCA_000468055.1_ASM46805v1_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/468/055/GCA_000468055.1_ASM46805v1/GCA_000468055.1_ASM46805v1_rna_from_genomic.fna.gz\n",
      "genome: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/501/035/GCA_000501035.1_BorGarIPT95/GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n",
      "protein: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/501/035/GCA_000501035.1_BorGarIPT95/GCA_000501035.1_BorGarIPT95_protein.faa.gz\n",
      "rna: https://ftp.ncbi.nih.gov/genomes/all/GCA/000/501/035/GCA_000501035.1_BorGarIPT95/GCA_000501035.1_BorGarIPT95_rna_from_genomic.fna.gz\n"
     ]
    }
   ],
   "source": [
    "!python 2018-test_datasets/download_genbank_datasets.py --genbank --protein --rna --subfolder 2018-test_datasets/denticola.csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!python 2018-test_datasets/download_genbank_datasets.py --genbank --protein --rna --subfolder 2018-test_datasets/bacteroides.csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "!python 2018-test_datasets/download_genbank_datasets.py --genbank --protein --rna --subfolder 2018-test_datasets/gingivalis.csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "69\n",
      "GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n",
      "GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "GCA_000500065.1_AB2_genomic.fna.gz\n",
      "GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n",
      "GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "GCA_000818865.1_ASM81886v1_genomic.fna.gz\n",
      "GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "GCA_001829165.1_ASM182916v1_genomic.fna.gz\n",
      "GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "GCA_002069655.1_ASM206965v1_genomic.fna.gz\n",
      "GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "GCA_900147745.1_138_1_genomic.fna.gz\n",
      "GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n"
     ]
    }
   ],
   "source": [
    "# look and see how many genomes were downloaded for denticola\n",
    "!ls 2018-test_datasets/denticola/genomic/ | wc -l\n",
    "!ls 2018-test_datasets/denticola/genomic/\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Generating signatures using sourmash\n",
    "\n",
    "We will generate signatures for our data. One signature can hold multiple k sizes, but only one scaled value and only one molecule type/encoding (i.e. DNA or protein)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000011065.1_ASM1106v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 33 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000157015.1_ASM15701v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 5 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000163055.2_ASM16305v2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 76 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 19 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 50 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000373705.1_ASM37370v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 6 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 114 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000431275.1_MGS755_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 175 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000432495.1_MGS558_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 103 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000466425.1_ASM46642v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 59 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000495955.1_NUHP2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 395 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000501415.1_BorBurgIPT48_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 13 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 48 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 449 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000598325.1_ASM59832v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 48 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000611675.1_ASM61167v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 92 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000614865.1_ASM61486v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 105 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000614895.1_ASM61489v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000626635.1_ASM62663v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 298 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 179 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 77 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000785025.1_ASM78502v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 55 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000814325.1_ASM81432v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 68 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000821925.1_34AVR_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 188 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000938735.2_ASM93873v2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 212 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000974365.1_ASM97436v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 175 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001049535.1_3731_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001049555.1_7330_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001049555.1_7330_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 407 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001049555.1_7330_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001049555.1_7330_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 351 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001055755.1_ASM105575v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 307 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001304045.1_ASM130404v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 75 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001310955.1_ASM131095v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 1730 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001373135.1_2e6A_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 69 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001398875.1_Mother10-2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 52 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001405895.1_14207_7_22_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 23 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001405935.1_13414_6_56_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001413955.1_ASM141395v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001443605.1_ASM144360v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001548195.1_ASM154819v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 108 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001578565.1_ASM157856v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 136 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 659 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001603535.1_ASM160353v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 260 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001603695.1_ASM160369v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 423 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001614375.1_ASM161437v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 81 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001659685.2_ASM165968v2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 14 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001693695.1_ASM169369v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001761325.1_ASM176132v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 102 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001816245.1_ASM181624v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 580 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001915365.1_ASM191536v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 84 sequences in 2018-test_datasets/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_002036985.1_ASM203698v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 78 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900032635.1_14555_6_27_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 6 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900104585.1_PRJEB16348_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 76 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 38 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 74 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 89 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 58 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 37 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 25 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900128475.1_PRJEB18047_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 13 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 11 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Ksaved signature(s) to sigs/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.sig. Note: signature license is CC0.\r"
     ]
    }
   ],
   "source": [
    "# calculate signatures for RNA and DNA\n",
    "!mkdir -p sigs/bacteroides/genomic\n",
    "!for infile in 2018-test_datasets/bacteroides/genomic/*fna.gz; do out_name=$(basename $infile .fna.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/bacteroides/genomic/${out_name}.sig ${infile}; done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mv: cannot stat 'sigs/bacteroides/genomic/*protein.sig': No such file or directory\r\n"
     ]
    }
   ],
   "source": [
    "##worker Bee\n",
    "\n",
    "!mkdir -p sigs/bacteroides/genomic2\n",
    "!for infile in 2018-test_datasets/bacteroides/genomic2/*fna.gz; \\\n",
    "do out_name=$(basename $infile .fna.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/bacteroides/genomic/${out_name}.sig ${infile}; done\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "usage: sourmash [--protein] [--no-protein] [--dayhoff] [--no-dayhoff] [--dna]\r\n",
      "                [--no-dna] [-q] [--input-is-protein] [-k KSIZES]\r\n",
      "                [-n NUM_HASHES] [--check-sequence] [-f] [-o OUTPUT]\r\n",
      "                [--singleton] [--merge MERGED] [--name-from-first]\r\n",
      "                [--input-is-10x] [--count-valid-reads COUNT_VALID_READS]\r\n",
      "                [--write-barcode-meta-csv WRITE_BARCODE_META_CSV]\r\n",
      "                [-p PROCESSES] [--save-fastas SAVE_FASTAS]\r\n",
      "                [--line-count LINE_COUNT] [--track-abundance]\r\n",
      "                [--scaled SCALED] [--seed SEED] [--randomize]\r\n",
      "                [--license LICENSE]\r\n",
      "                [--rename-10x-barcodes RENAME_10X_BARCODES]\r\n",
      "                [--barcodes-file BARCODES_FILE]\r\n",
      "                filenames [filenames ...]\r\n",
      "sourmash: error: unrecognized arguments: -h\r\n"
     ]
    }
   ],
   "source": [
    "# Using the code above, and the sourmash compute help message, calculate signatures for proteins\n",
    "!sourmash compute -h\n",
    "\n",
    "!mkdir -p sigs/bacteroides/proteomic\n",
    "!for infile in 2018-test_datasets/bacteroides/protein/*faa.gz; do out_name=$(basename $infile .faa.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/bacteroides/proteomic/${out_name}.sig ${infile}; done\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Kloaded 61 signatures total.                                                    _assembly_genomic.sig\n",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\u001b[K\n",
      "min similarity in matrix: 0.000\n",
      "\u001b[Ksaving labels to: sourmash_compare/bacteroides_k31_dna_comp.labels.txt\n",
      "\u001b[Ksaving distance matrix to: sourmash_compare/bacteroides_k31_dna_comp\n"
     ]
    }
   ],
   "source": [
    "# Now you can use sourmash compare to compare signatures that were calculated with the same k size, the same \n",
    "# molecule type, and the same encoding (e.g. dayoff or h-p).\n",
    "# Take a look at the metapallette paper for some applications of k-sizes to detection of relatedness across\n",
    "# evolutionary distances, or take our word that k = 21 ~ genus level similarlity, k = 31 ~ species level \n",
    "# similarity, and k = 51 ~ strain level similarity. https://msystems.asm.org/content/1/3/e00020-16\n",
    "!mkdir -p sourmash_compare\n",
    "!sourmash compare -k 31 -o sourmash_compare/bacteroides_k31_dna_comp sigs/bacteroides/genomic/*.sig"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Sourmash has built-in plotting capabilities that you can use to visualize the comparison matrix\n",
    "# You can open these pdf files from your computer, or go back to the jupyter dashboard an click \n",
    "# on them to open view them.\n",
    "!sourmash plot --labels --pdf sourmash_compare/bacteroides_k31_dna_comp\n",
    "%mv *.pdf /mnt/c/Users/user/Desktop/TESTcsv/\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Kloaded 61 signatures total.                                                    _assembly_genomic.sig\n",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\u001b[K\n",
      "min similarity in matrix: 0.000\n"
     ]
    }
   ],
   "source": [
    "# Alternatively, you can output the sourmash compare matrix as a csv, and import it into R to visualize:\n",
    "!sourmash compare -k 31 --csv sourmash_compare/bacteroides_k31_dna_comp.csv sigs/bacteroides/genomic/*.sig\n",
    "# see this link for generic R code to visualize the matrix:\n",
    "# https://sourmash.readthedocs.io/en/latest/other-languages.html#r-code-for-working-with-compare-output\n",
    "\n",
    "%mv sourmash_compare/*.csv /mnt/c/Users/user/Desktop/TESTcsv/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Your next rotation task!\n",
    "\n",
    "Use this notebook and sourmash help messages to create the following signatures for all files in the bacteroides, denticola, and gingivalis folders:\n",
    "+ DNA k = 21, 31, 51; scaled = 2000\n",
    "+ RNA k = 21, 31, 51; scaled = 2000\n",
    "+ protein k = 7, 11, 17; scaled = 2000; no encoding\n",
    "+ protein k = 7, 11, 17; scaled = 2000; dayhoff encoding\n",
    "+ protein k = 7, 11, 17; scaled = 2000; hp encoding\n",
    "\n",
    "Then, generate `sourmash compare` matrices for each set of signatures (i.e. one for denticola DNA k = 21, scaled = 2000). In total, this will be 45 sourmash compare matrices. Make sure you carefully select a consistent naming scheme so you will be able to tell the difference between each of these. \n",
    "\n",
    "Then, generate visualizations for these compare matrices. We want to know whether these accurately capture taxonomic relationships across evolutionary distances. Feel free to be creative with visualizations! You can use built-in sourmash plots, use the R code linked above, or even explore things like tanglegrams to compare a two trees. \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "*here I got really overwhelmed, so this is what I will want to be able to run in rstudio markdowns once i figure out how to install python and packages for it... >.< *"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Note: where does 45 come from? well: #of k's to try (3) * versions (5) * #plots (3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!git clone https://github.com/bluegenes/2018-test_datasets.git\n",
    "##Download files\n",
    "#!python 2018-test_datasets/download_genbank_datasets.py --genbank --protein --rna --subfolder 2018-test_datasets/denticola.csv\n",
    "#!python 2018-test_datasets/download_genbank_datasets.py --genbank --protein --rna --subfolder 2018-test_datasets/bacteroides.csv\n",
    "#!python 2018-test_datasets/download_genbank_datasets.py --genbank --protein --rna --subfolder 2018-test_datasets/gingivalis.csv\n",
    "\n",
    "#count up\n",
    "# look and see how many genomes were downloaded for denticola\n",
    "#!ls 2018-test_datasets/denticola/genomic/ | wc -l\n",
    "#!ls 2018-test_datasets/denticola/genomic/\n",
    "\n",
    "\n",
    "\n",
    "#make a dir for the signature for each genome in the dataset (6, 3orgs*2spaces)\n",
    "!mkdir -p sigs/bacteroides/genomic\n",
    "!mkdir -p sigs/bacteroides/proteomic\n",
    "\n",
    "!mkdir -p sigs/denticola/genomic\n",
    "!mkdir -p sigs/denticola/proteomic\n",
    "\n",
    "!mkdir -p sigs/gingivalis/genomic\n",
    "!mkdir -p sigs/gingivalis/proteomic\n",
    "\n",
    "#unzip and compute signature\n",
    "!for infile in 2018-test_datasets/bacteroides/genomic/*fna.gz; do out_name=$(basename $infile .fna.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/bacteroides/genomic/${out_name}.sig ${infile}; done\n",
    "\n",
    "!for infile in 2018-test_datasets/bacteroides/protein/*faa.gz; do out_name=$(basename $infile .faa.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/bacteroides/proteomic/${out_name}.sig ${infile}; done\n",
    "\n",
    "\n",
    "# compare matrix as a csv, and import it into R to visualize:\n",
    "!sourmash compare -k 31 --csv sourmash_compare/bacteroides_k31_dna_comp.csv sigs/bacteroides/genomic/*.sig\n",
    "# see this link for generic R code to visualize the matrix:\n",
    "# https://sourmash.readthedocs.io/en/latest/other-languages.html#r-code-for-working-with-compare-output\n",
    "\n",
    "%mv sourmash_compare/*.csv /mnt/c/Users/user/Desktop/TESTcsv/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "#make a dir for the signature for each genome in the dataset (6, 3orgs*2spaces)\n",
    "!mkdir -p sigs/bacteroides/genomic\n",
    "!mkdir -p sigs/bacteroides/proteomic\n",
    "\n",
    "!mkdir -p sigs/denticola/genomic\n",
    "!mkdir -p sigs/denticola/proteomic\n",
    "\n",
    "!mkdir -p sigs/gingivalis/genomic\n",
    "!mkdir -p sigs/gingivalis/proteomic"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000011065.1_ASM1106v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 33 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000157015.1_ASM15701v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 5 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000163055.2_ASM16305v2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 76 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 19 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 50 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000373705.1_ASM37370v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 6 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 114 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000431275.1_MGS755_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 175 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000432495.1_MGS558_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 103 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000466425.1_ASM46642v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 59 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000495955.1_NUHP2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 395 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000501415.1_BorBurgIPT48_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 13 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 48 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 449 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000598325.1_ASM59832v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 48 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000611675.1_ASM61167v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 92 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000614865.1_ASM61486v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 105 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000614895.1_ASM61489v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000626635.1_ASM62663v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 298 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 179 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 77 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000785025.1_ASM78502v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 55 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000814325.1_ASM81432v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 68 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000821925.1_34AVR_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 188 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000938735.2_ASM93873v2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 212 sequences in 2018-test_datasets/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_000974365.1_ASM97436v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 175 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001049535.1_3731_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001049555.1_7330_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001049555.1_7330_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 407 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001049555.1_7330_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001049555.1_7330_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 351 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001055755.1_ASM105575v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 307 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001304045.1_ASM130404v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 75 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001310955.1_ASM131095v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 1730 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001373135.1_2e6A_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 69 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001398875.1_Mother10-2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 52 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001405895.1_14207_7_22_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 23 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001405935.1_13414_6_56_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001413955.1_ASM141395v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001443605.1_ASM144360v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001548195.1_ASM154819v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 108 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001578565.1_ASM157856v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 136 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 659 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001603535.1_ASM160353v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 260 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001603695.1_ASM160369v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 423 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001614375.1_ASM161437v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 81 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001659685.2_ASM165968v2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 14 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001693695.1_ASM169369v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001761325.1_ASM176132v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 102 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001816245.1_ASM181624v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 580 sequences in 2018-test_datasets/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_001915365.1_ASM191536v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 84 sequences in 2018-test_datasets/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_002036985.1_ASM203698v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 78 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900032635.1_14555_6_27_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 6 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900104585.1_PRJEB16348_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 76 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 38 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 74 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 89 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 58 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 37 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 25 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900128475.1_PRJEB18047_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 13 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 11 sequences in 2018-test_datasets/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000011065.1_ASM1106v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000011065.1_ASM1106v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4825 sequences in 2018-test_datasets/bacteroides/protein/GCA_000011065.1_ASM1106v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000011065.1_ASM1106v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000157015.1_ASM15701v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000157015.1_ASM15701v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4927 sequences in 2018-test_datasets/bacteroides/protein/GCA_000157015.1_ASM15701v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000157015.1_ASM15701v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 4131 sequences in 2018-test_datasets/bacteroides/protein/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000163055.2_ASM16305v2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000163055.2_ASM16305v2_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1960 sequences in 2018-test_datasets/bacteroides/protein/GCA_000163055.2_ASM16305v2_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000163055.2_ASM16305v2_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000224595.1_Prevotella_sp_C561_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000224595.1_Prevotella_sp_C561_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3402 sequences in 2018-test_datasets/bacteroides/protein/GCA_000224595.1_Prevotella_sp_C561_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 5524 sequences in 2018-test_datasets/bacteroides/protein/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000373705.1_ASM37370v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000373705.1_ASM37370v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_000373705.1_ASM37370v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4799 sequences in 2018-test_datasets/bacteroides/protein/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000431275.1_MGS755_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000431275.1_MGS755_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1936 sequences in 2018-test_datasets/bacteroides/protein/GCA_000431275.1_MGS755_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000431275.1_MGS755_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000432495.1_MGS558_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000432495.1_MGS558_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3729 sequences in 2018-test_datasets/bacteroides/protein/GCA_000432495.1_MGS558_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000432495.1_MGS558_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000466425.1_ASM46642v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000466425.1_ASM46642v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2778 sequences in 2018-test_datasets/bacteroides/protein/GCA_000466425.1_ASM46642v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000466425.1_ASM46642v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000495955.1_NUHP2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000495955.1_NUHP2_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_000495955.1_NUHP2_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000501415.1_BorBurgIPT48_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000501415.1_BorBurgIPT48_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_000501415.1_BorBurgIPT48_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2680 sequences in 2018-test_datasets/bacteroides/protein/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000550025.1_Stap_aure_M1496_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000550025.1_Stap_aure_M1496_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2661 sequences in 2018-test_datasets/bacteroides/protein/GCA_000550025.1_Stap_aure_M1496_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000598325.1_ASM59832v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000598325.1_ASM59832v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4833 sequences in 2018-test_datasets/bacteroides/protein/GCA_000598325.1_ASM59832v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000598325.1_ASM59832v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000611675.1_ASM61167v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000611675.1_ASM61167v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_000611675.1_ASM61167v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3971 sequences in 2018-test_datasets/bacteroides/protein/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000707425.1_CFSAN007430_01.0_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000707425.1_CFSAN007430_01.0_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3663 sequences in 2018-test_datasets/bacteroides/protein/GCA_000707425.1_CFSAN007430_01.0_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000759315.1_04_NF40_HMP9302v01_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000759315.1_04_NF40_HMP9302v01_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2014 sequences in 2018-test_datasets/bacteroides/protein/GCA_000759315.1_04_NF40_HMP9302v01_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000785025.1_ASM78502v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000785025.1_ASM78502v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_000785025.1_ASM78502v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000814325.1_ASM81432v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000814325.1_ASM81432v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3302 sequences in 2018-test_datasets/bacteroides/protein/GCA_000814325.1_ASM81432v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000814325.1_ASM81432v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000821925.1_34AVR_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000821925.1_34AVR_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_000821925.1_34AVR_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000938735.2_ASM93873v2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000938735.2_ASM93873v2_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4793 sequences in 2018-test_datasets/bacteroides/protein/GCA_000938735.2_ASM93873v2_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000938735.2_ASM93873v2_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_000974365.1_ASM97436v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_000974365.1_ASM97436v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4411 sequences in 2018-test_datasets/bacteroides/protein/GCA_000974365.1_ASM97436v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_000974365.1_ASM97436v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001049535.1_3731_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001049535.1_3731_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001049555.1_7330_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001049555.1_7330_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_001049555.1_7330_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001055755.1_ASM105575v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001055755.1_ASM105575v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_001055755.1_ASM105575v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001304045.1_ASM130404v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001304045.1_ASM130404v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2092 sequences in 2018-test_datasets/bacteroides/protein/GCA_001304045.1_ASM130404v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001304045.1_ASM130404v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001310955.1_ASM131095v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001310955.1_ASM131095v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_001310955.1_ASM131095v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001373135.1_2e6A_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001373135.1_2e6A_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_001373135.1_2e6A_assembly_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001398875.1_Mother10-2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001398875.1_Mother10-2_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_001398875.1_Mother10-2_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001405895.1_14207_7_22_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001405895.1_14207_7_22_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 5036 sequences in 2018-test_datasets/bacteroides/protein/GCA_001405895.1_14207_7_22_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001405895.1_14207_7_22_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001405935.1_13414_6_56_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001405935.1_13414_6_56_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4272 sequences in 2018-test_datasets/bacteroides/protein/GCA_001405935.1_13414_6_56_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001405935.1_13414_6_56_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001413955.1_ASM141395v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001413955.1_ASM141395v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3385 sequences in 2018-test_datasets/bacteroides/protein/GCA_001413955.1_ASM141395v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001413955.1_ASM141395v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3865 sequences in 2018-test_datasets/bacteroides/protein/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001548195.1_ASM154819v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001548195.1_ASM154819v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2510 sequences in 2018-test_datasets/bacteroides/protein/GCA_001548195.1_ASM154819v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001548195.1_ASM154819v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001578565.1_ASM157856v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001578565.1_ASM157856v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 5105 sequences in 2018-test_datasets/bacteroides/protein/GCA_001578565.1_ASM157856v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001578565.1_ASM157856v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001595155.1_CFSAN033860_01.0_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001595155.1_CFSAN033860_01.0_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4763 sequences in 2018-test_datasets/bacteroides/protein/GCA_001595155.1_CFSAN033860_01.0_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001603535.1_ASM160353v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001603535.1_ASM160353v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_001603535.1_ASM160353v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001603695.1_ASM160369v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001603695.1_ASM160369v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_001603695.1_ASM160369v1_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001614375.1_ASM161437v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001614375.1_ASM161437v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_001614375.1_ASM161437v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001659685.2_ASM165968v2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001659685.2_ASM165968v2_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_001659685.2_ASM165968v2_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001693695.1_ASM169369v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001693695.1_ASM169369v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4070 sequences in 2018-test_datasets/bacteroides/protein/GCA_001693695.1_ASM169369v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001693695.1_ASM169369v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001761325.1_ASM176132v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001761325.1_ASM176132v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2970 sequences in 2018-test_datasets/bacteroides/protein/GCA_001761325.1_ASM176132v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001761325.1_ASM176132v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001816245.1_ASM181624v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001816245.1_ASM181624v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4956 sequences in 2018-test_datasets/bacteroides/protein/GCA_001816245.1_ASM181624v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001816245.1_ASM181624v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_001915365.1_ASM191536v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_001915365.1_ASM191536v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3281 sequences in 2018-test_datasets/bacteroides/protein/GCA_001915365.1_ASM191536v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_001915365.1_ASM191536v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_002036985.1_ASM203698v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_002036985.1_ASM203698v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_002036985.1_ASM203698v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900032635.1_14555_6_27_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900032635.1_14555_6_27_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 2030 sequences in 2018-test_datasets/bacteroides/protein/GCA_900032635.1_14555_6_27_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_900032635.1_14555_6_27_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900104585.1_PRJEB16348_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900104585.1_PRJEB16348_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_900104585.1_PRJEB16348_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4750 sequences in 2018-test_datasets/bacteroides/protein/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4847 sequences in 2018-test_datasets/bacteroides/protein/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4892 sequences in 2018-test_datasets/bacteroides/protein/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4090 sequences in 2018-test_datasets/bacteroides/protein/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3057 sequences in 2018-test_datasets/bacteroides/protein/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 5087 sequences in 2018-test_datasets/bacteroides/protein/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900128475.1_PRJEB18047_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900128475.1_PRJEB18047_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/bacteroides/protein/GCA_900128475.1_PRJEB18047_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3335 sequences in 2018-test_datasets/bacteroides/protein/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/bacteroides/protein/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/bacteroides/protein/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2333 sequences in 2018-test_datasets/bacteroides/protein/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==s/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000147075.1_ASM14707v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 316 sequences in 2018-test_datasets/denticola/genomic/GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000216715.2_CLC_glsol140_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 338 sequences in 2018-test_datasets/denticola/genomic/GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000217015.3_CLC_glsol119_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000217655.1_ASM21765v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 17 sequences in 2018-test_datasets/denticola/genomic/GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000222305.1_ASM22230v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000236685.1_ASM23668v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 3 sequences in 2018-test_datasets/denticola/genomic/GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000239475.1_ASM23947v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000242595.3_ASM24259v3_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 667 sequences in 2018-test_datasets/denticola/genomic/GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000246415.2_CLC_glsol068_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000246815.1_ASM24681v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 147 sequences in 2018-test_datasets/denticola/genomic/GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000260795.1_ASM26079v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 116 sequences in 2018-test_datasets/denticola/genomic/GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000301975.1_ASM30197v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 3 sequences in 2018-test_datasets/denticola/genomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/denticola/genomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/denticola/genomic/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000340645.1_Trep_dent_H-22_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/denticola/genomic/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 140 sequences in 2018-test_datasets/denticola/genomic/GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000342705.1_CA_glsol153_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 162 sequences in 2018-test_datasets/denticola/genomic/GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000343205.1_CA_glsol121_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 7 sequences in 2018-test_datasets/denticola/genomic/GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000382565.1_ASM38256v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 8 sequences in 2018-test_datasets/denticola/genomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 6 sequences in 2018-test_datasets/denticola/genomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 3 sequences in 2018-test_datasets/denticola/genomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 29 sequences in 2018-test_datasets/denticola/genomic/GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000426705.1_ASM42670v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 558 sequences in 2018-test_datasets/denticola/genomic/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000465015.1_Treponema_pedis_T_M1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 21 sequences in 2018-test_datasets/denticola/genomic/GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000468055.1_ASM46805v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000500065.1_AB2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000500065.1_AB2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 168 sequences in 2018-test_datasets/denticola/genomic/GCA_000500065.1_AB2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000500065.1_AB2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 194 sequences in 2018-test_datasets/denticola/genomic/GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000501035.1_BorGarIPT95_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 713 sequences in 2018-test_datasets/denticola/genomic/GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000501115.1_BorGarIPT101_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 537 sequences in 2018-test_datasets/denticola/genomic/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000501555.1_BorAfzIPT109_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 193 sequences in 2018-test_datasets/denticola/genomic/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000502195.1_BorAfzIPT138_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000507245.1_ASM50724v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 10 sequences in 2018-test_datasets/denticola/genomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 74 sequences in 2018-test_datasets/denticola/genomic/GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000568735.2_ASM56873v2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 72 sequences in 2018-test_datasets/denticola/genomic/GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000758165.1_Spiroch1.0_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 275 sequences in 2018-test_datasets/denticola/genomic/GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000768055.1_ASM76805v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000775995.1_ASM77599v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 55 sequences in 2018-test_datasets/denticola/genomic/GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000802305.1_ASM80230v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000818865.1_ASM81886v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000818865.1_ASM81886v1_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/denticola/genomic/GCA_000818865.1_ASM81886v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000818865.1_ASM81886v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 5 sequences in 2018-test_datasets/denticola/genomic/GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_000941035.1_ASM94103v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 286 sequences in 2018-test_datasets/denticola/genomic/GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001281815.1_ASM128181v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 72 sequences in 2018-test_datasets/denticola/genomic/GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001405175.1_13470_2_54_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/denticola/genomic/GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001443305.1_ASM144330v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 255 sequences in 2018-test_datasets/denticola/genomic/GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001567655.1_ASM156765v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 592 sequences in 2018-test_datasets/denticola/genomic/GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001604335.1_ASM160433v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001712955.1_ASM171295v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/denticola/genomic/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001713255.1_ASM171325v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 178 sequences in 2018-test_datasets/denticola/genomic/GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001746215.1_ASM174621v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 273 sequences in 2018-test_datasets/denticola/genomic/GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001829125.1_ASM182912v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001829165.1_ASM182916v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001829165.1_ASM182916v1_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 373 sequences in 2018-test_datasets/denticola/genomic/GCA_001829165.1_ASM182916v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001829165.1_ASM182916v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 138 sequences in 2018-test_datasets/denticola/genomic/GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001829295.1_ASM182929v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 77 sequences in 2018-test_datasets/denticola/genomic/GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001829415.1_ASM182941v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 209 sequences in 2018-test_datasets/denticola/genomic/GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001829475.1_ASM182947v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 245 sequences in 2018-test_datasets/denticola/genomic/GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001829505.1_ASM182950v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 267 sequences in 2018-test_datasets/denticola/genomic/GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001830585.1_ASM183058v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 67 sequences in 2018-test_datasets/denticola/genomic/GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001830635.1_ASM183063v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 440 sequences in 2018-test_datasets/denticola/genomic/GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001894145.1_ASM189414v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 160 sequences in 2018-test_datasets/denticola/genomic/GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001896735.1_ASM189673v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 72 sequences in 2018-test_datasets/denticola/genomic/GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_001940825.1_ASM194082v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 337 sequences in 2018-test_datasets/denticola/genomic/GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_002009835.1_ASM200983v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_002069655.1_ASM206965v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_002069655.1_ASM206965v1_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 122 sequences in 2018-test_datasets/denticola/genomic/GCA_002069655.1_ASM206965v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_002069655.1_ASM206965v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 160 sequences in 2018-test_datasets/denticola/genomic/GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_002069965.1_ASM206996v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 72 sequences in 2018-test_datasets/denticola/genomic/GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_900018035.1_7521_5_26_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_900147745.1_138_1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_900147745.1_138_1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 282 sequences in 2018-test_datasets/denticola/genomic/GCA_900147745.1_138_1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_900147745.1_138_1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 65 sequences in 2018-test_datasets/denticola/genomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 31 sequences in 2018-test_datasets/denticola/genomic/GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_900164975.1_16852_2_85_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/genomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/genomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 32 sequences in 2018-test_datasets/denticola/genomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==genomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000147075.1_ASM14707v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000147075.1_ASM14707v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2203 sequences in 2018-test_datasets/denticola/protein/GCA_000147075.1_ASM14707v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000147075.1_ASM14707v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000216715.2_CLC_glsol140_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000216715.2_CLC_glsol140_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000216715.2_CLC_glsol140_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000217015.3_CLC_glsol119_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000217015.3_CLC_glsol119_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000217015.3_CLC_glsol119_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000217655.1_ASM21765v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000217655.1_ASM21765v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1010 sequences in 2018-test_datasets/denticola/protein/GCA_000217655.1_ASM21765v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000217655.1_ASM21765v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000222305.1_ASM22230v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000222305.1_ASM22230v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1463 sequences in 2018-test_datasets/denticola/protein/GCA_000222305.1_ASM22230v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000222305.1_ASM22230v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000236685.1_ASM23668v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000236685.1_ASM23668v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3159 sequences in 2018-test_datasets/denticola/protein/GCA_000236685.1_ASM23668v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000236685.1_ASM23668v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000239475.1_ASM23947v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000239475.1_ASM23947v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 956 sequences in 2018-test_datasets/denticola/protein/GCA_000239475.1_ASM23947v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000239475.1_ASM23947v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000242595.3_ASM24259v3_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000242595.3_ASM24259v3_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2782 sequences in 2018-test_datasets/denticola/protein/GCA_000242595.3_ASM24259v3_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000242595.3_ASM24259v3_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000246415.2_CLC_glsol068_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000246415.2_CLC_glsol068_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000246415.2_CLC_glsol068_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1059 sequences in 2018-test_datasets/denticola/protein/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000246815.1_ASM24681v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000260795.1_ASM26079v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000260795.1_ASM26079v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2570 sequences in 2018-test_datasets/denticola/protein/GCA_000260795.1_ASM26079v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000260795.1_ASM26079v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000301975.1_ASM30197v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000301975.1_ASM30197v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000301975.1_ASM30197v1_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2531 sequences in 2018-test_datasets/denticola/protein/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2647 sequences in 2018-test_datasets/denticola/protein/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2569 sequences in 2018-test_datasets/denticola/protein/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2705 sequences in 2018-test_datasets/denticola/protein/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2520 sequences in 2018-test_datasets/denticola/protein/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2632 sequences in 2018-test_datasets/denticola/protein/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2551 sequences in 2018-test_datasets/denticola/protein/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000342705.1_CA_glsol153_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000342705.1_CA_glsol153_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000342705.1_CA_glsol153_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000343205.1_CA_glsol121_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000343205.1_CA_glsol121_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000343205.1_CA_glsol121_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000382565.1_ASM38256v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000382565.1_ASM38256v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 832 sequences in 2018-test_datasets/denticola/protein/GCA_000382565.1_ASM38256v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000382565.1_ASM38256v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2610 sequences in 2018-test_datasets/denticola/protein/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2793 sequences in 2018-test_datasets/denticola/protein/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2614 sequences in 2018-test_datasets/denticola/protein/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000426705.1_ASM42670v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000426705.1_ASM42670v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000426705.1_ASM42670v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000465015.1_Treponema_pedis_T_M1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000465015.1_Treponema_pedis_T_M1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000465015.1_Treponema_pedis_T_M1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000468055.1_ASM46805v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000468055.1_ASM46805v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2175 sequences in 2018-test_datasets/denticola/protein/GCA_000468055.1_ASM46805v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000468055.1_ASM46805v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000500065.1_AB2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000500065.1_AB2_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000500065.1_AB2_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000501035.1_BorGarIPT95_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000501035.1_BorGarIPT95_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000501035.1_BorGarIPT95_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000501115.1_BorGarIPT101_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000501115.1_BorGarIPT101_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000501115.1_BorGarIPT101_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000501555.1_BorAfzIPT109_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000501555.1_BorAfzIPT109_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000501555.1_BorAfzIPT109_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000502195.1_BorAfzIPT138_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000502195.1_BorAfzIPT138_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000502195.1_BorAfzIPT138_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000507245.1_ASM50724v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000507245.1_ASM50724v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3397 sequences in 2018-test_datasets/denticola/protein/GCA_000507245.1_ASM50724v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000507245.1_ASM50724v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2683 sequences in 2018-test_datasets/denticola/protein/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000568735.2_ASM56873v2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000568735.2_ASM56873v2_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 1085 sequences in 2018-test_datasets/denticola/protein/GCA_000568735.2_ASM56873v2_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000568735.2_ASM56873v2_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000758165.1_Spiroch1.0_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000758165.1_Spiroch1.0_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2303 sequences in 2018-test_datasets/denticola/protein/GCA_000758165.1_Spiroch1.0_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000758165.1_Spiroch1.0_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000768055.1_ASM76805v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000768055.1_ASM76805v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2732 sequences in 2018-test_datasets/denticola/protein/GCA_000768055.1_ASM76805v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000768055.1_ASM76805v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000775995.1_ASM77599v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000775995.1_ASM77599v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2236 sequences in 2018-test_datasets/denticola/protein/GCA_000775995.1_ASM77599v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000775995.1_ASM77599v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000802305.1_ASM80230v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000802305.1_ASM80230v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3509 sequences in 2018-test_datasets/denticola/protein/GCA_000802305.1_ASM80230v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000802305.1_ASM80230v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000818865.1_ASM81886v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000818865.1_ASM81886v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_000818865.1_ASM81886v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_000941035.1_ASM94103v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_000941035.1_ASM94103v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4141 sequences in 2018-test_datasets/denticola/protein/GCA_000941035.1_ASM94103v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_000941035.1_ASM94103v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001281815.1_ASM128181v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001281815.1_ASM128181v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 5401 sequences in 2018-test_datasets/denticola/protein/GCA_001281815.1_ASM128181v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001281815.1_ASM128181v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001405175.1_13470_2_54_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001405175.1_13470_2_54_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3201 sequences in 2018-test_datasets/denticola/protein/GCA_001405175.1_13470_2_54_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001405175.1_13470_2_54_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001443305.1_ASM144330v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001443305.1_ASM144330v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3754 sequences in 2018-test_datasets/denticola/protein/GCA_001443305.1_ASM144330v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001443305.1_ASM144330v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001567655.1_ASM156765v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001567655.1_ASM156765v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_001567655.1_ASM156765v1_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001604335.1_ASM160433v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001604335.1_ASM160433v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_001604335.1_ASM160433v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001712955.1_ASM171295v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001712955.1_ASM171295v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 970 sequences in 2018-test_datasets/denticola/protein/GCA_001712955.1_ASM171295v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001712955.1_ASM171295v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 971 sequences in 2018-test_datasets/denticola/protein/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001713255.1_ASM171325v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001746215.1_ASM174621v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001746215.1_ASM174621v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2455 sequences in 2018-test_datasets/denticola/protein/GCA_001746215.1_ASM174621v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001746215.1_ASM174621v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001829125.1_ASM182912v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001829125.1_ASM182912v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3454 sequences in 2018-test_datasets/denticola/protein/GCA_001829125.1_ASM182912v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001829125.1_ASM182912v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001829165.1_ASM182916v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001829165.1_ASM182916v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4234 sequences in 2018-test_datasets/denticola/protein/GCA_001829165.1_ASM182916v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001829165.1_ASM182916v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001829295.1_ASM182929v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001829295.1_ASM182929v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2893 sequences in 2018-test_datasets/denticola/protein/GCA_001829295.1_ASM182929v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001829295.1_ASM182929v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001829415.1_ASM182941v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001829415.1_ASM182941v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3176 sequences in 2018-test_datasets/denticola/protein/GCA_001829415.1_ASM182941v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001829415.1_ASM182941v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001829475.1_ASM182947v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001829475.1_ASM182947v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2271 sequences in 2018-test_datasets/denticola/protein/GCA_001829475.1_ASM182947v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001829475.1_ASM182947v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001829505.1_ASM182950v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001829505.1_ASM182950v1_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 2855 sequences in 2018-test_datasets/denticola/protein/GCA_001829505.1_ASM182950v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001829505.1_ASM182950v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001830585.1_ASM183058v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001830585.1_ASM183058v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3119 sequences in 2018-test_datasets/denticola/protein/GCA_001830585.1_ASM183058v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001830585.1_ASM183058v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001830635.1_ASM183063v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001830635.1_ASM183063v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2898 sequences in 2018-test_datasets/denticola/protein/GCA_001830635.1_ASM183063v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001830635.1_ASM183063v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001894145.1_ASM189414v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001894145.1_ASM189414v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 5527 sequences in 2018-test_datasets/denticola/protein/GCA_001894145.1_ASM189414v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_001894145.1_ASM189414v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001896735.1_ASM189673v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001896735.1_ASM189673v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_001896735.1_ASM189673v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_001940825.1_ASM194082v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_001940825.1_ASM194082v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_001940825.1_ASM194082v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_002009835.1_ASM200983v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_002009835.1_ASM200983v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_002009835.1_ASM200983v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_002069655.1_ASM206965v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_002069655.1_ASM206965v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2059 sequences in 2018-test_datasets/denticola/protein/GCA_002069655.1_ASM206965v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_002069655.1_ASM206965v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_002069965.1_ASM206996v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_002069965.1_ASM206996v1_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 2775 sequences in 2018-test_datasets/denticola/protein/GCA_002069965.1_ASM206996v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_002069965.1_ASM206996v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_900018035.1_7521_5_26_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_900018035.1_7521_5_26_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2780 sequences in 2018-test_datasets/denticola/protein/GCA_900018035.1_7521_5_26_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_900018035.1_7521_5_26_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_900147745.1_138_1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_900147745.1_138_1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/denticola/protein/GCA_900147745.1_138_1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2898 sequences in 2018-test_datasets/denticola/protein/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_900164975.1_16852_2_85_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_900164975.1_16852_2_85_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3654 sequences in 2018-test_datasets/denticola/protein/GCA_900164975.1_16852_2_85_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_900164975.1_16852_2_85_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/denticola/protein/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/denticola/protein/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2193 sequences in 2018-test_datasets/denticola/protein/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000007585.1_ASM758v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000010505.1_ASM1050v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000064305.2_ASM6430v2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 39 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 61 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000162415.1_ASM16241v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 250 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000174775.1_ASM17477v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000212375.1_ASM21237v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 463 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000217375.2_CLC_glsol081_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 12 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 851 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000417845.2_ASM41784v2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 73 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000428105.1_ASM42810v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 21 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000439065.1_ASM43906v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 113 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000467955.1_ASM46795v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 29 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000482365.1_ASM48236v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 117 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000503975.1_SJD2_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1291 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000510525.1_BU063sc679_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1263 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000510545.1_BU063sc13_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 299 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000583675.1_ASM58367v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 84 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000599245.1_ASM59924v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 65 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000599325.1_ASM59932v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 44 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000614325.1_ASM61432v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 92 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000614865.1_ASM61486v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 105 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000614895.1_ASM61489v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000626635.1_ASM62663v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000739415.1_ASM73941v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 31 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000765945.1_ASM76594v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 30 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000768975.1_ASM76897v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 47 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000769015.1_ASM76901v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 32 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000769035.1_ASM76903v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 19 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000769135.1_ASM76913v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 336 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000945515.1_Velvet_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 19 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000949475.1_ASM94947v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 33 sequences in 2018-test_datasets/gingivalis/genomic/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_000993115.1_ASM99311v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 175 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001049535.1_3731_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 179 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001064135.1_ASM106413v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 196 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001261715.1_HGC4_v3_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001263815.1_ASM126381v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001274615.1_ASM127461v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 2 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 20 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001404575.1_13470_2_67_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 81 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001406635.1_13470_2_63_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 109 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001438125.1_ASM143812v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 77 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001438205.1_ASM143820v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001443605.1_ASM144360v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 155 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 92 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001512095.1_7521_8_38_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 493 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001602775.1_ASM160277v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 301 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001607025.1_ASM160702v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 206 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001637845.1_ASM163784v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 52 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001670785.1_RCAD0183_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 123 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001724935.1_ASM172493v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 110 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001768555.1_ASM176855v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 103 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001808555.1_ASM180855v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 88 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001873105.1_ASM187310v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 1 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001889045.1_ASM188904v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 191 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001896575.1_ASM189657v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 99 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001897595.1_ASM189759v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 80 sequences in 2018-test_datasets/gingivalis/genomic/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_001898165.1_ASM189816v1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 24 sequences in 2018-test_datasets/gingivalis/genomic/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_900085525.1_12045_7_37_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 68 sequences in 2018-test_datasets/gingivalis/genomic/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_900089535.1_A04_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 94 sequences in 2018-test_datasets/gingivalis/genomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 32 sequences in 2018-test_datasets/gingivalis/genomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 33 sequences in 2018-test_datasets/gingivalis/genomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 47 sequences in 2018-test_datasets/gingivalis/genomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 51 sequences in 2018-test_datasets/gingivalis/genomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 81 sequences in 2018-test_datasets/gingivalis/genomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 72 sequences in 2018-test_datasets/gingivalis/genomic/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_900157215.1_Strain_3-3_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/genomic/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/genomic/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\u001b[Kcalculated 3 signatures for 56 sequences in 2018-test_datasets/gingivalis/genomic/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/genomic/GCA_900157325.1_3A1_genomic.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1909 sequences in 2018-test_datasets/gingivalis/protein/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000007585.1_ASM758v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2090 sequences in 2018-test_datasets/gingivalis/protein/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000010505.1_ASM1050v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2446 sequences in 2018-test_datasets/gingivalis/protein/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000064305.2_ASM6430v2_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4706 sequences in 2018-test_datasets/gingivalis/protein/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2926 sequences in 2018-test_datasets/gingivalis/protein/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000162415.1_ASM16241v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1977 sequences in 2018-test_datasets/gingivalis/protein/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000174775.1_ASM17477v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1699 sequences in 2018-test_datasets/gingivalis/protein/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000212375.1_ASM21237v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_000217375.2_CLC_glsol081_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 4041 sequences in 2018-test_datasets/gingivalis/protein/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2354 sequences in 2018-test_datasets/gingivalis/protein/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3209 sequences in 2018-test_datasets/gingivalis/protein/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000417845.2_ASM41784v2_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_000428105.1_ASM42810v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_000439065.1_ASM43906v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2236 sequences in 2018-test_datasets/gingivalis/protein/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000467955.1_ASM46795v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_000482365.1_ASM48236v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2020 sequences in 2018-test_datasets/gingivalis/protein/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000503975.1_SJD2_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000510525.1_BU063sc679_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 2935 sequences in 2018-test_datasets/gingivalis/protein/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000510525.1_BU063sc679_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2371 sequences in 2018-test_datasets/gingivalis/protein/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000510545.1_BU063sc13_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4663 sequences in 2018-test_datasets/gingivalis/protein/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000583675.1_ASM58367v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4668 sequences in 2018-test_datasets/gingivalis/protein/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000599245.1_ASM59924v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4638 sequences in 2018-test_datasets/gingivalis/protein/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000599325.1_ASM59932v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_000614325.1_ASM61432v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000626635.1_ASM62663v1_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 3971 sequences in 2018-test_datasets/gingivalis/protein/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1958 sequences in 2018-test_datasets/gingivalis/protein/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000739415.1_ASM73941v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1904 sequences in 2018-test_datasets/gingivalis/protein/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000765945.1_ASM76594v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1607 sequences in 2018-test_datasets/gingivalis/protein/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000768975.1_ASM76897v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1840 sequences in 2018-test_datasets/gingivalis/protein/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000769015.1_ASM76901v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1843 sequences in 2018-test_datasets/gingivalis/protein/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000769035.1_ASM76903v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1433 sequences in 2018-test_datasets/gingivalis/protein/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000769135.1_ASM76913v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_000945515.1_Velvet_assembly_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4126 sequences in 2018-test_datasets/gingivalis/protein/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000949475.1_ASM94947v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 978 sequences in 2018-test_datasets/gingivalis/protein/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_000993115.1_ASM99311v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001049535.1_3731_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001049535.1_3731_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_001064135.1_ASM106413v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_001261715.1_HGC4_v3_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2004 sequences in 2018-test_datasets/gingivalis/protein/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001263815.1_ASM126381v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2002 sequences in 2018-test_datasets/gingivalis/protein/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001274615.1_ASM127461v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3545 sequences in 2018-test_datasets/gingivalis/protein/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001404575.1_13470_2_67_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001406635.1_13470_2_63_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 3813 sequences in 2018-test_datasets/gingivalis/protein/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001406635.1_13470_2_63_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1662 sequences in 2018-test_datasets/gingivalis/protein/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001438125.1_ASM143812v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1495 sequences in 2018-test_datasets/gingivalis/protein/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001438205.1_ASM143820v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3865 sequences in 2018-test_datasets/gingivalis/protein/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4589 sequences in 2018-test_datasets/gingivalis/protein/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_001512095.1_7521_8_38_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_001602775.1_ASM160277v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 5573 sequences in 2018-test_datasets/gingivalis/protein/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001607025.1_ASM160702v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_001637845.1_ASM163784v1_protein.faa.gz'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2017 sequences in 2018-test_datasets/gingivalis/protein/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001670785.1_RCAD0183_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 898 sequences in 2018-test_datasets/gingivalis/protein/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001724935.1_ASM172493v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1959 sequences in 2018-test_datasets/gingivalis/protein/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001768555.1_ASM176855v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1615 sequences in 2018-test_datasets/gingivalis/protein/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001808555.1_ASM180855v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2282 sequences in 2018-test_datasets/gingivalis/protein/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001873105.1_ASM187310v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2661 sequences in 2018-test_datasets/gingivalis/protein/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001889045.1_ASM188904v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_001896575.1_ASM189657v1_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4208 sequences in 2018-test_datasets/gingivalis/protein/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001897595.1_ASM189759v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 2942 sequences in 2018-test_datasets/gingivalis/protein/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_001898165.1_ASM189816v1_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_900085525.1_12045_7_37_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[Kcalculated 3 signatures for 4961 sequences in 2018-test_datasets/gingivalis/protein/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_900085525.1_12045_7_37_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_900089535.1_A04_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_900089535.1_A04_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for '2018-test_datasets/gingivalis/protein/GCA_900089535.1_A04_protein.faa.gz'\n",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3334 sequences in 2018-test_datasets/gingivalis/protein/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3560 sequences in 2018-test_datasets/gingivalis/protein/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 3925 sequences in 2018-test_datasets/gingivalis/protein/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4005 sequences in 2018-test_datasets/gingivalis/protein/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4015 sequences in 2018-test_datasets/gingivalis/protein/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 4109 sequences in 2018-test_datasets/gingivalis/protein/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\u001b[Kcalculated 3 signatures for 1779 sequences in 2018-test_datasets/gingivalis/protein/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\u001b[K== This is sourmash version 2.2.0. ==/proteomic/GCA_900157215.1_Strain_3-3_protein.sig. Note: signature license is CC0.\n",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\u001b[Kcomputing signatures for files: 2018-test_datasets/gingivalis/protein/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\u001b[K... reading sequences from 2018-test_datasets/gingivalis/protein/GCA_900157325.1_3A1_protein.faa.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r",
      "\u001b[K...2018-test_datasets/gingivalis/protein/GCA_900157325.1_3A1_protein.faa.gz 1845 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1846 sequences in 2018-test_datasets/gingivalis/protein/GCA_900157325.1_3A1_protein.faa.gz\r\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig. Note: signature license is CC0.\r"
     ]
    }
   ],
   "source": [
    "#unzip and compute signature\n",
    "\n",
    "!for infile in 2018-test_datasets/bacteroides/genomic/*fna.gz; do out_name=$(basename $infile .fna.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/bacteroides/genomic/${out_name}.sig ${infile}; done\n",
    "!for infile in 2018-test_datasets/bacteroides/protein/*faa.gz; do out_name=$(basename $infile .faa.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/bacteroides/proteomic/${out_name}.sig ${infile}; done\n",
    "\n",
    "\n",
    "!for infile in 2018-test_datasets/denticola/genomic/*fna.gz; do out_name=$(basename $infile .fna.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/denticola/genomic/${out_name}.sig ${infile}; done\n",
    "!for infile in 2018-test_datasets/denticola/protein/*faa.gz; do out_name=$(basename $infile .faa.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/denticola/proteomic/${out_name}.sig ${infile}; done\n",
    "\n",
    "\n",
    "!for infile in 2018-test_datasets/gingivalis/genomic/*fna.gz; do out_name=$(basename $infile .fna.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/gingivalis/genomic/${out_name}.sig ${infile}; done\n",
    "!for infile in 2018-test_datasets/gingivalis/protein/*faa.gz; do out_name=$(basename $infile .faa.gz); sourmash compute -k 21,31,51 --scaled 2000 --track-abundance -o sigs/gingivalis/proteomic/${out_name}.sig ${infile}; done\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\r\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\r\n",
      "\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 40 signatures total.\r\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\r\n",
      "\r",
      "\u001b[K\r\n",
      "min similarity in matrix: 0.000\r\n"
     ]
    }
   ],
   "source": [
    "#%pwd\n",
    "#!mkdir -p sourmash_compare/compare_CSVs\n",
    "\n",
    "# 31-mer\n",
    "##Bacteroides, genomic\n",
    "#!sourmash compare -k 31 -o sourmash_compare/bacteroides_k31_genomic_comp sigs/bacteroides/genomic/*.sig\n",
    "#!sourmash plot --labels --pdf sourmash_compare/bacteroides_k31_genomic_comp\n",
    "#%mv *.pdf /mnt/c/Users/user/Desktop/CheckPlotPDFs/\n",
    "#!sourmash compare -k 31 --csv sourmash_compare/compare_CSVs/bacteroides_k31_genomic_comp.csv sigs/bacteroides/genomic/*.sig\n",
    "#%mv sourmash_compare/compare_CSVs/*.csv /mnt/c/Users/user/Desktop/ROTATIONS/DIBlabRotationProject/CompCSVs/\n",
    "\n",
    "##Bacteroides, proteomic\n",
    "#!sourmash compare -k 31 -o sourmash_compare/bacteroides_k31_proteomic_comp sigs/bacteroides/proteomic/*.sig\n",
    "#!sourmash plot --labels --pdf sourmash_compare/bacteroides_k31_proteomic_comp\n",
    "#%mv *.pdf /mnt/c/Users/user/Desktop/CheckPlotPDFs/\n",
    "##This doesnt appear useful (looks like only one seq is present)\n",
    "#!sourmash compare -k 31 --csv sourmash_compare/compare_CSVs/bacteroides_k31_proteomic_comp.csv sigs/bacteroides/proteomic/*.sig\n",
    "#%mv sourmash_compare/compare_CSVs/*.csv /mnt/c/Users/user/Desktop/ROTATIONS/DIBlabRotationProject/CompCSVs/\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "we gon make a turduckin style for loop:\n",
    "\n",
    "for space= genomic, proteomic\n",
    "for organism= bacteroides, gingivalis, denticola\n",
    "for k=31, 21, 51\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "hello\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "\n",
    "#unzip and compute signature\n",
    "#seqSpace = {\"genomic\", \"proteomic\"}\n",
    "#organism = {\"bacteroides\", \"gingivalis\", \"denticola\"}\n",
    "#kMer = {31, 21, 51}\n",
    "\n",
    "#for z in seqSpace:\n",
    "#    for y in organism:\n",
    "#        for x in kMer\n",
    " #       do\n",
    "  #          sourmash compare -k (x) -o sourmash_compare/bacteroides_k31_genomic_comp sigs/bacteroides/genomic/*.sig\n",
    "   #     done\n",
    "#!sourmash plot --labels --pdf sourmash_compare/bacteroides_k31_genomic_comp\n",
    "#%mv *.pdf /mnt/c/Users/user/Desktop/CheckPlotPDFs/\n",
    "#!sourmash compare -k 31 --csv sourmash_compare/compare_CSVs/bacteroides_k31_genomic_comp.csv sigs/bacteroides/genomic/*.sig\n",
    "#%mv sourmash_compare/compare_CSVs/*.csv /mnt/c/Users/user/Desktop/ROTATIONS/DIBlabRotationProject/CompCSVs/\n",
    "\n",
    "#!sourmash compare -k 31 -o sourmash_compare/bacteroides_k31_genomic_comp sigs/bacteroides/genomic/*.sig\n",
    "#!sourmash plot --labels --pdf sourmash_compare/bacteroides_k31_genomic_comp\n",
    "#%mv *.pdf /mnt/c/Users/user/Desktop/CheckPlotPDFs/\n",
    "#!sourmash compare -k 31 --csv sourmash_compare/compare_CSVs/bacteroides_k31_genomic_comp.csv sigs/bacteroides/genomic/*.sig\n",
    "#%mv sourmash_compare/compare_CSVs/*.csv /mnt/c/Users/user/Desktop/ROTATIONS/DIBlabRotationProject/CompCSVs/\n",
    "\n",
    "\n",
    "echo hello"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n",
      "min similarity in matrix: 0.000\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049555.1_7330_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 61 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/bacteroides_k21_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/bacteroides_k21_genomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/bacteroides_k21_genomic_comp...\n",
      "\r",
      "\u001b[K...got 61 x 61 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/bacteroides_k21_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => bacteroides_k21_genomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: bacteroides_k21_genomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: bacteroides_k21_genomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049555.1_7330_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 61 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000007585.1_ASM758v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000010505.1_ASM1050v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000064305.2_ASM6430v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000162415.1_ASM16241v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000174775.1_ASM17477v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000212375.1_ASM21237v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000217375.2_CLC_glsol081_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000417845.2_ASM41784v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000428105.1_ASM42810v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000439065.1_ASM43906v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000467955.1_ASM46795v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000482365.1_ASM48236v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000503975.1_SJD2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510525.1_BU063sc679_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510545.1_BU063sc13_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000583675.1_ASM58367v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599245.1_ASM59924v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599325.1_ASM59932v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614325.1_ASM61432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000739415.1_ASM73941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000765945.1_ASM76594v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000768975.1_ASM76897v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769015.1_ASM76901v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769035.1_ASM76903v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769135.1_ASM76913v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000945515.1_Velvet_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000949475.1_ASM94947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000993115.1_ASM99311v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001064135.1_ASM106413v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001261715.1_HGC4_v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001263815.1_ASM126381v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001274615.1_ASM127461v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001404575.1_13470_2_67_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001406635.1_13470_2_63_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438125.1_ASM143812v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438205.1_ASM143820v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001512095.1_7521_8_38_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001602775.1_ASM160277v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001607025.1_ASM160702v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001637845.1_ASM163784v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001670785.1_RCAD0183_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001724935.1_ASM172493v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001768555.1_ASM176855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001808555.1_ASM180855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001873105.1_ASM187310v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001889045.1_ASM188904v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001896575.1_ASM189657v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001897595.1_ASM189759v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001898165.1_ASM189816v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900085525.1_12045_7_37_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900089535.1_A04_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157215.1_Strain_3-3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157325.1_3A1_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/gingivalis_k21_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/gingivalis_k21_genomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/gingivalis_k21_genomic_comp...\n",
      "\r",
      "\u001b[K...got 69 x 69 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/gingivalis_k21_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => gingivalis_k21_genomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: gingivalis_k21_genomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: gingivalis_k21_genomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000007585.1_ASM758v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000010505.1_ASM1050v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000064305.2_ASM6430v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000162415.1_ASM16241v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000174775.1_ASM17477v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000212375.1_ASM21237v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000217375.2_CLC_glsol081_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000417845.2_ASM41784v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000428105.1_ASM42810v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000439065.1_ASM43906v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000467955.1_ASM46795v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000482365.1_ASM48236v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000503975.1_SJD2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510525.1_BU063sc679_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510545.1_BU063sc13_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000583675.1_ASM58367v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599245.1_ASM59924v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599325.1_ASM59932v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614325.1_ASM61432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000739415.1_ASM73941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000765945.1_ASM76594v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000768975.1_ASM76897v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769015.1_ASM76901v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769035.1_ASM76903v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769135.1_ASM76913v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000945515.1_Velvet_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000949475.1_ASM94947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000993115.1_ASM99311v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001064135.1_ASM106413v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001261715.1_HGC4_v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001263815.1_ASM126381v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001274615.1_ASM127461v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001404575.1_13470_2_67_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001406635.1_13470_2_63_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438125.1_ASM143812v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438205.1_ASM143820v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001512095.1_7521_8_38_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001602775.1_ASM160277v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001607025.1_ASM160702v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001637845.1_ASM163784v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001670785.1_RCAD0183_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001724935.1_ASM172493v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001768555.1_ASM176855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001808555.1_ASM180855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001873105.1_ASM187310v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001889045.1_ASM188904v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001896575.1_ASM189657v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001897595.1_ASM189759v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001898165.1_ASM189816v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900085525.1_12045_7_37_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900089535.1_A04_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157215.1_Strain_3-3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157325.1_3A1_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000147075.1_ASM14707v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000216715.2_CLC_glsol140_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217015.3_CLC_glsol119_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217655.1_ASM21765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000222305.1_ASM22230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000236685.1_ASM23668v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000239475.1_ASM23947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000242595.3_ASM24259v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246415.2_CLC_glsol068_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246815.1_ASM24681v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000260795.1_ASM26079v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000301975.1_ASM30197v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340645.1_Trep_dent_H-22_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000342705.1_CA_glsol153_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000343205.1_CA_glsol121_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000382565.1_ASM38256v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000426705.1_ASM42670v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000465015.1_Treponema_pedis_T_M1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000468055.1_ASM46805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000500065.1_AB2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501035.1_BorGarIPT95_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501115.1_BorGarIPT101_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501555.1_BorAfzIPT109_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000502195.1_BorAfzIPT138_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000507245.1_ASM50724v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000568735.2_ASM56873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000758165.1_Spiroch1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000768055.1_ASM76805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000775995.1_ASM77599v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000802305.1_ASM80230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000818865.1_ASM81886v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000941035.1_ASM94103v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001281815.1_ASM128181v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001405175.1_13470_2_54_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001443305.1_ASM144330v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001567655.1_ASM156765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001604335.1_ASM160433v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001712955.1_ASM171295v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001713255.1_ASM171325v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001746215.1_ASM174621v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829125.1_ASM182912v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829165.1_ASM182916v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829295.1_ASM182929v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829415.1_ASM182941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829475.1_ASM182947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829505.1_ASM182950v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830585.1_ASM183058v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830635.1_ASM183063v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001894145.1_ASM189414v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001896735.1_ASM189673v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001940825.1_ASM194082v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002009835.1_ASM200983v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069655.1_ASM206965v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069965.1_ASM206996v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900018035.1_7521_5_26_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900147745.1_138_1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900164975.1_16852_2_85_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/denticola_k21_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/denticola_k21_genomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/denticola_k21_genomic_comp...\n",
      "\r",
      "\u001b[K...got 69 x 69 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/denticola_k21_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => denticola_k21_genomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: denticola_k21_genomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: denticola_k21_genomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000147075.1_ASM14707v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000216715.2_CLC_glsol140_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217015.3_CLC_glsol119_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217655.1_ASM21765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000222305.1_ASM22230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000236685.1_ASM23668v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000239475.1_ASM23947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000242595.3_ASM24259v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246415.2_CLC_glsol068_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246815.1_ASM24681v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000260795.1_ASM26079v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000301975.1_ASM30197v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340645.1_Trep_dent_H-22_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000342705.1_CA_glsol153_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000343205.1_CA_glsol121_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000382565.1_ASM38256v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000426705.1_ASM42670v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000465015.1_Treponema_pedis_T_M1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000468055.1_ASM46805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000500065.1_AB2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501035.1_BorGarIPT95_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501115.1_BorGarIPT101_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501555.1_BorAfzIPT109_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000502195.1_BorAfzIPT138_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000507245.1_ASM50724v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000568735.2_ASM56873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000758165.1_Spiroch1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000768055.1_ASM76805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000775995.1_ASM77599v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000802305.1_ASM80230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000818865.1_ASM81886v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000941035.1_ASM94103v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001281815.1_ASM128181v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001405175.1_13470_2_54_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001443305.1_ASM144330v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001567655.1_ASM156765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001604335.1_ASM160433v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001712955.1_ASM171295v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001713255.1_ASM171325v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001746215.1_ASM174621v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829125.1_ASM182912v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829165.1_ASM182916v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829295.1_ASM182929v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829415.1_ASM182941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829475.1_ASM182947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829505.1_ASM182950v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830585.1_ASM183058v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830635.1_ASM183063v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001894145.1_ASM189414v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001896735.1_ASM189673v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001940825.1_ASM194082v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002009835.1_ASM200983v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069655.1_ASM206965v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069965.1_ASM206996v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900018035.1_7521_5_26_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900147745.1_138_1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900164975.1_16852_2_85_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049555.1_7330_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 61 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/bacteroides_k31_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/bacteroides_k31_genomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/bacteroides_k31_genomic_comp...\n",
      "\r",
      "\u001b[K...got 61 x 61 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/bacteroides_k31_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => bacteroides_k31_genomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: bacteroides_k31_genomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: bacteroides_k31_genomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049555.1_7330_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 61 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000007585.1_ASM758v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000010505.1_ASM1050v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000064305.2_ASM6430v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000162415.1_ASM16241v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000174775.1_ASM17477v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000212375.1_ASM21237v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000217375.2_CLC_glsol081_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000417845.2_ASM41784v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000428105.1_ASM42810v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000439065.1_ASM43906v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000467955.1_ASM46795v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000482365.1_ASM48236v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000503975.1_SJD2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510525.1_BU063sc679_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510545.1_BU063sc13_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000583675.1_ASM58367v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599245.1_ASM59924v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599325.1_ASM59932v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614325.1_ASM61432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000739415.1_ASM73941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000765945.1_ASM76594v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000768975.1_ASM76897v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769015.1_ASM76901v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769035.1_ASM76903v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769135.1_ASM76913v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000945515.1_Velvet_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000949475.1_ASM94947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000993115.1_ASM99311v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001064135.1_ASM106413v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001261715.1_HGC4_v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001263815.1_ASM126381v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001274615.1_ASM127461v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001404575.1_13470_2_67_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001406635.1_13470_2_63_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438125.1_ASM143812v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438205.1_ASM143820v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001512095.1_7521_8_38_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001602775.1_ASM160277v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001607025.1_ASM160702v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001637845.1_ASM163784v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001670785.1_RCAD0183_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001724935.1_ASM172493v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001768555.1_ASM176855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001808555.1_ASM180855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001873105.1_ASM187310v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001889045.1_ASM188904v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001896575.1_ASM189657v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001897595.1_ASM189759v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001898165.1_ASM189816v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900085525.1_12045_7_37_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900089535.1_A04_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157215.1_Strain_3-3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157325.1_3A1_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/gingivalis_k31_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/gingivalis_k31_genomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/gingivalis_k31_genomic_comp...\n",
      "\r",
      "\u001b[K...got 69 x 69 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/gingivalis_k31_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => gingivalis_k31_genomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: gingivalis_k31_genomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: gingivalis_k31_genomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000007585.1_ASM758v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000010505.1_ASM1050v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000064305.2_ASM6430v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000162415.1_ASM16241v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000174775.1_ASM17477v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000212375.1_ASM21237v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000217375.2_CLC_glsol081_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000417845.2_ASM41784v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000428105.1_ASM42810v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000439065.1_ASM43906v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000467955.1_ASM46795v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000482365.1_ASM48236v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000503975.1_SJD2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510525.1_BU063sc679_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510545.1_BU063sc13_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000583675.1_ASM58367v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599245.1_ASM59924v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599325.1_ASM59932v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614325.1_ASM61432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000739415.1_ASM73941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000765945.1_ASM76594v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000768975.1_ASM76897v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769015.1_ASM76901v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769035.1_ASM76903v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769135.1_ASM76913v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000945515.1_Velvet_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000949475.1_ASM94947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000993115.1_ASM99311v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001064135.1_ASM106413v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001261715.1_HGC4_v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001263815.1_ASM126381v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001274615.1_ASM127461v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001404575.1_13470_2_67_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001406635.1_13470_2_63_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438125.1_ASM143812v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438205.1_ASM143820v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001512095.1_7521_8_38_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001602775.1_ASM160277v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001607025.1_ASM160702v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001637845.1_ASM163784v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001670785.1_RCAD0183_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001724935.1_ASM172493v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001768555.1_ASM176855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001808555.1_ASM180855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001873105.1_ASM187310v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001889045.1_ASM188904v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001896575.1_ASM189657v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001897595.1_ASM189759v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001898165.1_ASM189816v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900085525.1_12045_7_37_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900089535.1_A04_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157215.1_Strain_3-3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157325.1_3A1_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000147075.1_ASM14707v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000216715.2_CLC_glsol140_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217015.3_CLC_glsol119_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217655.1_ASM21765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000222305.1_ASM22230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000236685.1_ASM23668v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000239475.1_ASM23947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000242595.3_ASM24259v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246415.2_CLC_glsol068_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246815.1_ASM24681v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000260795.1_ASM26079v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000301975.1_ASM30197v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340645.1_Trep_dent_H-22_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000342705.1_CA_glsol153_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000343205.1_CA_glsol121_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000382565.1_ASM38256v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000426705.1_ASM42670v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000465015.1_Treponema_pedis_T_M1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000468055.1_ASM46805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000500065.1_AB2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501035.1_BorGarIPT95_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501115.1_BorGarIPT101_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501555.1_BorAfzIPT109_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000502195.1_BorAfzIPT138_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000507245.1_ASM50724v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000568735.2_ASM56873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000758165.1_Spiroch1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000768055.1_ASM76805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000775995.1_ASM77599v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000802305.1_ASM80230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000818865.1_ASM81886v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000941035.1_ASM94103v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001281815.1_ASM128181v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001405175.1_13470_2_54_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001443305.1_ASM144330v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001567655.1_ASM156765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001604335.1_ASM160433v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001712955.1_ASM171295v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001713255.1_ASM171325v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001746215.1_ASM174621v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829125.1_ASM182912v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829165.1_ASM182916v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829295.1_ASM182929v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829415.1_ASM182941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829475.1_ASM182947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829505.1_ASM182950v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830585.1_ASM183058v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830635.1_ASM183063v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001894145.1_ASM189414v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001896735.1_ASM189673v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001940825.1_ASM194082v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002009835.1_ASM200983v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069655.1_ASM206965v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069965.1_ASM206996v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900018035.1_7521_5_26_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900147745.1_138_1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900164975.1_16852_2_85_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/denticola_k31_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/denticola_k31_genomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/denticola_k31_genomic_comp...\n",
      "\r",
      "\u001b[K...got 69 x 69 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/denticola_k31_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => denticola_k31_genomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: denticola_k31_genomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: denticola_k31_genomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000147075.1_ASM14707v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000216715.2_CLC_glsol140_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217015.3_CLC_glsol119_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217655.1_ASM21765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000222305.1_ASM22230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000236685.1_ASM23668v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000239475.1_ASM23947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000242595.3_ASM24259v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246415.2_CLC_glsol068_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246815.1_ASM24681v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000260795.1_ASM26079v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000301975.1_ASM30197v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340645.1_Trep_dent_H-22_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000342705.1_CA_glsol153_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000343205.1_CA_glsol121_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000382565.1_ASM38256v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000426705.1_ASM42670v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000465015.1_Treponema_pedis_T_M1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000468055.1_ASM46805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000500065.1_AB2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501035.1_BorGarIPT95_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501115.1_BorGarIPT101_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501555.1_BorAfzIPT109_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000502195.1_BorAfzIPT138_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000507245.1_ASM50724v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000568735.2_ASM56873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000758165.1_Spiroch1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000768055.1_ASM76805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000775995.1_ASM77599v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000802305.1_ASM80230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000818865.1_ASM81886v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000941035.1_ASM94103v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001281815.1_ASM128181v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001405175.1_13470_2_54_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001443305.1_ASM144330v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001567655.1_ASM156765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001604335.1_ASM160433v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001712955.1_ASM171295v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001713255.1_ASM171325v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001746215.1_ASM174621v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829125.1_ASM182912v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829165.1_ASM182916v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829295.1_ASM182929v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829415.1_ASM182941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829475.1_ASM182947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829505.1_ASM182950v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830585.1_ASM183058v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830635.1_ASM183063v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001894145.1_ASM189414v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001896735.1_ASM189673v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001940825.1_ASM194082v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002009835.1_ASM200983v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069655.1_ASM206965v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069965.1_ASM206996v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900018035.1_7521_5_26_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900147745.1_138_1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900164975.1_16852_2_85_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049555.1_7330_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 61 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/bacteroides_k51_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/bacteroides_k51_genomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/bacteroides_k51_genomic_comp...\n",
      "\r",
      "\u001b[K...got 61 x 61 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/bacteroides_k51_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => bacteroides_k51_genomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: bacteroides_k51_genomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: bacteroides_k51_genomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000011065.1_ASM1106v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000157015.1_ASM15701v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000159855.2_Bact_sp_3_2_5_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000163055.2_ASM16305v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000224595.1_Prevotella_sp_C561_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000373705.1_ASM37370v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000431275.1_MGS755_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000432495.1_MGS558_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000466425.1_ASM46642v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000495955.1_NUHP2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000501415.1_BorBurgIPT48_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000550025.1_Stap_aure_M1496_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000598325.1_ASM59832v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000611675.1_ASM61167v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000707425.1_CFSAN007430_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000759315.1_04_NF40_HMP9302v01_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000785025.1_ASM78502v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000814325.1_ASM81432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000821925.1_34AVR_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000938735.2_ASM93873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_000974365.1_ASM97436v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001049555.1_7330_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001055755.1_ASM105575v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001304045.1_ASM130404v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001310955.1_ASM131095v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001373135.1_2e6A_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001398875.1_Mother10-2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405895.1_14207_7_22_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001405935.1_13414_6_56_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001413955.1_ASM141395v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001548195.1_ASM154819v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001578565.1_ASM157856v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001595155.1_CFSAN033860_01.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603535.1_ASM160353v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001603695.1_ASM160369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001614375.1_ASM161437v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001659685.2_ASM165968v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001693695.1_ASM169369v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001761325.1_ASM176132v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001816245.1_ASM181624v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_001915365.1_ASM191536v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_002036985.1_ASM203698v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900032635.1_14555_6_27_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900104585.1_PRJEB16348_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900128475.1_PRJEB18047_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/genomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 61 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000007585.1_ASM758v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000010505.1_ASM1050v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000064305.2_ASM6430v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000162415.1_ASM16241v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000174775.1_ASM17477v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000212375.1_ASM21237v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000217375.2_CLC_glsol081_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000417845.2_ASM41784v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000428105.1_ASM42810v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000439065.1_ASM43906v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000467955.1_ASM46795v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000482365.1_ASM48236v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000503975.1_SJD2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510525.1_BU063sc679_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510545.1_BU063sc13_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000583675.1_ASM58367v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599245.1_ASM59924v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599325.1_ASM59932v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614325.1_ASM61432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000739415.1_ASM73941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000765945.1_ASM76594v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000768975.1_ASM76897v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769015.1_ASM76901v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769035.1_ASM76903v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769135.1_ASM76913v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000945515.1_Velvet_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000949475.1_ASM94947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000993115.1_ASM99311v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001064135.1_ASM106413v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001261715.1_HGC4_v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001263815.1_ASM126381v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001274615.1_ASM127461v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001404575.1_13470_2_67_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001406635.1_13470_2_63_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438125.1_ASM143812v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438205.1_ASM143820v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001512095.1_7521_8_38_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001602775.1_ASM160277v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001607025.1_ASM160702v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001637845.1_ASM163784v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001670785.1_RCAD0183_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001724935.1_ASM172493v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001768555.1_ASM176855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001808555.1_ASM180855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001873105.1_ASM187310v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001889045.1_ASM188904v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001896575.1_ASM189657v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001897595.1_ASM189759v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001898165.1_ASM189816v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900085525.1_12045_7_37_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900089535.1_A04_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157215.1_Strain_3-3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157325.1_3A1_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/gingivalis_k51_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/gingivalis_k51_genomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/gingivalis_k51_genomic_comp...\n",
      "\r",
      "\u001b[K...got 69 x 69 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/gingivalis_k51_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => gingivalis_k51_genomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: gingivalis_k51_genomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: gingivalis_k51_genomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000007585.1_ASM758v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000010505.1_ASM1050v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000064305.2_ASM6430v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000162415.1_ASM16241v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000174775.1_ASM17477v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000212375.1_ASM21237v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000217375.2_CLC_glsol081_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000417845.2_ASM41784v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000428105.1_ASM42810v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000439065.1_ASM43906v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000467955.1_ASM46795v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000482365.1_ASM48236v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000503975.1_SJD2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510525.1_BU063sc679_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000510545.1_BU063sc13_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000583675.1_ASM58367v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599245.1_ASM59924v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000599325.1_ASM59932v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614325.1_ASM61432v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614865.1_ASM61486v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000614895.1_ASM61489v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000626635.1_ASM62663v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000739415.1_ASM73941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000765945.1_ASM76594v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000768975.1_ASM76897v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769015.1_ASM76901v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769035.1_ASM76903v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000769135.1_ASM76913v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000945515.1_Velvet_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000949475.1_ASM94947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_000993115.1_ASM99311v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001049535.1_3731_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001064135.1_ASM106413v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001261715.1_HGC4_v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001263815.1_ASM126381v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001274615.1_ASM127461v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001404575.1_13470_2_67_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001406635.1_13470_2_63_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438125.1_ASM143812v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001438205.1_ASM143820v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001443605.1_ASM144360v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001512095.1_7521_8_38_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001602775.1_ASM160277v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001607025.1_ASM160702v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001637845.1_ASM163784v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001670785.1_RCAD0183_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001724935.1_ASM172493v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001768555.1_ASM176855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001808555.1_ASM180855v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001873105.1_ASM187310v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001889045.1_ASM188904v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001896575.1_ASM189657v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001897595.1_ASM189759v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_001898165.1_ASM189816v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900085525.1_12045_7_37_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900089535.1_A04_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157215.1_Strain_3-3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/genomic/GCA_900157325.1_3A1_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000147075.1_ASM14707v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000216715.2_CLC_glsol140_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217015.3_CLC_glsol119_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217655.1_ASM21765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000222305.1_ASM22230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000236685.1_ASM23668v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000239475.1_ASM23947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000242595.3_ASM24259v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246415.2_CLC_glsol068_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246815.1_ASM24681v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000260795.1_ASM26079v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000301975.1_ASM30197v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340645.1_Trep_dent_H-22_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000342705.1_CA_glsol153_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000343205.1_CA_glsol121_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000382565.1_ASM38256v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000426705.1_ASM42670v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000465015.1_Treponema_pedis_T_M1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000468055.1_ASM46805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000500065.1_AB2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501035.1_BorGarIPT95_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501115.1_BorGarIPT101_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501555.1_BorAfzIPT109_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000502195.1_BorAfzIPT138_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000507245.1_ASM50724v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000568735.2_ASM56873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000758165.1_Spiroch1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000768055.1_ASM76805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000775995.1_ASM77599v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000802305.1_ASM80230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000818865.1_ASM81886v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000941035.1_ASM94103v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001281815.1_ASM128181v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001405175.1_13470_2_54_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001443305.1_ASM144330v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001567655.1_ASM156765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001604335.1_ASM160433v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001712955.1_ASM171295v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001713255.1_ASM171325v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001746215.1_ASM174621v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829125.1_ASM182912v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829165.1_ASM182916v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829295.1_ASM182929v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829415.1_ASM182941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829475.1_ASM182947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829505.1_ASM182950v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830585.1_ASM183058v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830635.1_ASM183063v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001894145.1_ASM189414v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001896735.1_ASM189673v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001940825.1_ASM194082v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002009835.1_ASM200983v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069655.1_ASM206965v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069965.1_ASM206996v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900018035.1_7521_5_26_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900147745.1_138_1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900164975.1_16852_2_85_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/denticola_k51_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/denticola_k51_genomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/denticola_k51_genomic_comp...\n",
      "\r",
      "\u001b[K...got 69 x 69 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/denticola_k51_genomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => denticola_k51_genomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: denticola_k51_genomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: denticola_k51_genomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000147075.1_ASM14707v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000216715.2_CLC_glsol140_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217015.3_CLC_glsol119_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000217655.1_ASM21765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000222305.1_ASM22230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000236685.1_ASM23668v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000239475.1_ASM23947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000242595.3_ASM24259v3_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246415.2_CLC_glsol068_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000246815.1_ASM24681v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000260795.1_ASM26079v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000301975.1_ASM30197v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340645.1_Trep_dent_H-22_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000342705.1_CA_glsol153_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000343205.1_CA_glsol121_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000382565.1_ASM38256v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000426705.1_ASM42670v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000465015.1_Treponema_pedis_T_M1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000468055.1_ASM46805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000500065.1_AB2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501035.1_BorGarIPT95_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501115.1_BorGarIPT101_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000501555.1_BorAfzIPT109_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000502195.1_BorAfzIPT138_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000507245.1_ASM50724v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000568735.2_ASM56873v2_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000758165.1_Spiroch1.0_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000768055.1_ASM76805v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000775995.1_ASM77599v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000802305.1_ASM80230v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000818865.1_ASM81886v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_000941035.1_ASM94103v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001281815.1_ASM128181v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001405175.1_13470_2_54_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001443305.1_ASM144330v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001567655.1_ASM156765v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001604335.1_ASM160433v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001712955.1_ASM171295v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001713255.1_ASM171325v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001746215.1_ASM174621v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829125.1_ASM182912v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829165.1_ASM182916v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829295.1_ASM182929v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829415.1_ASM182941v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829475.1_ASM182947v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001829505.1_ASM182950v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830585.1_ASM183058v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001830635.1_ASM183063v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001894145.1_ASM189414v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001896735.1_ASM189673v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_001940825.1_ASM194082v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002009835.1_ASM200983v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069655.1_ASM206965v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_002069965.1_ASM206996v1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900018035.1_7521_5_26_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900147745.1_138_1_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900164975.1_16852_2_85_genomic.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/genomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 69 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/bacteroides_k7_proteomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/bacteroides_k7_proteomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/gingivalis_k7_proteomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/gingivalis_k7_proteomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/denticola_k7_proteomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/denticola_k7_proteomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/bacteroides_k9_proteomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/bacteroides_k9_proteomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/gingivalis_k9_proteomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/gingivalis_k9_proteomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/denticola_k9_proteomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/denticola_k9_proteomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/bacteroides_k12_proteomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/bacteroides_k12_proteomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/gingivalis_k12_proteomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/gingivalis_k12_proteomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/denticola_k12_proteomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/denticola_k12_proteomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 40 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/bacteroides_k21_proteomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/bacteroides_k21_proteomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/bacteroides_k21_proteomic_comp...\n",
      "\r",
      "\u001b[K...got 40 x 40 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/bacteroides_k21_proteomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => bacteroides_k21_proteomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: bacteroides_k21_proteomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: bacteroides_k21_proteomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 40 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 52 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/gingivalis_k21_proteomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/gingivalis_k21_proteomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/gingivalis_k21_proteomic_comp...\n",
      "\r",
      "\u001b[K...got 52 x 52 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/gingivalis_k21_proteomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => gingivalis_k21_proteomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: gingivalis_k21_proteomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: gingivalis_k21_proteomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000007585.1_ASM758v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000010505.1_ASM1050v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000064305.2_ASM6430v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000162415.1_ASM16241v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000174775.1_ASM17477v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000212375.1_ASM21237v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000217375.2_CLC_glsol081_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000417845.2_ASM41784v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000428105.1_ASM42810v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000439065.1_ASM43906v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000467955.1_ASM46795v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000482365.1_ASM48236v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000503975.1_SJD2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510525.1_BU063sc679_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000510545.1_BU063sc13_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000583675.1_ASM58367v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599245.1_ASM59924v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000599325.1_ASM59932v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614325.1_ASM61432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000739415.1_ASM73941v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000765945.1_ASM76594v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000768975.1_ASM76897v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769015.1_ASM76901v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769035.1_ASM76903v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000769135.1_ASM76913v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_000945515.1_Velvet_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000949475.1_ASM94947v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_000993115.1_ASM99311v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001064135.1_ASM106413v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001261715.1_HGC4_v3_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001263815.1_ASM126381v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001274615.1_ASM127461v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001282625.1_Lascolabacillus_massiliensis_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001404575.1_13470_2_67_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001406635.1_13470_2_63_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438125.1_ASM143812v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001438205.1_ASM143820v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001512095.1_7521_8_38_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001602775.1_ASM160277v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001607025.1_ASM160702v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001637845.1_ASM163784v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001670785.1_RCAD0183_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001724935.1_ASM172493v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001768555.1_ASM176855v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001808555.1_ASM180855v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001873105.1_ASM187310v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001889045.1_ASM188904v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_001896575.1_ASM189657v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001897595.1_ASM189759v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_001898165.1_ASM189816v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900085525.1_12045_7_37_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/gingivalis/proteomic/GCA_900089535.1_A04_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157215.1_Strain_3-3_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/gingivalis/proteomic/GCA_900157325.1_3A1_protein.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 52 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 49 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n",
      "\r",
      "\u001b[Ksaving labels to: sourmash_compare/denticola_k21_proteomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving distance matrix to: sourmash_compare/denticola_k21_proteomic_comp\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/denticola_k21_proteomic_comp...\n",
      "\r",
      "\u001b[K...got 49 x 49 matrix.\n",
      "\r",
      "\u001b[Kloading labels from sourmash_compare/denticola_k21_proteomic_comp.labels.txt\n",
      "\r",
      "\u001b[Ksaving histogram of matrix values => denticola_k21_proteomic_comp.hist.pdf\n",
      "\r",
      "\u001b[Kwrote dendrogram to: denticola_k21_proteomic_comp.dendro.pdf\n",
      "\r",
      "\u001b[Kwrote numpy distance matrix to: denticola_k21_proteomic_comp.matrix.pdf\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000147075.1_ASM14707v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000216715.2_CLC_glsol140_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000217015.3_CLC_glsol119_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000217655.1_ASM21765v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000222305.1_ASM22230v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000236685.1_ASM23668v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000239475.1_ASM23947v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000242595.3_ASM24259v3_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000246415.2_CLC_glsol068_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000246815.1_ASM24681v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000260795.1_ASM26079v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000301975.1_ASM30197v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000342705.1_CA_glsol153_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000343205.1_CA_glsol121_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000382565.1_ASM38256v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000426705.1_ASM42670v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000465015.1_Treponema_pedis_T_M1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000468055.1_ASM46805v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000500065.1_AB2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501035.1_BorGarIPT95_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501115.1_BorGarIPT101_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000501555.1_BorAfzIPT109_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000502195.1_BorAfzIPT138_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000507245.1_ASM50724v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000568735.2_ASM56873v2_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000758165.1_Spiroch1.0_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000768055.1_ASM76805v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000775995.1_ASM77599v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000802305.1_ASM80230v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_000818865.1_ASM81886v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_000941035.1_ASM94103v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001281815.1_ASM128181v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001405175.1_13470_2_54_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001443305.1_ASM144330v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001567655.1_ASM156765v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001604335.1_ASM160433v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001712955.1_ASM171295v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001713255.1_ASM171325v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001746215.1_ASM174621v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829125.1_ASM182912v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829165.1_ASM182916v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829295.1_ASM182929v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829415.1_ASM182941v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829475.1_ASM182947v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001829505.1_ASM182950v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830585.1_ASM183058v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001830635.1_ASM183063v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001894145.1_ASM189414v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001896735.1_ASM189673v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_001940825.1_ASM194082v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_002009835.1_ASM200983v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069655.1_ASM206965v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_002069965.1_ASM206996v1_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900018035.1_7521_5_26_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/denticola/proteomic/GCA_900147745.1_138_1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900164975.1_16852_2_85_protein.sig\r",
      "\r",
      "\u001b[Kloading sigs/denticola/proteomic/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 49 signatures total.\n",
      "\r",
      "\u001b[Kdownsampling to scaled value of 2000\n",
      "\r",
      "\u001b[K\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for k in 21 31 51\n",
    "    do\n",
    "   for org in bacteroides gingivalis denticola\n",
    "       do\n",
    "        for mol in genomic\n",
    "            do \n",
    "sourmash compare -k ${k} -o sourmash_compare/${org}_k${k}_${mol}_comp sigs/${org}/${mol}/*.sig\n",
    "sourmash plot --labels --pdf sourmash_compare/${org}_k${k}_${mol}_comp\n",
    "sourmash compare -k ${k} --csv sourmash_compare/compare_CSVs/${org}_k${k}_${mol}_comp.csv sigs/${org}/${mol}/*.sig\n",
    "done\n",
    "done\n",
    "done\n",
    "\n",
    "for k in 7 9 12 21\n",
    "    do\n",
    "   for org in bacteroides gingivalis denticola\n",
    "       do\n",
    "        for mol in proteomic\n",
    "            do \n",
    "sourmash compare -k ${k} -o sourmash_compare/${org}_k${k}_${mol}_comp sigs/${org}/${mol}/*.sig\n",
    "sourmash plot --labels --pdf sourmash_compare/${org}_k${k}_${mol}_comp\n",
    "sourmash compare -k ${k} --csv sourmash_compare/compare_CSVs/${org}_k${k}_${mol}_comp.csv sigs/${org}/${mol}/*.sig\n",
    "done\n",
    "done\n",
    "done\n",
    "\n",
    "mv *.pdf /mnt/c/Users/user/Desktop/CheckPlotPDFs/\n",
    "mv sourmash_compare/compare_CSVs/*.csv /mnt/c/Users/user/Desktop/ROTATIONS/DIBlabRotationProject/CompCSVs/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/bacteroides_k7_genomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/bacteroides_k7_genomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/bacteroides_k9_genomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/bacteroides_k9_genomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading comparison matrix from sourmash_compare/bacteroides_k12_genomic_comp...\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/commands.py\", line 218, in plot\n",
      "    D = numpy.load(open(D_filename, 'rb'))\n",
      "FileNotFoundError: [Errno 2] No such file or directory: 'sourmash_compare/bacteroides_k12_genomic_comp'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\n",
      "                                       \n",
      "                     (right here) ------^\n",
      "\n",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for k in 7\n",
    "#9 12\n",
    "    do\n",
    "   for org in bacteroides \n",
    "#gingivalis denticola\n",
    "       do\n",
    "        for mol in proteomic\n",
    "            do \n",
    "sourmash compare -k ${k} -o sourmash_compare/${org}_k${k}_${mol}_comp sigs/${org}/${mol}/*.sig\n",
    "sourmash plot --labels --pdf sourmash_compare/bacteroides_k${k}_genomic_comp\n",
    "sourmash compare -k ${k} --csv sourmash_compare/compare_CSVs/${org}_k${k}_${mol}_comp.csv sigs/${org}/${mol}/*.sig\n",
    "done\n",
    "done\n",
    "done\n",
    "mv sourmash_compare/compare_CSVs/*.csv /mnt/c/Users/user/Desktop/ROTATIONS/DIBlabRotationProject/CompCSVs/Test"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mv: cannot stat '*.pdf': No such file or directory\r\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "for k in 7 9 12\n",
    "    do\n",
    "   for org in bacteroides gingivalis denticola\n",
    "       do\n",
    "        for mol in proteomic\n",
    "            do \n",
    "sourmash plot --labels --pdf sourmash_compare/bacteroides_k${k}_genomic_comp\n",
    "done\n",
    "done\n",
    "done\n",
    "mv sourmash_compare/compare_CSVs/*.csv /mnt/c/Users/user/Desktop/ROTATIONS/DIBlabRotationProject/CompCSVs/\n",
    "\n",
    "#%mv *.pdf /mnt/c/Users/user/Desktop/CheckPlotPDFs/\n",
    "#%mv sourmash_compare/compare_CSVs/*.csv /mnt/c/Users/user/Desktop/ROTATIONS/DIBlabRotationProject/CompCSVs/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\r\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\r\n",
      "\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000011065.1_ASM1106v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000157015.1_ASM15701v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000159855.2_Bact_sp_3_2_5_V2_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000163055.2_ASM16305v2_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000224595.1_Prevotella_sp_C561_V1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000273215.1_Bact_ovat_CL03T12C18_V1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000373705.1_ASM37370v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000403155.2_Bact_thet_dnLKV9_V1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000431275.1_MGS755_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000432495.1_MGS558_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000466425.1_ASM46642v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000495955.1_NUHP2_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000501415.1_BorBurgIPT48_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000542015.1_Stap_aure_FVRH6074_V1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000550025.1_Stap_aure_M1496_V1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000598325.1_ASM59832v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000611675.1_ASM61167v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614865.1_ASM61486v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000614895.1_ASM61489v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000626635.1_ASM62663v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000707425.1_CFSAN007430_01.0_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000759315.1_04_NF40_HMP9302v01_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000785025.1_ASM78502v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000814325.1_ASM81432v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000821925.1_34AVR_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000938735.2_ASM93873v2_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_000974365.1_ASM97436v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049535.1_3731_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001049555.1_7330_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001055755.1_ASM105575v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001304045.1_ASM130404v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001310955.1_ASM131095v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001373135.1_2e6A_assembly_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001398875.1_Mother10-2_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405895.1_14207_7_22_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001405935.1_13414_6_56_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001413955.1_ASM141395v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001443605.1_ASM144360v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001548195.1_ASM154819v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001578565.1_ASM157856v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001595155.1_CFSAN033860_01.0_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603535.1_ASM160353v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001603695.1_ASM160369v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001614375.1_ASM161437v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001659685.2_ASM165968v2_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001693695.1_ASM169369v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001761325.1_ASM176132v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001816245.1_ASM181624v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_001915365.1_ASM191536v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_002036985.1_ASM203698v1_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900032635.1_14555_6_27_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900104585.1_PRJEB16348_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900106755.1_IMG-taxon_2693429916_annotated_assembly_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900109385.1_IMG-taxon_2593339226_annotated_assembly_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900110645.1_IMG-taxon_2693429910_annotated_assembly_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900111425.1_IMG-taxon_2693429902_annotated_assembly_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900112795.1_IMG-taxon_2693429893_annotated_assembly_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900116355.1_IMG-taxon_2622736518_annotated_assembly_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r",
      "\r",
      "\u001b[KError in parsing signature; quitting.\r\n",
      "\r",
      "\u001b[KException: parse error: premature EOF\r\n",
      "                                       \r\n",
      "                     (right here) ------^\r\n",
      "\r\n",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900128475.1_PRJEB18047_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900142325.1_IMG-taxon_2698536696_annotated_assembly_protein.sig\r\n",
      "\r",
      "\u001b[Kloading sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r",
      "\r",
      "\u001b[K\r\n",
      "warning: no signatures loaded at given ksize/molecule type from sigs/bacteroides/proteomic/GCA_900167375.1_IMG-taxon_2588254259_annotated_assembly_protein.sig\r\n",
      "\r",
      "\u001b[K                                                                               \r",
      "\r",
      "\u001b[Kloaded 0 signatures total.\r\n",
      "\r",
      "\u001b[Kcannot mix scaled signatures with bounded signatures\r\n"
     ]
    }
   ],
   "source": [
    "\n",
    "# 7-mer\n",
    "#Bacteroides, proteomic\n",
    "!sourmash compare -k 7 -o sourmash_compare/bacteroides_k7_proteomic_comp sigs/bacteroides/proteomic/*.sig\n",
    "#!sourmash plot --labels --pdf sourmash_compare/bacteroides_k7_proteomic_comp\n",
    "#%mv *.pdf /mnt/c/Users/user/Desktop/ROTATIONS/DIBlabRotationProject/CompCSVs/Test\n",
    "\n",
    "#!sourmash compare -k 7 --csv sourmash_compare/compare_CSVs/bacteroides_k7_proteomic_comp.csv sigs/bacteroides/proteomic/*.sig\n",
    "#%mv sourmash_compare/compare_CSVs/*.csv /mnt/c/Users/user/Desktop/ROTATIONS/DIBlabRotationProject/CompCSVs/Test\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bacteroides_k21_genomic_comp\r\n",
      "bacteroides_k21_genomic_comp.labels.txt\r\n",
      "bacteroides_k21_proteomic_comp\r\n",
      "bacteroides_k21_proteomic_comp.labels.txt\r\n",
      "bacteroides_k31_genomic_comp\r\n",
      "bacteroides_k31_genomic_comp.csv\r\n",
      "bacteroides_k31_genomic_comp.labels.txt\r\n",
      "bacteroides_k31_proteomic_comp\r\n",
      "bacteroides_k31_proteomic_comp.labels.txt\r\n",
      "bacteroides_k51_genomic_comp\r\n",
      "bacteroides_k51_genomic_comp.labels.txt\r\n",
      "bacteroides_k51_proteomic_comp\r\n",
      "bacteroides_k51_proteomic_comp.labels.txt\r\n",
      "\u001b[0m\u001b[34;42mcompare_CSVs\u001b[0m/\r\n",
      "denticola_k21_genomic_comp\r\n",
      "denticola_k21_genomic_comp.labels.txt\r\n",
      "denticola_k21_proteomic_comp\r\n",
      "denticola_k21_proteomic_comp.labels.txt\r\n",
      "denticola_k31_genomic_comp\r\n",
      "denticola_k31_genomic_comp.labels.txt\r\n",
      "denticola_k31_proteomic_comp\r\n",
      "denticola_k31_proteomic_comp.labels.txt\r\n",
      "denticola_k51_genomic_comp\r\n",
      "denticola_k51_genomic_comp.labels.txt\r\n",
      "denticola_k51_proteomic_comp\r\n",
      "denticola_k51_proteomic_comp.labels.txt\r\n",
      "gingivalis_k21_genomic_comp\r\n",
      "gingivalis_k21_genomic_comp.labels.txt\r\n",
      "gingivalis_k21_proteomic_comp\r\n",
      "gingivalis_k21_proteomic_comp.labels.txt\r\n",
      "gingivalis_k31_genomic_comp\r\n",
      "gingivalis_k31_genomic_comp.labels.txt\r\n",
      "gingivalis_k31_proteomic_comp\r\n",
      "gingivalis_k31_proteomic_comp.labels.txt\r\n",
      "gingivalis_k51_genomic_comp\r\n",
      "gingivalis_k51_genomic_comp.labels.txt\r\n",
      "gingivalis_k51_proteomic_comp\r\n",
      "gingivalis_k51_proteomic_comp.labels.txt\r\n"
     ]
    }
   ],
   "source": [
    "%ls sourmash_compare/bac\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
