{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "    #activate environment\n",
    "#conda activate rotation\n",
    "    #install pandas, os, re, argparse, ipfsapi, requests, shutil; e.g.\n",
    "#conda install pandas\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/hehouts/2019-fall-rotation-dib/sm_rotation_proj\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "mkdir github_source_files\n",
    "cd github_souce_files\n",
    "\n",
    "git clone https://github.com/bluegenes/2018-test_datasets.git\n",
    "\n",
    "cp 2018-test_datasets/* .\n",
    "rm -r 2018-test_datasets "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "#This should be better than the below block, but is not tested.\n",
    "#Hypothetically, new organisms can be added willy nilly\n",
    "\n",
    "for org in bacteroides denticola gingivalis\n",
    "do\n",
    "    for mol in genomic protein  rna \n",
    "    do\n",
    "    \n",
    "        #Downloads seqs\n",
    "        python github_souce_files/download_genbank_datasets.py --genbank --protein --rna --subfolder github_souce_files/$org.csv\n",
    "\n",
    "        #This organizes folders\n",
    "        mv github_souce_files/$org.csv github_souce_files/$org/\n",
    "        mkdir -p $org\n",
    "        mv github_souce_files/$org/* $org\n",
    "        rmdir github_souce_files/$org/\n",
    "\n",
    "        cd $org\n",
    "        mkdir $mol/seqs\n",
    "        mv $mol/*.gz $mol/seqs\n",
    "        cd .. \n",
    "    done\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "#REDUNDANT\n",
    "\n",
    "python github_souce_files/download_genbank_datasets.py --genbank --protein --rna --subfolder github_souce_files/bacteroides.csv\n",
    "\n",
    "python github_souce_files/download_genbank_datasets.py --genbank --protein --rna --subfolder github_souce_files/denticola.csv\n",
    "\n",
    "python github_souce_files/download_genbank_datasets.py --genbank --protein --rna --subfolder github_souce_files/gingivalis.csv\n",
    " \n",
    " \n",
    "#This organizes folders\n",
    "mv github_souce_files/bacteroides.csv github_souce_files/bacteroides/\n",
    "mkdir -p bacteroides\n",
    "mv github_souce_files/bacteroides/* bacteroides\n",
    "rmdir github_souce_files/bacteroides/\n",
    "\n",
    "mv github_souce_files/denticola.csv github_souce_files/denticola\n",
    "mkdir denticola\n",
    "mv github_souce_files/denticola/* denticola\n",
    "rmdir github_souce_files/denticola/\n",
    "\n",
    "mv github_souce_files/gingivalis.csv github_souce_files/gingivalis\n",
    "mkdir gingivalis\n",
    "mv github_souce_files/gingivalis/* gingivalis\n",
    "rmdir github_souce_files/gingivalis/\n",
    "\n",
    "\n",
    "\n",
    "#This is to organize seqs\n",
    "for org in bacteroides denticola gingivalis\n",
    "    do\n",
    "    for mol in genomic protein  rna \n",
    "    do\n",
    "        cd $org\n",
    "        mkdir $mol/seqs\n",
    "        mv $mol/*a.gz $mol/seqs\n",
    "        cd .. \n",
    "    done\n",
    "done\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000147075.1_ASM14707v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00333 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000147075.1_ASM14707v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000216715.2_CLC_glsol140_genomic.fna.gz 315 sequences\r",
      "\u001b[Kcalculated 3 signatures for 316 sequences in genomic/seqs/GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00701 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000216715.2_CLC_glsol140_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000217015.3_CLC_glsol119_genomic.fna.gz 337 sequences\r",
      "\u001b[Kcalculated 3 signatures for 338 sequences in genomic/seqs/GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00665 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000217015.3_CLC_glsol119_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000217655.1_ASM21765v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00158 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000217655.1_ASM21765v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000222305.1_ASM22230v1_genomic.fna.gz 16 sequences\r",
      "\u001b[Kcalculated 3 signatures for 17 sequences in genomic/seqs/GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00180 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000222305.1_ASM22230v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000236685.1_ASM23668v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00474 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000236685.1_ASM23668v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000239475.1_ASM23947v1_genomic.fna.gz 2 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3 sequences in genomic/seqs/GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00141 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000239475.1_ASM23947v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000242595.3_ASM24259v3_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00453 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000242595.3_ASM24259v3_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000246415.2_CLC_glsol068_genomic.fna.gz 666 sequences\r",
      "\u001b[Kcalculated 3 signatures for 667 sequences in genomic/seqs/GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00669 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000246415.2_CLC_glsol068_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000246815.1_ASM24681v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00159 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000246815.1_ASM24681v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000260795.1_ASM26079v1_genomic.fna.gz 146 sequences\r",
      "\u001b[Kcalculated 3 signatures for 147 sequences in genomic/seqs/GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00414 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000260795.1_ASM26079v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000301975.1_ASM30197v1_genomic.fna.gz 115 sequences\r",
      "\u001b[Kcalculated 3 signatures for 116 sequences in genomic/seqs/GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00571 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000301975.1_ASM30197v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz 2 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3 sequences in genomic/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00387 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2 sequences in genomic/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00385 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2 sequences in genomic/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00380 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00428 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00390 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000340645.1_Trep_dent_H-22_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00395 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2 sequences in genomic/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00366 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000342705.1_CA_glsol153_genomic.fna.gz 139 sequences\r",
      "\u001b[Kcalculated 3 signatures for 140 sequences in genomic/seqs/GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00598 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000342705.1_CA_glsol153_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000343205.1_CA_glsol121_genomic.fna.gz 161 sequences\r",
      "\u001b[Kcalculated 3 signatures for 162 sequences in genomic/seqs/GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00595 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000343205.1_CA_glsol121_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000382565.1_ASM38256v1_genomic.fna.gz 6 sequences\r",
      "\u001b[Kcalculated 3 signatures for 7 sequences in genomic/seqs/GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00125 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000382565.1_ASM38256v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz 7 sequences\r",
      "\u001b[Kcalculated 3 signatures for 8 sequences in genomic/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00397 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz 5 sequences\r",
      "\u001b[Kcalculated 3 signatures for 6 sequences in genomic/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00660 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz 2 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3 sequences in genomic/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00371 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000426705.1_ASM42670v1_genomic.fna.gz 28 sequences\r",
      "\u001b[Kcalculated 3 signatures for 29 sequences in genomic/seqs/GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00568 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000426705.1_ASM42670v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz 557 sequences\r",
      "\u001b[Kcalculated 3 signatures for 558 sequences in genomic/seqs/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00389 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000465015.1_Treponema_pedis_T_M1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000468055.1_ASM46805v1_genomic.fna.gz 20 sequences\r",
      "\u001b[Kcalculated 3 signatures for 21 sequences in genomic/seqs/GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00338 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000468055.1_ASM46805v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000500065.1_AB2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000500065.1_AB2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000500065.1_AB2_genomic.fna.gz 167 sequences\r",
      "\u001b[Kcalculated 3 signatures for 168 sequences in genomic/seqs/GCA_000500065.1_AB2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00489 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000500065.1_AB2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000501035.1_BorGarIPT95_genomic.fna.gz 193 sequences\r",
      "\u001b[Kcalculated 3 signatures for 194 sequences in genomic/seqs/GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00033 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000501035.1_BorGarIPT95_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000501115.1_BorGarIPT101_genomic.fna.gz 712 sequences\r",
      "\u001b[Kcalculated 3 signatures for 713 sequences in genomic/seqs/GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00181 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000501115.1_BorGarIPT101_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz 536 sequences\r",
      "\u001b[Kcalculated 3 signatures for 537 sequences in genomic/seqs/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00174 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000501555.1_BorAfzIPT109_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz 192 sequences\r",
      "\u001b[Kcalculated 3 signatures for 193 sequences in genomic/seqs/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00186 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000502195.1_BorAfzIPT138_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000507245.1_ASM50724v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.01123 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000507245.1_ASM50724v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz 9 sequences\r",
      "\u001b[Kcalculated 3 signatures for 10 sequences in genomic/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00393 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000568735.2_ASM56873v2_genomic.fna.gz 73 sequences\r",
      "\u001b[Kcalculated 3 signatures for 74 sequences in genomic/seqs/GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00151 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000568735.2_ASM56873v2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000758165.1_Spiroch1.0_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 3 signatures for 72 sequences in genomic/seqs/GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00443 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000758165.1_Spiroch1.0_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000768055.1_ASM76805v1_genomic.fna.gz 274 sequences\r",
      "\u001b[Kcalculated 3 signatures for 275 sequences in genomic/seqs/GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00527 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000768055.1_ASM76805v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000775995.1_ASM77599v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00365 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000775995.1_ASM77599v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000802305.1_ASM80230v1_genomic.fna.gz 54 sequences\r",
      "\u001b[Kcalculated 3 signatures for 55 sequences in genomic/seqs/GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00562 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000802305.1_ASM80230v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000818865.1_ASM81886v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000818865.1_ASM81886v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000818865.1_ASM81886v1_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2 sequences in genomic/seqs/GCA_000818865.1_ASM81886v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00515 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000818865.1_ASM81886v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000941035.1_ASM94103v1_genomic.fna.gz 4 sequences\r",
      "\u001b[Kcalculated 3 signatures for 5 sequences in genomic/seqs/GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00958 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000941035.1_ASM94103v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001281815.1_ASM128181v1_genomic.fna.gz 285 sequences\r",
      "\u001b[Kcalculated 3 signatures for 286 sequences in genomic/seqs/GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00964 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001281815.1_ASM128181v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001405175.1_13470_2_54_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 3 signatures for 72 sequences in genomic/seqs/GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00656 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001405175.1_13470_2_54_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001443305.1_ASM144330v1_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2 sequences in genomic/seqs/GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00624 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001443305.1_ASM144330v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001567655.1_ASM156765v1_genomic.fna.gz 254 sequences\r",
      "\u001b[Kcalculated 3 signatures for 255 sequences in genomic/seqs/GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00851 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001567655.1_ASM156765v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001604335.1_ASM160433v1_genomic.fna.gz 591 sequences\r",
      "\u001b[Kcalculated 3 signatures for 592 sequences in genomic/seqs/GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00260 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001604335.1_ASM160433v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001712955.1_ASM171295v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00160 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001712955.1_ASM171295v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001713255.1_ASM171325v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00160 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001713255.1_ASM171325v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001746215.1_ASM174621v1_genomic.fna.gz 177 sequences\r",
      "\u001b[Kcalculated 3 signatures for 178 sequences in genomic/seqs/GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00416 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001746215.1_ASM174621v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829125.1_ASM182912v1_genomic.fna.gz 272 sequences\r",
      "\u001b[Kcalculated 3 signatures for 273 sequences in genomic/seqs/GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00591 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829125.1_ASM182912v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829165.1_ASM182916v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829165.1_ASM182916v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829165.1_ASM182916v1_genomic.fna.gz 372 sequences\r",
      "\u001b[Kcalculated 3 signatures for 373 sequences in genomic/seqs/GCA_001829165.1_ASM182916v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00661 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829165.1_ASM182916v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829295.1_ASM182929v1_genomic.fna.gz 137 sequences\r",
      "\u001b[Kcalculated 3 signatures for 138 sequences in genomic/seqs/GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00467 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829295.1_ASM182929v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829415.1_ASM182941v1_genomic.fna.gz 76 sequences\r",
      "\u001b[Kcalculated 3 signatures for 77 sequences in genomic/seqs/GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00453 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829415.1_ASM182941v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829475.1_ASM182947v1_genomic.fna.gz 208 sequences\r",
      "\u001b[Kcalculated 3 signatures for 209 sequences in genomic/seqs/GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00435 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829475.1_ASM182947v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829505.1_ASM182950v1_genomic.fna.gz 244 sequences\r",
      "\u001b[Kcalculated 3 signatures for 245 sequences in genomic/seqs/GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00515 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829505.1_ASM182950v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001830585.1_ASM183058v1_genomic.fna.gz 266 sequences\r",
      "\u001b[Kcalculated 3 signatures for 267 sequences in genomic/seqs/GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00566 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001830585.1_ASM183058v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001830635.1_ASM183063v1_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 3 signatures for 67 sequences in genomic/seqs/GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00466 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001830635.1_ASM183063v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001894145.1_ASM189414v1_genomic.fna.gz 439 sequences\r",
      "\u001b[Kcalculated 3 signatures for 440 sequences in genomic/seqs/GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00806 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001894145.1_ASM189414v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001896735.1_ASM189673v1_genomic.fna.gz 159 sequences\r",
      "\u001b[Kcalculated 3 signatures for 160 sequences in genomic/seqs/GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00448 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001896735.1_ASM189673v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001940825.1_ASM194082v1_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 3 signatures for 72 sequences in genomic/seqs/GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00335 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001940825.1_ASM194082v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_002009835.1_ASM200983v1_genomic.fna.gz 336 sequences\r",
      "\u001b[Kcalculated 3 signatures for 337 sequences in genomic/seqs/GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00541 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_002009835.1_ASM200983v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_002069655.1_ASM206965v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_002069655.1_ASM206965v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_002069655.1_ASM206965v1_genomic.fna.gz 121 sequences\r",
      "\u001b[Kcalculated 3 signatures for 122 sequences in genomic/seqs/GCA_002069655.1_ASM206965v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00338 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_002069655.1_ASM206965v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_002069965.1_ASM206996v1_genomic.fna.gz 159 sequences\r",
      "\u001b[Kcalculated 3 signatures for 160 sequences in genomic/seqs/GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00410 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_002069965.1_ASM206996v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900018035.1_7521_5_26_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 3 signatures for 72 sequences in genomic/seqs/GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00393 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900018035.1_7521_5_26_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900147745.1_138_1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900147745.1_138_1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900147745.1_138_1_genomic.fna.gz 281 sequences\r",
      "\u001b[Kcalculated 3 signatures for 282 sequences in genomic/seqs/GCA_900147745.1_138_1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00857 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900147745.1_138_1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 3 signatures for 65 sequences in genomic/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00452 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900164975.1_16852_2_85_genomic.fna.gz 30 sequences\r",
      "\u001b[Kcalculated 3 signatures for 31 sequences in genomic/seqs/GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00560 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900164975.1_16852_2_85_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz 31 sequences\r",
      "\u001b[Kcalculated 3 signatures for 32 sequences in genomic/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00312 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "bash: line 8: cd: denticola: No such file or directory\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000147075.1_ASM14707v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000147075.1_ASM14707v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000147075.1_ASM14707v1_rna_from_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 3 signatures for 52 sequences in rna/seqs/GCA_000147075.1_ASM14707v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000147075.1_ASM14707v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000216715.2_CLC_glsol140_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000216715.2_CLC_glsol140_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000216715.2_CLC_glsol140_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000217015.3_CLC_glsol119_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000217015.3_CLC_glsol119_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000217015.3_CLC_glsol119_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000217655.1_ASM21765v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000217655.1_ASM21765v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000217655.1_ASM21765v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_000217655.1_ASM21765v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000217655.1_ASM21765v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000222305.1_ASM22230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000222305.1_ASM22230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000222305.1_ASM22230v1_rna_from_genomic.fna.gz 37 sequences\r",
      "\u001b[Kcalculated 3 signatures for 38 sequences in rna/seqs/GCA_000222305.1_ASM22230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000222305.1_ASM22230v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000236685.1_ASM23668v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000236685.1_ASM23668v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000236685.1_ASM23668v1_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 3 signatures for 67 sequences in rna/seqs/GCA_000236685.1_ASM23668v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000236685.1_ASM23668v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000239475.1_ASM23947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000239475.1_ASM23947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000239475.1_ASM23947v1_rna_from_genomic.fna.gz 35 sequences\r",
      "\u001b[Kcalculated 3 signatures for 36 sequences in rna/seqs/GCA_000239475.1_ASM23947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000239475.1_ASM23947v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000242595.3_ASM24259v3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000242595.3_ASM24259v3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000242595.3_ASM24259v3_rna_from_genomic.fna.gz 55 sequences\r",
      "\u001b[Kcalculated 3 signatures for 56 sequences in rna/seqs/GCA_000242595.3_ASM24259v3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000242595.3_ASM24259v3_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000246415.2_CLC_glsol068_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000246415.2_CLC_glsol068_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000246415.2_CLC_glsol068_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000246815.1_ASM24681v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000260795.1_ASM26079v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000260795.1_ASM26079v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000260795.1_ASM26079v1_rna_from_genomic.fna.gz 43 sequences\r",
      "\u001b[Kcalculated 3 signatures for 44 sequences in rna/seqs/GCA_000260795.1_ASM26079v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000260795.1_ASM26079v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000301975.1_ASM30197v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000301975.1_ASM30197v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000301975.1_ASM30197v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.fna.gz 47 sequences\r",
      "\u001b[Kcalculated 3 signatures for 48 sequences in rna/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000342705.1_CA_glsol153_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000342705.1_CA_glsol153_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000342705.1_CA_glsol153_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000343205.1_CA_glsol121_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000343205.1_CA_glsol121_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000343205.1_CA_glsol121_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000382565.1_ASM38256v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000382565.1_ASM38256v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000382565.1_ASM38256v1_rna_from_genomic.fna.gz 34 sequences\r",
      "\u001b[Kcalculated 3 signatures for 35 sequences in rna/seqs/GCA_000382565.1_ASM38256v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000382565.1_ASM38256v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000426705.1_ASM42670v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000426705.1_ASM42670v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000426705.1_ASM42670v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000465015.1_Treponema_pedis_T_M1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000465015.1_Treponema_pedis_T_M1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000465015.1_Treponema_pedis_T_M1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000468055.1_ASM46805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000468055.1_ASM46805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000468055.1_ASM46805v1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000468055.1_ASM46805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000468055.1_ASM46805v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000500065.1_AB2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000500065.1_AB2_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000500065.1_AB2_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000501035.1_BorGarIPT95_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000501035.1_BorGarIPT95_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000501035.1_BorGarIPT95_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000501115.1_BorGarIPT101_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000501115.1_BorGarIPT101_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000501115.1_BorGarIPT101_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000501555.1_BorAfzIPT109_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000501555.1_BorAfzIPT109_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000501555.1_BorAfzIPT109_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000502195.1_BorAfzIPT138_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000502195.1_BorAfzIPT138_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000502195.1_BorAfzIPT138_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000507245.1_ASM50724v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000507245.1_ASM50724v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000507245.1_ASM50724v1_rna_from_genomic.fna.gz 49 sequences\r",
      "\u001b[Kcalculated 3 signatures for 50 sequences in rna/seqs/GCA_000507245.1_ASM50724v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000507245.1_ASM50724v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.fna.gz 75 sequences\r",
      "\u001b[Kcalculated 3 signatures for 76 sequences in rna/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000568735.2_ASM56873v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000568735.2_ASM56873v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000568735.2_ASM56873v2_rna_from_genomic.fna.gz 33 sequences\r",
      "\u001b[Kcalculated 3 signatures for 34 sequences in rna/seqs/GCA_000568735.2_ASM56873v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000568735.2_ASM56873v2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000758165.1_Spiroch1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000758165.1_Spiroch1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000758165.1_Spiroch1.0_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_000758165.1_Spiroch1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000758165.1_Spiroch1.0_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000768055.1_ASM76805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000768055.1_ASM76805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000768055.1_ASM76805v1_rna_from_genomic.fna.gz 23 sequences\r",
      "\u001b[Kcalculated 3 signatures for 24 sequences in rna/seqs/GCA_000768055.1_ASM76805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000768055.1_ASM76805v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000775995.1_ASM77599v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000775995.1_ASM77599v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000775995.1_ASM77599v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_000775995.1_ASM77599v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000775995.1_ASM77599v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000802305.1_ASM80230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000802305.1_ASM80230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000802305.1_ASM80230v1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000802305.1_ASM80230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000802305.1_ASM80230v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000818865.1_ASM81886v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000818865.1_ASM81886v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000818865.1_ASM81886v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000941035.1_ASM94103v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000941035.1_ASM94103v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000941035.1_ASM94103v1_rna_from_genomic.fna.gz 41 sequences\r",
      "\u001b[Kcalculated 3 signatures for 42 sequences in rna/seqs/GCA_000941035.1_ASM94103v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000941035.1_ASM94103v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001281815.1_ASM128181v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001281815.1_ASM128181v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001281815.1_ASM128181v1_rna_from_genomic.fna.gz 109 sequences\r",
      "\u001b[Kcalculated 3 signatures for 110 sequences in rna/seqs/GCA_001281815.1_ASM128181v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001281815.1_ASM128181v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001405175.1_13470_2_54_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001405175.1_13470_2_54_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001405175.1_13470_2_54_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 3 signatures for 67 sequences in rna/seqs/GCA_001405175.1_13470_2_54_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001405175.1_13470_2_54_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001443305.1_ASM144330v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001443305.1_ASM144330v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001443305.1_ASM144330v1_rna_from_genomic.fna.gz 40 sequences\r",
      "\u001b[Kcalculated 3 signatures for 41 sequences in rna/seqs/GCA_001443305.1_ASM144330v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001443305.1_ASM144330v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001567655.1_ASM156765v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001567655.1_ASM156765v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001567655.1_ASM156765v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001604335.1_ASM160433v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001604335.1_ASM160433v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001604335.1_ASM160433v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001712955.1_ASM171295v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001712955.1_ASM171295v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001712955.1_ASM171295v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_001712955.1_ASM171295v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001712955.1_ASM171295v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001713255.1_ASM171325v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001713255.1_ASM171325v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001713255.1_ASM171325v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_001713255.1_ASM171325v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001713255.1_ASM171325v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001746215.1_ASM174621v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001746215.1_ASM174621v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001746215.1_ASM174621v1_rna_from_genomic.fna.gz 37 sequences\r",
      "\u001b[Kcalculated 3 signatures for 38 sequences in rna/seqs/GCA_001746215.1_ASM174621v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001746215.1_ASM174621v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829125.1_ASM182912v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829125.1_ASM182912v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829125.1_ASM182912v1_rna_from_genomic.fna.gz 38 sequences\r",
      "\u001b[Kcalculated 3 signatures for 39 sequences in rna/seqs/GCA_001829125.1_ASM182912v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829125.1_ASM182912v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829165.1_ASM182916v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829165.1_ASM182916v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829165.1_ASM182916v1_rna_from_genomic.fna.gz 99 sequences\r",
      "\u001b[Kcalculated 3 signatures for 100 sequences in rna/seqs/GCA_001829165.1_ASM182916v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829165.1_ASM182916v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829295.1_ASM182929v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829295.1_ASM182929v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829295.1_ASM182929v1_rna_from_genomic.fna.gz 45 sequences\r",
      "\u001b[Kcalculated 3 signatures for 46 sequences in rna/seqs/GCA_001829295.1_ASM182929v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829295.1_ASM182929v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829415.1_ASM182941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829415.1_ASM182941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829415.1_ASM182941v1_rna_from_genomic.fna.gz 45 sequences\r",
      "\u001b[Kcalculated 3 signatures for 46 sequences in rna/seqs/GCA_001829415.1_ASM182941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829415.1_ASM182941v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829475.1_ASM182947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829475.1_ASM182947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829475.1_ASM182947v1_rna_from_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 3 signatures for 33 sequences in rna/seqs/GCA_001829475.1_ASM182947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829475.1_ASM182947v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829505.1_ASM182950v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829505.1_ASM182950v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829505.1_ASM182950v1_rna_from_genomic.fna.gz 36 sequences\r",
      "\u001b[Kcalculated 3 signatures for 37 sequences in rna/seqs/GCA_001829505.1_ASM182950v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829505.1_ASM182950v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001830585.1_ASM183058v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001830585.1_ASM183058v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001830585.1_ASM183058v1_rna_from_genomic.fna.gz 28 sequences\r",
      "\u001b[Kcalculated 3 signatures for 29 sequences in rna/seqs/GCA_001830585.1_ASM183058v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001830585.1_ASM183058v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001830635.1_ASM183063v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001830635.1_ASM183063v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001830635.1_ASM183063v1_rna_from_genomic.fna.gz 47 sequences\r",
      "\u001b[Kcalculated 3 signatures for 48 sequences in rna/seqs/GCA_001830635.1_ASM183063v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001830635.1_ASM183063v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001894145.1_ASM189414v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001894145.1_ASM189414v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001894145.1_ASM189414v1_rna_from_genomic.fna.gz 105 sequences\r",
      "\u001b[Kcalculated 3 signatures for 106 sequences in rna/seqs/GCA_001894145.1_ASM189414v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001894145.1_ASM189414v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001896735.1_ASM189673v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001896735.1_ASM189673v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001896735.1_ASM189673v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001940825.1_ASM194082v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001940825.1_ASM194082v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001940825.1_ASM194082v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_002009835.1_ASM200983v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_002009835.1_ASM200983v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_002009835.1_ASM200983v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_002069655.1_ASM206965v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_002069655.1_ASM206965v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_002069655.1_ASM206965v1_rna_from_genomic.fna.gz 28 sequences\r",
      "\u001b[Kcalculated 3 signatures for 29 sequences in rna/seqs/GCA_002069655.1_ASM206965v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_002069655.1_ASM206965v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_002069965.1_ASM206996v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_002069965.1_ASM206996v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_002069965.1_ASM206996v1_rna_from_genomic.fna.gz 45 sequences\r",
      "\u001b[Kcalculated 3 signatures for 46 sequences in rna/seqs/GCA_002069965.1_ASM206996v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_002069965.1_ASM206996v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900018035.1_7521_5_26_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900018035.1_7521_5_26_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900018035.1_7521_5_26_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 3 signatures for 67 sequences in rna/seqs/GCA_900018035.1_7521_5_26_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900018035.1_7521_5_26_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900147745.1_138_1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900147745.1_138_1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_900147745.1_138_1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.fna.gz 57 sequences\r",
      "\u001b[Kcalculated 3 signatures for 58 sequences in rna/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900164975.1_16852_2_85_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900164975.1_16852_2_85_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900164975.1_16852_2_85_rna_from_genomic.fna.gz 91 sequences\r",
      "\u001b[Kcalculated 3 signatures for 92 sequences in rna/seqs/GCA_900164975.1_16852_2_85_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900164975.1_16852_2_85_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.fna.gz 55 sequences\r",
      "\u001b[Kcalculated 3 signatures for 56 sequences in rna/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "bash: line 21: cd: denticola: No such file or directory\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000147075.1_ASM14707v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000147075.1_ASM14707v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000147075.1_ASM14707v1_protein.faa.gz 2202 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2203 sequences in protein/seqs/GCA_000147075.1_ASM14707v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00044 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000147075.1_ASM14707v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000216715.2_CLC_glsol140_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000216715.2_CLC_glsol140_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000216715.2_CLC_glsol140_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217015.3_CLC_glsol119_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217015.3_CLC_glsol119_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217015.3_CLC_glsol119_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217655.1_ASM21765v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217655.1_ASM21765v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000217655.1_ASM21765v1_protein.faa.gz 1009 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1010 sequences in protein/seqs/GCA_000217655.1_ASM21765v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000217655.1_ASM21765v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000222305.1_ASM22230v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000222305.1_ASM22230v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000222305.1_ASM22230v1_protein.faa.gz 1462 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1463 sequences in protein/seqs/GCA_000222305.1_ASM22230v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00029 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000222305.1_ASM22230v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000236685.1_ASM23668v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000236685.1_ASM23668v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000236685.1_ASM23668v1_protein.faa.gz 3158 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3159 sequences in protein/seqs/GCA_000236685.1_ASM23668v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00170 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000236685.1_ASM23668v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000239475.1_ASM23947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000239475.1_ASM23947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000239475.1_ASM23947v1_protein.faa.gz 955 sequences\r",
      "\u001b[Kcalculated 3 signatures for 956 sequences in protein/seqs/GCA_000239475.1_ASM23947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00028 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000239475.1_ASM23947v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000242595.3_ASM24259v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000242595.3_ASM24259v3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000242595.3_ASM24259v3_protein.faa.gz 2781 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2782 sequences in protein/seqs/GCA_000242595.3_ASM24259v3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00062 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000242595.3_ASM24259v3_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000246415.2_CLC_glsol068_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000246415.2_CLC_glsol068_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000246415.2_CLC_glsol068_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000246815.1_ASM24681v1_protein.faa.gz 1058 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1059 sequences in protein/seqs/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000246815.1_ASM24681v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000260795.1_ASM26079v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000260795.1_ASM26079v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000260795.1_ASM26079v1_protein.faa.gz 2569 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2570 sequences in protein/seqs/GCA_000260795.1_ASM26079v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00059 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000260795.1_ASM26079v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000301975.1_ASM30197v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000301975.1_ASM30197v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000301975.1_ASM30197v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz 2530 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2531 sequences in protein/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00045 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz 2646 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2647 sequences in protein/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00046 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz 2568 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2569 sequences in protein/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00045 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz 2704 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2705 sequences in protein/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00086 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz 2519 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2520 sequences in protein/seqs/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz 2631 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2632 sequences in protein/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00047 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz 2550 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2551 sequences in protein/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00047 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000342705.1_CA_glsol153_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000342705.1_CA_glsol153_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000342705.1_CA_glsol153_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000343205.1_CA_glsol121_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000343205.1_CA_glsol121_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000343205.1_CA_glsol121_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000382565.1_ASM38256v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000382565.1_ASM38256v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000382565.1_ASM38256v1_protein.faa.gz 831 sequences\r",
      "\u001b[Kcalculated 3 signatures for 832 sequences in protein/seqs/GCA_000382565.1_ASM38256v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00029 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000382565.1_ASM38256v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz 2609 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2610 sequences in protein/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00177 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz 2792 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2793 sequences in protein/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00048 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz 2613 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2614 sequences in protein/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00046 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000426705.1_ASM42670v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000426705.1_ASM42670v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000426705.1_ASM42670v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000465015.1_Treponema_pedis_T_M1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000465015.1_Treponema_pedis_T_M1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000465015.1_Treponema_pedis_T_M1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000468055.1_ASM46805v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000468055.1_ASM46805v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000468055.1_ASM46805v1_protein.faa.gz 2174 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2175 sequences in protein/seqs/GCA_000468055.1_ASM46805v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00046 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000468055.1_ASM46805v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000500065.1_AB2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000500065.1_AB2_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000500065.1_AB2_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000501035.1_BorGarIPT95_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000501035.1_BorGarIPT95_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000501035.1_BorGarIPT95_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000501115.1_BorGarIPT101_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000501115.1_BorGarIPT101_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000501115.1_BorGarIPT101_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000501555.1_BorAfzIPT109_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000501555.1_BorAfzIPT109_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000501555.1_BorAfzIPT109_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000502195.1_BorAfzIPT138_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000502195.1_BorAfzIPT138_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000502195.1_BorAfzIPT138_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000507245.1_ASM50724v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000507245.1_ASM50724v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000507245.1_ASM50724v1_protein.faa.gz 3396 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3397 sequences in protein/seqs/GCA_000507245.1_ASM50724v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00067 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000507245.1_ASM50724v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz 2682 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2683 sequences in protein/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00045 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000568735.2_ASM56873v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000568735.2_ASM56873v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000568735.2_ASM56873v2_protein.faa.gz 1084 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1085 sequences in protein/seqs/GCA_000568735.2_ASM56873v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00028 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000568735.2_ASM56873v2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000758165.1_Spiroch1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000758165.1_Spiroch1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000758165.1_Spiroch1.0_protein.faa.gz 2302 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2303 sequences in protein/seqs/GCA_000758165.1_Spiroch1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000758165.1_Spiroch1.0_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768055.1_ASM76805v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768055.1_ASM76805v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768055.1_ASM76805v1_protein.faa.gz 2731 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2732 sequences in protein/seqs/GCA_000768055.1_ASM76805v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00065 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000768055.1_ASM76805v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000775995.1_ASM77599v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000775995.1_ASM77599v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000775995.1_ASM77599v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2236 sequences in protein/seqs/GCA_000775995.1_ASM77599v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00045 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000775995.1_ASM77599v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000802305.1_ASM80230v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000802305.1_ASM80230v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000802305.1_ASM80230v1_protein.faa.gz 3508 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3509 sequences in protein/seqs/GCA_000802305.1_ASM80230v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000802305.1_ASM80230v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000818865.1_ASM81886v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000818865.1_ASM81886v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000818865.1_ASM81886v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000941035.1_ASM94103v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000941035.1_ASM94103v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000941035.1_ASM94103v1_protein.faa.gz 4140 sequences\r",
      "\u001b[Kcalculated 3 signatures for 4141 sequences in protein/seqs/GCA_000941035.1_ASM94103v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00068 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000941035.1_ASM94103v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001281815.1_ASM128181v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001281815.1_ASM128181v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001281815.1_ASM128181v1_protein.faa.gz 5400 sequences\r",
      "\u001b[Kcalculated 3 signatures for 5401 sequences in protein/seqs/GCA_001281815.1_ASM128181v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00075 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001281815.1_ASM128181v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001405175.1_13470_2_54_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001405175.1_13470_2_54_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001405175.1_13470_2_54_protein.faa.gz 3200 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3201 sequences in protein/seqs/GCA_001405175.1_13470_2_54_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00065 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001405175.1_13470_2_54_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443305.1_ASM144330v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443305.1_ASM144330v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443305.1_ASM144330v1_protein.faa.gz 3753 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3754 sequences in protein/seqs/GCA_001443305.1_ASM144330v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00065 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001443305.1_ASM144330v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001567655.1_ASM156765v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001567655.1_ASM156765v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001567655.1_ASM156765v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001604335.1_ASM160433v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001604335.1_ASM160433v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001604335.1_ASM160433v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001712955.1_ASM171295v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001712955.1_ASM171295v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001712955.1_ASM171295v1_protein.faa.gz 969 sequences\r",
      "\u001b[Kcalculated 3 signatures for 970 sequences in protein/seqs/GCA_001712955.1_ASM171295v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00031 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001712955.1_ASM171295v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001713255.1_ASM171325v1_protein.faa.gz 970 sequences\r",
      "\u001b[Kcalculated 3 signatures for 971 sequences in protein/seqs/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001713255.1_ASM171325v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001746215.1_ASM174621v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001746215.1_ASM174621v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001746215.1_ASM174621v1_protein.faa.gz 2454 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2455 sequences in protein/seqs/GCA_001746215.1_ASM174621v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00044 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001746215.1_ASM174621v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829125.1_ASM182912v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829125.1_ASM182912v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829125.1_ASM182912v1_protein.faa.gz 3453 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3454 sequences in protein/seqs/GCA_001829125.1_ASM182912v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829125.1_ASM182912v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829165.1_ASM182916v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829165.1_ASM182916v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829165.1_ASM182916v1_protein.faa.gz 4233 sequences\r",
      "\u001b[Kcalculated 3 signatures for 4234 sequences in protein/seqs/GCA_001829165.1_ASM182916v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00070 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829165.1_ASM182916v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829295.1_ASM182929v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829295.1_ASM182929v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829295.1_ASM182929v1_protein.faa.gz 2892 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2893 sequences in protein/seqs/GCA_001829295.1_ASM182929v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00099 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829295.1_ASM182929v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829415.1_ASM182941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829415.1_ASM182941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829415.1_ASM182941v1_protein.faa.gz 3175 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3176 sequences in protein/seqs/GCA_001829415.1_ASM182941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00064 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829415.1_ASM182941v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829475.1_ASM182947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829475.1_ASM182947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829475.1_ASM182947v1_protein.faa.gz 2270 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2271 sequences in protein/seqs/GCA_001829475.1_ASM182947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00039 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829475.1_ASM182947v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829505.1_ASM182950v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829505.1_ASM182950v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829505.1_ASM182950v1_protein.faa.gz 2854 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2855 sequences in protein/seqs/GCA_001829505.1_ASM182950v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00054 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829505.1_ASM182950v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001830585.1_ASM183058v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001830585.1_ASM183058v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001830585.1_ASM183058v1_protein.faa.gz 3118 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3119 sequences in protein/seqs/GCA_001830585.1_ASM183058v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00050 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001830585.1_ASM183058v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001830635.1_ASM183063v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001830635.1_ASM183063v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001830635.1_ASM183063v1_protein.faa.gz 2897 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2898 sequences in protein/seqs/GCA_001830635.1_ASM183063v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00075 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001830635.1_ASM183063v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001894145.1_ASM189414v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001894145.1_ASM189414v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001894145.1_ASM189414v1_protein.faa.gz 5526 sequences\r",
      "\u001b[Kcalculated 3 signatures for 5527 sequences in protein/seqs/GCA_001894145.1_ASM189414v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00178 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001894145.1_ASM189414v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896735.1_ASM189673v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896735.1_ASM189673v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896735.1_ASM189673v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001940825.1_ASM194082v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001940825.1_ASM194082v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001940825.1_ASM194082v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_002009835.1_ASM200983v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_002009835.1_ASM200983v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_002009835.1_ASM200983v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_002069655.1_ASM206965v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_002069655.1_ASM206965v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_002069655.1_ASM206965v1_protein.faa.gz 2058 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2059 sequences in protein/seqs/GCA_002069655.1_ASM206965v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00042 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_002069655.1_ASM206965v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_002069965.1_ASM206996v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_002069965.1_ASM206996v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_002069965.1_ASM206996v1_protein.faa.gz 2774 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2775 sequences in protein/seqs/GCA_002069965.1_ASM206996v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00044 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_002069965.1_ASM206996v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900018035.1_7521_5_26_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900018035.1_7521_5_26_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900018035.1_7521_5_26_protein.faa.gz 2779 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2780 sequences in protein/seqs/GCA_900018035.1_7521_5_26_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00045 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900018035.1_7521_5_26_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900147745.1_138_1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900147745.1_138_1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900147745.1_138_1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz 2897 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2898 sequences in protein/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00047 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900164975.1_16852_2_85_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900164975.1_16852_2_85_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900164975.1_16852_2_85_protein.faa.gz 3653 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3654 sequences in protein/seqs/GCA_900164975.1_16852_2_85_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00062 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900164975.1_16852_2_85_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz 2192 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2193 sequences in protein/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00044 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "bash: line 8: cd: gingivalis: No such file or directory\n",
      "mkdir: cannot create directory genomic/sigs: File exists\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000147075.1_ASM14707v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000147075.1_ASM14707v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00323 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000147075.1_ASM14707v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000216715.2_CLC_glsol140_genomic.fna.gz 315 sequences\r",
      "\u001b[Kcalculated 3 signatures for 316 sequences in genomic/seqs/GCA_000216715.2_CLC_glsol140_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00607 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000216715.2_CLC_glsol140_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000217015.3_CLC_glsol119_genomic.fna.gz 337 sequences\r",
      "\u001b[Kcalculated 3 signatures for 338 sequences in genomic/seqs/GCA_000217015.3_CLC_glsol119_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00624 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000217015.3_CLC_glsol119_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000217655.1_ASM21765v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000217655.1_ASM21765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00156 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000217655.1_ASM21765v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000222305.1_ASM22230v1_genomic.fna.gz 16 sequences\r",
      "\u001b[Kcalculated 3 signatures for 17 sequences in genomic/seqs/GCA_000222305.1_ASM22230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00171 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000222305.1_ASM22230v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000236685.1_ASM23668v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000236685.1_ASM23668v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00483 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000236685.1_ASM23668v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000239475.1_ASM23947v1_genomic.fna.gz 2 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3 sequences in genomic/seqs/GCA_000239475.1_ASM23947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00137 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000239475.1_ASM23947v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000242595.3_ASM24259v3_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000242595.3_ASM24259v3_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00427 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000242595.3_ASM24259v3_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000246415.2_CLC_glsol068_genomic.fna.gz 666 sequences\r",
      "\u001b[Kcalculated 3 signatures for 667 sequences in genomic/seqs/GCA_000246415.2_CLC_glsol068_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00657 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000246415.2_CLC_glsol068_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000246815.1_ASM24681v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000246815.1_ASM24681v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00160 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000246815.1_ASM24681v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000260795.1_ASM26079v1_genomic.fna.gz 146 sequences\r",
      "\u001b[Kcalculated 3 signatures for 147 sequences in genomic/seqs/GCA_000260795.1_ASM26079v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00424 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000260795.1_ASM26079v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000301975.1_ASM30197v1_genomic.fna.gz 115 sequences\r",
      "\u001b[Kcalculated 3 signatures for 116 sequences in genomic/seqs/GCA_000301975.1_ASM30197v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00557 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000301975.1_ASM30197v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz 2 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3 sequences in genomic/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00375 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2 sequences in genomic/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00382 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2 sequences in genomic/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00364 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000338635.1_Trep_dent_ASLM_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00399 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000340605.1_Trep_dent_H1-T_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000340645.1_Trep_dent_H-22_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00376 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000340645.1_Trep_dent_H-22_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00378 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000340725.1_Trep_dent_AL-2_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2 sequences in genomic/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00363 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000340745.1_Trep_dent_US-Trep_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000342705.1_CA_glsol153_genomic.fna.gz 139 sequences\r",
      "\u001b[Kcalculated 3 signatures for 140 sequences in genomic/seqs/GCA_000342705.1_CA_glsol153_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00593 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000342705.1_CA_glsol153_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000343205.1_CA_glsol121_genomic.fna.gz 161 sequences\r",
      "\u001b[Kcalculated 3 signatures for 162 sequences in genomic/seqs/GCA_000343205.1_CA_glsol121_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00590 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000343205.1_CA_glsol121_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000382565.1_ASM38256v1_genomic.fna.gz 6 sequences\r",
      "\u001b[Kcalculated 3 signatures for 7 sequences in genomic/seqs/GCA_000382565.1_ASM38256v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00115 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000382565.1_ASM38256v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz 7 sequences\r",
      "\u001b[Kcalculated 3 signatures for 8 sequences in genomic/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00362 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz 5 sequences\r",
      "\u001b[Kcalculated 3 signatures for 6 sequences in genomic/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00379 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz 2 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3 sequences in genomic/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00362 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000426705.1_ASM42670v1_genomic.fna.gz 28 sequences\r",
      "\u001b[Kcalculated 3 signatures for 29 sequences in genomic/seqs/GCA_000426705.1_ASM42670v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00526 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000426705.1_ASM42670v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz 557 sequences\r",
      "\u001b[Kcalculated 3 signatures for 558 sequences in genomic/seqs/GCA_000465015.1_Treponema_pedis_T_M1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00382 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000465015.1_Treponema_pedis_T_M1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000468055.1_ASM46805v1_genomic.fna.gz 20 sequences\r",
      "\u001b[Kcalculated 3 signatures for 21 sequences in genomic/seqs/GCA_000468055.1_ASM46805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00354 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000468055.1_ASM46805v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000500065.1_AB2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000500065.1_AB2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000500065.1_AB2_genomic.fna.gz 167 sequences\r",
      "\u001b[Kcalculated 3 signatures for 168 sequences in genomic/seqs/GCA_000500065.1_AB2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00212 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000500065.1_AB2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000501035.1_BorGarIPT95_genomic.fna.gz 193 sequences\r",
      "\u001b[Kcalculated 3 signatures for 194 sequences in genomic/seqs/GCA_000501035.1_BorGarIPT95_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00031 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000501035.1_BorGarIPT95_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000501115.1_BorGarIPT101_genomic.fna.gz 712 sequences\r",
      "\u001b[Kcalculated 3 signatures for 713 sequences in genomic/seqs/GCA_000501115.1_BorGarIPT101_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00128 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000501115.1_BorGarIPT101_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz 536 sequences\r",
      "\u001b[Kcalculated 3 signatures for 537 sequences in genomic/seqs/GCA_000501555.1_BorAfzIPT109_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00285 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000501555.1_BorAfzIPT109_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz 192 sequences\r",
      "\u001b[Kcalculated 3 signatures for 193 sequences in genomic/seqs/GCA_000502195.1_BorAfzIPT138_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00182 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000502195.1_BorAfzIPT138_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000507245.1_ASM50724v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000507245.1_ASM50724v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00483 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000507245.1_ASM50724v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz 9 sequences\r",
      "\u001b[Kcalculated 3 signatures for 10 sequences in genomic/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00381 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000537795.1_Stap_aure_SMMC6007_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000568735.2_ASM56873v2_genomic.fna.gz 73 sequences\r",
      "\u001b[Kcalculated 3 signatures for 74 sequences in genomic/seqs/GCA_000568735.2_ASM56873v2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00143 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000568735.2_ASM56873v2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000758165.1_Spiroch1.0_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 3 signatures for 72 sequences in genomic/seqs/GCA_000758165.1_Spiroch1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00438 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000758165.1_Spiroch1.0_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000768055.1_ASM76805v1_genomic.fna.gz 274 sequences\r",
      "\u001b[Kcalculated 3 signatures for 275 sequences in genomic/seqs/GCA_000768055.1_ASM76805v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00537 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000768055.1_ASM76805v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000775995.1_ASM77599v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_000775995.1_ASM77599v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00366 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000775995.1_ASM77599v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000802305.1_ASM80230v1_genomic.fna.gz 54 sequences\r",
      "\u001b[Kcalculated 3 signatures for 55 sequences in genomic/seqs/GCA_000802305.1_ASM80230v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00578 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000802305.1_ASM80230v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000818865.1_ASM81886v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000818865.1_ASM81886v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000818865.1_ASM81886v1_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2 sequences in genomic/seqs/GCA_000818865.1_ASM81886v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00710 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000818865.1_ASM81886v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000941035.1_ASM94103v1_genomic.fna.gz 4 sequences\r",
      "\u001b[Kcalculated 3 signatures for 5 sequences in genomic/seqs/GCA_000941035.1_ASM94103v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00623 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000941035.1_ASM94103v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001281815.1_ASM128181v1_genomic.fna.gz 285 sequences\r",
      "\u001b[Kcalculated 3 signatures for 286 sequences in genomic/seqs/GCA_001281815.1_ASM128181v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00704 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001281815.1_ASM128181v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001405175.1_13470_2_54_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 3 signatures for 72 sequences in genomic/seqs/GCA_001405175.1_13470_2_54_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00470 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001405175.1_13470_2_54_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001443305.1_ASM144330v1_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2 sequences in genomic/seqs/GCA_001443305.1_ASM144330v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.01033 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001443305.1_ASM144330v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001567655.1_ASM156765v1_genomic.fna.gz 254 sequences\r",
      "\u001b[Kcalculated 3 signatures for 255 sequences in genomic/seqs/GCA_001567655.1_ASM156765v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00855 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001567655.1_ASM156765v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001604335.1_ASM160433v1_genomic.fna.gz 591 sequences\r",
      "\u001b[Kcalculated 3 signatures for 592 sequences in genomic/seqs/GCA_001604335.1_ASM160433v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00353 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001604335.1_ASM160433v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001712955.1_ASM171295v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_001712955.1_ASM171295v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00153 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001712955.1_ASM171295v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001713255.1_ASM171325v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1 sequences in genomic/seqs/GCA_001713255.1_ASM171325v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00151 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001713255.1_ASM171325v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001746215.1_ASM174621v1_genomic.fna.gz 177 sequences\r",
      "\u001b[Kcalculated 3 signatures for 178 sequences in genomic/seqs/GCA_001746215.1_ASM174621v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00416 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001746215.1_ASM174621v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829125.1_ASM182912v1_genomic.fna.gz 272 sequences\r",
      "\u001b[Kcalculated 3 signatures for 273 sequences in genomic/seqs/GCA_001829125.1_ASM182912v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00607 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829125.1_ASM182912v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829165.1_ASM182916v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829165.1_ASM182916v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829165.1_ASM182916v1_genomic.fna.gz 372 sequences\r",
      "\u001b[Kcalculated 3 signatures for 373 sequences in genomic/seqs/GCA_001829165.1_ASM182916v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00653 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829165.1_ASM182916v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829295.1_ASM182929v1_genomic.fna.gz 137 sequences\r",
      "\u001b[Kcalculated 3 signatures for 138 sequences in genomic/seqs/GCA_001829295.1_ASM182929v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00483 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829295.1_ASM182929v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829415.1_ASM182941v1_genomic.fna.gz 76 sequences\r",
      "\u001b[Kcalculated 3 signatures for 77 sequences in genomic/seqs/GCA_001829415.1_ASM182941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00472 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829415.1_ASM182941v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829475.1_ASM182947v1_genomic.fna.gz 208 sequences\r",
      "\u001b[Kcalculated 3 signatures for 209 sequences in genomic/seqs/GCA_001829475.1_ASM182947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00518 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829475.1_ASM182947v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001829505.1_ASM182950v1_genomic.fna.gz 244 sequences\r",
      "\u001b[Kcalculated 3 signatures for 245 sequences in genomic/seqs/GCA_001829505.1_ASM182950v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00875 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001829505.1_ASM182950v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001830585.1_ASM183058v1_genomic.fna.gz 266 sequences\r",
      "\u001b[Kcalculated 3 signatures for 267 sequences in genomic/seqs/GCA_001830585.1_ASM183058v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.01030 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001830585.1_ASM183058v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001830635.1_ASM183063v1_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 3 signatures for 67 sequences in genomic/seqs/GCA_001830635.1_ASM183063v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00461 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001830635.1_ASM183063v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001894145.1_ASM189414v1_genomic.fna.gz 439 sequences\r",
      "\u001b[Kcalculated 3 signatures for 440 sequences in genomic/seqs/GCA_001894145.1_ASM189414v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00824 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001894145.1_ASM189414v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001896735.1_ASM189673v1_genomic.fna.gz 159 sequences\r",
      "\u001b[Kcalculated 3 signatures for 160 sequences in genomic/seqs/GCA_001896735.1_ASM189673v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00450 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001896735.1_ASM189673v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001940825.1_ASM194082v1_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 3 signatures for 72 sequences in genomic/seqs/GCA_001940825.1_ASM194082v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00313 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001940825.1_ASM194082v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_002009835.1_ASM200983v1_genomic.fna.gz 336 sequences\r",
      "\u001b[Kcalculated 3 signatures for 337 sequences in genomic/seqs/GCA_002009835.1_ASM200983v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00712 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_002009835.1_ASM200983v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_002069655.1_ASM206965v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_002069655.1_ASM206965v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_002069655.1_ASM206965v1_genomic.fna.gz 121 sequences\r",
      "\u001b[Kcalculated 3 signatures for 122 sequences in genomic/seqs/GCA_002069655.1_ASM206965v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00360 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_002069655.1_ASM206965v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_002069965.1_ASM206996v1_genomic.fna.gz 159 sequences\r",
      "\u001b[Kcalculated 3 signatures for 160 sequences in genomic/seqs/GCA_002069965.1_ASM206996v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00447 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_002069965.1_ASM206996v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900018035.1_7521_5_26_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 3 signatures for 72 sequences in genomic/seqs/GCA_900018035.1_7521_5_26_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00386 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900018035.1_7521_5_26_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900147745.1_138_1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900147745.1_138_1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900147745.1_138_1_genomic.fna.gz 281 sequences\r",
      "\u001b[Kcalculated 3 signatures for 282 sequences in genomic/seqs/GCA_900147745.1_138_1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00853 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900147745.1_138_1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 3 signatures for 65 sequences in genomic/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00452 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900164975.1_16852_2_85_genomic.fna.gz 30 sequences\r",
      "\u001b[Kcalculated 3 signatures for 31 sequences in genomic/seqs/GCA_900164975.1_16852_2_85_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.03549 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900164975.1_16852_2_85_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz 31 sequences\r",
      "\u001b[Kcalculated 3 signatures for 32 sequences in genomic/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00363 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "bash: line 8: cd: gingivalis: No such file or directory\n",
      "mkdir: cannot create directory rna/sigs: File exists\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000147075.1_ASM14707v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000147075.1_ASM14707v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000147075.1_ASM14707v1_rna_from_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 3 signatures for 52 sequences in rna/seqs/GCA_000147075.1_ASM14707v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000147075.1_ASM14707v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000216715.2_CLC_glsol140_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000216715.2_CLC_glsol140_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000216715.2_CLC_glsol140_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000217015.3_CLC_glsol119_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000217015.3_CLC_glsol119_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000217015.3_CLC_glsol119_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000217655.1_ASM21765v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000217655.1_ASM21765v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000217655.1_ASM21765v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_000217655.1_ASM21765v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000217655.1_ASM21765v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000222305.1_ASM22230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000222305.1_ASM22230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000222305.1_ASM22230v1_rna_from_genomic.fna.gz 37 sequences\r",
      "\u001b[Kcalculated 3 signatures for 38 sequences in rna/seqs/GCA_000222305.1_ASM22230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00072 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000222305.1_ASM22230v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000236685.1_ASM23668v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000236685.1_ASM23668v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000236685.1_ASM23668v1_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 3 signatures for 67 sequences in rna/seqs/GCA_000236685.1_ASM23668v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000236685.1_ASM23668v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000239475.1_ASM23947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000239475.1_ASM23947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000239475.1_ASM23947v1_rna_from_genomic.fna.gz 35 sequences\r",
      "\u001b[Kcalculated 3 signatures for 36 sequences in rna/seqs/GCA_000239475.1_ASM23947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000239475.1_ASM23947v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000242595.3_ASM24259v3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000242595.3_ASM24259v3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000242595.3_ASM24259v3_rna_from_genomic.fna.gz 55 sequences\r",
      "\u001b[Kcalculated 3 signatures for 56 sequences in rna/seqs/GCA_000242595.3_ASM24259v3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000242595.3_ASM24259v3_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000246415.2_CLC_glsol068_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000246415.2_CLC_glsol068_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000246415.2_CLC_glsol068_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_000246815.1_ASM24681v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000246815.1_ASM24681v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000260795.1_ASM26079v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000260795.1_ASM26079v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000260795.1_ASM26079v1_rna_from_genomic.fna.gz 43 sequences\r",
      "\u001b[Kcalculated 3 signatures for 44 sequences in rna/seqs/GCA_000260795.1_ASM26079v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000260795.1_ASM26079v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000301975.1_ASM30197v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000301975.1_ASM30197v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000301975.1_ASM30197v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.fna.gz 47 sequences\r",
      "\u001b[Kcalculated 3 signatures for 48 sequences in rna/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000338635.1_Trep_dent_ASLM_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000340605.1_Trep_dent_H1-T_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000340645.1_Trep_dent_H-22_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000340725.1_Trep_dent_AL-2_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000340745.1_Trep_dent_US-Trep_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000342705.1_CA_glsol153_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000342705.1_CA_glsol153_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000342705.1_CA_glsol153_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000343205.1_CA_glsol121_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000343205.1_CA_glsol121_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000343205.1_CA_glsol121_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000382565.1_ASM38256v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000382565.1_ASM38256v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000382565.1_ASM38256v1_rna_from_genomic.fna.gz 34 sequences\r",
      "\u001b[Kcalculated 3 signatures for 35 sequences in rna/seqs/GCA_000382565.1_ASM38256v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000382565.1_ASM38256v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000426705.1_ASM42670v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000426705.1_ASM42670v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000426705.1_ASM42670v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000465015.1_Treponema_pedis_T_M1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000465015.1_Treponema_pedis_T_M1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000465015.1_Treponema_pedis_T_M1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000468055.1_ASM46805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000468055.1_ASM46805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000468055.1_ASM46805v1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000468055.1_ASM46805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000468055.1_ASM46805v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000500065.1_AB2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000500065.1_AB2_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000500065.1_AB2_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000501035.1_BorGarIPT95_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000501035.1_BorGarIPT95_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000501035.1_BorGarIPT95_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000501115.1_BorGarIPT101_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000501115.1_BorGarIPT101_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000501115.1_BorGarIPT101_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000501555.1_BorAfzIPT109_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000501555.1_BorAfzIPT109_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000501555.1_BorAfzIPT109_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000502195.1_BorAfzIPT138_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000502195.1_BorAfzIPT138_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000502195.1_BorAfzIPT138_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000507245.1_ASM50724v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000507245.1_ASM50724v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000507245.1_ASM50724v1_rna_from_genomic.fna.gz 49 sequences\r",
      "\u001b[Kcalculated 3 signatures for 50 sequences in rna/seqs/GCA_000507245.1_ASM50724v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000507245.1_ASM50724v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.fna.gz 75 sequences\r",
      "\u001b[Kcalculated 3 signatures for 76 sequences in rna/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000537795.1_Stap_aure_SMMC6007_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000568735.2_ASM56873v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000568735.2_ASM56873v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000568735.2_ASM56873v2_rna_from_genomic.fna.gz 33 sequences\r",
      "\u001b[Kcalculated 3 signatures for 34 sequences in rna/seqs/GCA_000568735.2_ASM56873v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000568735.2_ASM56873v2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000758165.1_Spiroch1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000758165.1_Spiroch1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000758165.1_Spiroch1.0_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_000758165.1_Spiroch1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000758165.1_Spiroch1.0_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000768055.1_ASM76805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000768055.1_ASM76805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000768055.1_ASM76805v1_rna_from_genomic.fna.gz 23 sequences\r",
      "\u001b[Kcalculated 3 signatures for 24 sequences in rna/seqs/GCA_000768055.1_ASM76805v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000768055.1_ASM76805v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000775995.1_ASM77599v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000775995.1_ASM77599v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000775995.1_ASM77599v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_000775995.1_ASM77599v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000775995.1_ASM77599v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000802305.1_ASM80230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000802305.1_ASM80230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000802305.1_ASM80230v1_rna_from_genomic.fna.gz 48 sequences\r",
      "\u001b[Kcalculated 3 signatures for 49 sequences in rna/seqs/GCA_000802305.1_ASM80230v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000802305.1_ASM80230v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000818865.1_ASM81886v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000818865.1_ASM81886v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000818865.1_ASM81886v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000941035.1_ASM94103v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000941035.1_ASM94103v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000941035.1_ASM94103v1_rna_from_genomic.fna.gz 41 sequences\r",
      "\u001b[Kcalculated 3 signatures for 42 sequences in rna/seqs/GCA_000941035.1_ASM94103v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000941035.1_ASM94103v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001281815.1_ASM128181v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001281815.1_ASM128181v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001281815.1_ASM128181v1_rna_from_genomic.fna.gz 109 sequences\r",
      "\u001b[Kcalculated 3 signatures for 110 sequences in rna/seqs/GCA_001281815.1_ASM128181v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001281815.1_ASM128181v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001405175.1_13470_2_54_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001405175.1_13470_2_54_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001405175.1_13470_2_54_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 3 signatures for 67 sequences in rna/seqs/GCA_001405175.1_13470_2_54_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001405175.1_13470_2_54_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001443305.1_ASM144330v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001443305.1_ASM144330v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001443305.1_ASM144330v1_rna_from_genomic.fna.gz 40 sequences\r",
      "\u001b[Kcalculated 3 signatures for 41 sequences in rna/seqs/GCA_001443305.1_ASM144330v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001443305.1_ASM144330v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001567655.1_ASM156765v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001567655.1_ASM156765v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001567655.1_ASM156765v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001604335.1_ASM160433v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001604335.1_ASM160433v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001604335.1_ASM160433v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001712955.1_ASM171295v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001712955.1_ASM171295v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001712955.1_ASM171295v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_001712955.1_ASM171295v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001712955.1_ASM171295v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001713255.1_ASM171325v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001713255.1_ASM171325v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001713255.1_ASM171325v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 3 signatures for 54 sequences in rna/seqs/GCA_001713255.1_ASM171325v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001713255.1_ASM171325v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001746215.1_ASM174621v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001746215.1_ASM174621v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001746215.1_ASM174621v1_rna_from_genomic.fna.gz 37 sequences\r",
      "\u001b[Kcalculated 3 signatures for 38 sequences in rna/seqs/GCA_001746215.1_ASM174621v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001746215.1_ASM174621v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829125.1_ASM182912v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829125.1_ASM182912v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829125.1_ASM182912v1_rna_from_genomic.fna.gz 38 sequences\r",
      "\u001b[Kcalculated 3 signatures for 39 sequences in rna/seqs/GCA_001829125.1_ASM182912v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829125.1_ASM182912v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829165.1_ASM182916v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829165.1_ASM182916v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829165.1_ASM182916v1_rna_from_genomic.fna.gz 99 sequences\r",
      "\u001b[Kcalculated 3 signatures for 100 sequences in rna/seqs/GCA_001829165.1_ASM182916v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829165.1_ASM182916v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829295.1_ASM182929v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829295.1_ASM182929v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829295.1_ASM182929v1_rna_from_genomic.fna.gz 45 sequences\r",
      "\u001b[Kcalculated 3 signatures for 46 sequences in rna/seqs/GCA_001829295.1_ASM182929v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829295.1_ASM182929v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829415.1_ASM182941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829415.1_ASM182941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829415.1_ASM182941v1_rna_from_genomic.fna.gz 45 sequences\r",
      "\u001b[Kcalculated 3 signatures for 46 sequences in rna/seqs/GCA_001829415.1_ASM182941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829415.1_ASM182941v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829475.1_ASM182947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829475.1_ASM182947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829475.1_ASM182947v1_rna_from_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 3 signatures for 33 sequences in rna/seqs/GCA_001829475.1_ASM182947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829475.1_ASM182947v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001829505.1_ASM182950v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001829505.1_ASM182950v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001829505.1_ASM182950v1_rna_from_genomic.fna.gz 36 sequences\r",
      "\u001b[Kcalculated 3 signatures for 37 sequences in rna/seqs/GCA_001829505.1_ASM182950v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001829505.1_ASM182950v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001830585.1_ASM183058v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001830585.1_ASM183058v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001830585.1_ASM183058v1_rna_from_genomic.fna.gz 28 sequences\r",
      "\u001b[Kcalculated 3 signatures for 29 sequences in rna/seqs/GCA_001830585.1_ASM183058v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001830585.1_ASM183058v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001830635.1_ASM183063v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001830635.1_ASM183063v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001830635.1_ASM183063v1_rna_from_genomic.fna.gz 47 sequences\r",
      "\u001b[Kcalculated 3 signatures for 48 sequences in rna/seqs/GCA_001830635.1_ASM183063v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001830635.1_ASM183063v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001894145.1_ASM189414v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001894145.1_ASM189414v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001894145.1_ASM189414v1_rna_from_genomic.fna.gz 105 sequences\r",
      "\u001b[Kcalculated 3 signatures for 106 sequences in rna/seqs/GCA_001894145.1_ASM189414v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001894145.1_ASM189414v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001896735.1_ASM189673v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001896735.1_ASM189673v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001896735.1_ASM189673v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001940825.1_ASM194082v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001940825.1_ASM194082v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001940825.1_ASM194082v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_002009835.1_ASM200983v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_002009835.1_ASM200983v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_002009835.1_ASM200983v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_002069655.1_ASM206965v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_002069655.1_ASM206965v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_002069655.1_ASM206965v1_rna_from_genomic.fna.gz 28 sequences\r",
      "\u001b[Kcalculated 3 signatures for 29 sequences in rna/seqs/GCA_002069655.1_ASM206965v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_002069655.1_ASM206965v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_002069965.1_ASM206996v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_002069965.1_ASM206996v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_002069965.1_ASM206996v1_rna_from_genomic.fna.gz 45 sequences\r",
      "\u001b[Kcalculated 3 signatures for 46 sequences in rna/seqs/GCA_002069965.1_ASM206996v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_002069965.1_ASM206996v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900018035.1_7521_5_26_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900018035.1_7521_5_26_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900018035.1_7521_5_26_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 3 signatures for 67 sequences in rna/seqs/GCA_900018035.1_7521_5_26_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900018035.1_7521_5_26_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900147745.1_138_1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900147745.1_138_1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_900147745.1_138_1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.fna.gz 57 sequences\r",
      "\u001b[Kcalculated 3 signatures for 58 sequences in rna/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900164975.1_16852_2_85_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900164975.1_16852_2_85_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900164975.1_16852_2_85_rna_from_genomic.fna.gz 91 sequences\r",
      "\u001b[Kcalculated 3 signatures for 92 sequences in rna/seqs/GCA_900164975.1_16852_2_85_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900164975.1_16852_2_85_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21, 31, 51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.fna.gz 55 sequences\r",
      "\u001b[Kcalculated 3 signatures for 56 sequences in rna/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "bash: line 21: cd: gingivalis: No such file or directory\n",
      "mkdir: cannot create directory protein/sigs: File exists\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000147075.1_ASM14707v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000147075.1_ASM14707v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000147075.1_ASM14707v1_protein.faa.gz 2202 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2203 sequences in protein/seqs/GCA_000147075.1_ASM14707v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00042 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000147075.1_ASM14707v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000216715.2_CLC_glsol140_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000216715.2_CLC_glsol140_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000216715.2_CLC_glsol140_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217015.3_CLC_glsol119_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217015.3_CLC_glsol119_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217015.3_CLC_glsol119_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217655.1_ASM21765v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217655.1_ASM21765v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000217655.1_ASM21765v1_protein.faa.gz 1009 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1010 sequences in protein/seqs/GCA_000217655.1_ASM21765v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000217655.1_ASM21765v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000222305.1_ASM22230v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000222305.1_ASM22230v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000222305.1_ASM22230v1_protein.faa.gz 1462 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1463 sequences in protein/seqs/GCA_000222305.1_ASM22230v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00029 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000222305.1_ASM22230v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000236685.1_ASM23668v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000236685.1_ASM23668v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000236685.1_ASM23668v1_protein.faa.gz 3158 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3159 sequences in protein/seqs/GCA_000236685.1_ASM23668v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00060 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000236685.1_ASM23668v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000239475.1_ASM23947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000239475.1_ASM23947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000239475.1_ASM23947v1_protein.faa.gz 955 sequences\r",
      "\u001b[Kcalculated 3 signatures for 956 sequences in protein/seqs/GCA_000239475.1_ASM23947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00028 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000239475.1_ASM23947v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000242595.3_ASM24259v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000242595.3_ASM24259v3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000242595.3_ASM24259v3_protein.faa.gz 2781 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2782 sequences in protein/seqs/GCA_000242595.3_ASM24259v3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00055 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000242595.3_ASM24259v3_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000246415.2_CLC_glsol068_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000246415.2_CLC_glsol068_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000246415.2_CLC_glsol068_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000246815.1_ASM24681v1_protein.faa.gz 1058 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1059 sequences in protein/seqs/GCA_000246815.1_ASM24681v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00031 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000246815.1_ASM24681v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000260795.1_ASM26079v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000260795.1_ASM26079v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000260795.1_ASM26079v1_protein.faa.gz 2569 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2570 sequences in protein/seqs/GCA_000260795.1_ASM26079v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00053 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000260795.1_ASM26079v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000301975.1_ASM30197v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000301975.1_ASM30197v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000301975.1_ASM30197v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz 2530 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2531 sequences in protein/seqs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00046 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000338595.1_Trep_dent_ATCC_33521_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz 2646 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2647 sequences in protein/seqs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00046 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000338615.1_Trep_dent_ATCC_33520_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz 2568 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2569 sequences in protein/seqs/GCA_000338635.1_Trep_dent_ASLM_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00046 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000338635.1_Trep_dent_ASLM_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz 2704 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2705 sequences in protein/seqs/GCA_000340605.1_Trep_dent_H1-T_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00140 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000340605.1_Trep_dent_H1-T_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz 2519 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2520 sequences in protein/seqs/GCA_000340645.1_Trep_dent_H-22_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00046 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000340645.1_Trep_dent_H-22_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz 2631 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2632 sequences in protein/seqs/GCA_000340725.1_Trep_dent_AL-2_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00047 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000340725.1_Trep_dent_AL-2_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz 2550 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2551 sequences in protein/seqs/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00047 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000340745.1_Trep_dent_US-Trep_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000342705.1_CA_glsol153_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000342705.1_CA_glsol153_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000342705.1_CA_glsol153_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000343205.1_CA_glsol121_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000343205.1_CA_glsol121_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000343205.1_CA_glsol121_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000382565.1_ASM38256v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000382565.1_ASM38256v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000382565.1_ASM38256v1_protein.faa.gz 831 sequences\r",
      "\u001b[Kcalculated 3 signatures for 832 sequences in protein/seqs/GCA_000382565.1_ASM38256v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00027 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000382565.1_ASM38256v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz 2609 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2610 sequences in protein/seqs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00047 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000413075.1_Trep_dent_SP44_F0466_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz 2792 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2793 sequences in protein/seqs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00047 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000413095.1_Trep_dent_SP32_F0464_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz 2613 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2614 sequences in protein/seqs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00046 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000413115.1_Trep_dent_SP23_F0463_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000426705.1_ASM42670v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000426705.1_ASM42670v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000426705.1_ASM42670v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000465015.1_Treponema_pedis_T_M1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000465015.1_Treponema_pedis_T_M1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000465015.1_Treponema_pedis_T_M1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000468055.1_ASM46805v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000468055.1_ASM46805v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000468055.1_ASM46805v1_protein.faa.gz 2174 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2175 sequences in protein/seqs/GCA_000468055.1_ASM46805v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00047 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000468055.1_ASM46805v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000500065.1_AB2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000500065.1_AB2_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000500065.1_AB2_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000501035.1_BorGarIPT95_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000501035.1_BorGarIPT95_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000501035.1_BorGarIPT95_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000501115.1_BorGarIPT101_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000501115.1_BorGarIPT101_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000501115.1_BorGarIPT101_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000501555.1_BorAfzIPT109_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000501555.1_BorAfzIPT109_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000501555.1_BorAfzIPT109_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000502195.1_BorAfzIPT138_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000502195.1_BorAfzIPT138_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000502195.1_BorAfzIPT138_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000507245.1_ASM50724v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000507245.1_ASM50724v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000507245.1_ASM50724v1_protein.faa.gz 3396 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3397 sequences in protein/seqs/GCA_000507245.1_ASM50724v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00060 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000507245.1_ASM50724v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz 2682 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2683 sequences in protein/seqs/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00045 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000537795.1_Stap_aure_SMMC6007_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000568735.2_ASM56873v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000568735.2_ASM56873v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000568735.2_ASM56873v2_protein.faa.gz 1084 sequences\r",
      "\u001b[Kcalculated 3 signatures for 1085 sequences in protein/seqs/GCA_000568735.2_ASM56873v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00029 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000568735.2_ASM56873v2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000758165.1_Spiroch1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000758165.1_Spiroch1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000758165.1_Spiroch1.0_protein.faa.gz 2302 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2303 sequences in protein/seqs/GCA_000758165.1_Spiroch1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00103 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000758165.1_Spiroch1.0_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768055.1_ASM76805v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768055.1_ASM76805v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768055.1_ASM76805v1_protein.faa.gz 2731 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2732 sequences in protein/seqs/GCA_000768055.1_ASM76805v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00054 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000768055.1_ASM76805v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000775995.1_ASM77599v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000775995.1_ASM77599v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000775995.1_ASM77599v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2236 sequences in protein/seqs/GCA_000775995.1_ASM77599v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00044 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000775995.1_ASM77599v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000802305.1_ASM80230v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000802305.1_ASM80230v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000802305.1_ASM80230v1_protein.faa.gz 3508 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3509 sequences in protein/seqs/GCA_000802305.1_ASM80230v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00053 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000802305.1_ASM80230v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000818865.1_ASM81886v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000818865.1_ASM81886v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000818865.1_ASM81886v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000941035.1_ASM94103v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000941035.1_ASM94103v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000941035.1_ASM94103v1_protein.faa.gz 4140 sequences\r",
      "\u001b[Kcalculated 3 signatures for 4141 sequences in protein/seqs/GCA_000941035.1_ASM94103v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000941035.1_ASM94103v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001281815.1_ASM128181v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001281815.1_ASM128181v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001281815.1_ASM128181v1_protein.faa.gz 5400 sequences\r",
      "\u001b[Kcalculated 3 signatures for 5401 sequences in protein/seqs/GCA_001281815.1_ASM128181v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00066 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001281815.1_ASM128181v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001405175.1_13470_2_54_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001405175.1_13470_2_54_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001405175.1_13470_2_54_protein.faa.gz 3200 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3201 sequences in protein/seqs/GCA_001405175.1_13470_2_54_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00057 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001405175.1_13470_2_54_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443305.1_ASM144330v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443305.1_ASM144330v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443305.1_ASM144330v1_protein.faa.gz 3753 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3754 sequences in protein/seqs/GCA_001443305.1_ASM144330v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00059 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001443305.1_ASM144330v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001567655.1_ASM156765v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001567655.1_ASM156765v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001567655.1_ASM156765v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001604335.1_ASM160433v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001604335.1_ASM160433v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001604335.1_ASM160433v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001712955.1_ASM171295v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001712955.1_ASM171295v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001712955.1_ASM171295v1_protein.faa.gz 969 sequences\r",
      "\u001b[Kcalculated 3 signatures for 970 sequences in protein/seqs/GCA_001712955.1_ASM171295v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00054 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001712955.1_ASM171295v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001713255.1_ASM171325v1_protein.faa.gz 970 sequences\r",
      "\u001b[Kcalculated 3 signatures for 971 sequences in protein/seqs/GCA_001713255.1_ASM171325v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001713255.1_ASM171325v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001746215.1_ASM174621v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001746215.1_ASM174621v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001746215.1_ASM174621v1_protein.faa.gz 2454 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2455 sequences in protein/seqs/GCA_001746215.1_ASM174621v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00040 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001746215.1_ASM174621v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829125.1_ASM182912v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829125.1_ASM182912v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829125.1_ASM182912v1_protein.faa.gz 3453 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3454 sequences in protein/seqs/GCA_001829125.1_ASM182912v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00059 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829125.1_ASM182912v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829165.1_ASM182916v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829165.1_ASM182916v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829165.1_ASM182916v1_protein.faa.gz 4233 sequences\r",
      "\u001b[Kcalculated 3 signatures for 4234 sequences in protein/seqs/GCA_001829165.1_ASM182916v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829165.1_ASM182916v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829295.1_ASM182929v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829295.1_ASM182929v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829295.1_ASM182929v1_protein.faa.gz 2892 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2893 sequences in protein/seqs/GCA_001829295.1_ASM182929v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00056 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829295.1_ASM182929v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829415.1_ASM182941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829415.1_ASM182941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829415.1_ASM182941v1_protein.faa.gz 3175 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3176 sequences in protein/seqs/GCA_001829415.1_ASM182941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00059 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829415.1_ASM182941v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829475.1_ASM182947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829475.1_ASM182947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829475.1_ASM182947v1_protein.faa.gz 2270 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2271 sequences in protein/seqs/GCA_001829475.1_ASM182947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829475.1_ASM182947v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001829505.1_ASM182950v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001829505.1_ASM182950v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001829505.1_ASM182950v1_protein.faa.gz 2854 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2855 sequences in protein/seqs/GCA_001829505.1_ASM182950v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00046 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001829505.1_ASM182950v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001830585.1_ASM183058v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001830585.1_ASM183058v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001830585.1_ASM183058v1_protein.faa.gz 3118 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3119 sequences in protein/seqs/GCA_001830585.1_ASM183058v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00045 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001830585.1_ASM183058v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001830635.1_ASM183063v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001830635.1_ASM183063v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001830635.1_ASM183063v1_protein.faa.gz 2897 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2898 sequences in protein/seqs/GCA_001830635.1_ASM183063v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00055 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001830635.1_ASM183063v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001894145.1_ASM189414v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001894145.1_ASM189414v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001894145.1_ASM189414v1_protein.faa.gz 5526 sequences\r",
      "\u001b[Kcalculated 3 signatures for 5527 sequences in protein/seqs/GCA_001894145.1_ASM189414v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00072 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001894145.1_ASM189414v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896735.1_ASM189673v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896735.1_ASM189673v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896735.1_ASM189673v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001940825.1_ASM194082v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001940825.1_ASM194082v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001940825.1_ASM194082v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_002009835.1_ASM200983v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_002009835.1_ASM200983v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_002009835.1_ASM200983v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_002069655.1_ASM206965v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_002069655.1_ASM206965v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_002069655.1_ASM206965v1_protein.faa.gz 2058 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2059 sequences in protein/seqs/GCA_002069655.1_ASM206965v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00041 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_002069655.1_ASM206965v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_002069965.1_ASM206996v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_002069965.1_ASM206996v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_002069965.1_ASM206996v1_protein.faa.gz 2774 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2775 sequences in protein/seqs/GCA_002069965.1_ASM206996v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00045 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_002069965.1_ASM206996v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900018035.1_7521_5_26_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900018035.1_7521_5_26_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900018035.1_7521_5_26_protein.faa.gz 2779 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2780 sequences in protein/seqs/GCA_900018035.1_7521_5_26_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00045 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900018035.1_7521_5_26_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900147745.1_138_1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900147745.1_138_1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900147745.1_138_1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz 2897 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2898 sequences in protein/seqs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00047 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900156105.1_IMG-taxon_2706795025_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900164975.1_16852_2_85_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900164975.1_16852_2_85_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900164975.1_16852_2_85_protein.faa.gz 3653 sequences\r",
      "\u001b[Kcalculated 3 signatures for 3654 sequences in protein/seqs/GCA_900164975.1_16852_2_85_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00055 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900164975.1_16852_2_85_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7, 12, 15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 3 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz 2192 sequences\r",
      "\u001b[Kcalculated 3 signatures for 2193 sequences in protein/seqs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00044 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900167145.1_IMG-taxon_2582581326_annotated_assembly_protein.sig. Note: signature license is CC0.\r"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "#Calc signatures\n",
    "\n",
    "\n",
    "for org in bacteroides denticola gingivalis\n",
    "do\n",
    "    for mol in genomic rna \n",
    "    do\n",
    "        cd $org\n",
    "        mkdir $mol/sigs\n",
    "        for infile in $mol/seqs/*a.gz \n",
    "        do \n",
    "        for k in 21 31 51\n",
    "        do\n",
    "        \n",
    "            out_name=$(basename $infile .fna.gz)\n",
    "            sourmash compute -k $k --scaled 2000 --track-abundance -o $mol/sigs/${out_name}.sig ${infile}\n",
    "            \n",
    "            #back to relative home dir, overwrite and append this file\n",
    "            cd ..\n",
    "            echo \"$k\" >calcd_k_nuclc_acids.txt\n",
    "        done\n",
    "        done\n",
    "    done\n",
    "    \n",
    "    \n",
    "    for mol in protein\n",
    "    do\n",
    "\n",
    "        cd $org\n",
    "        mkdir $mol/sigs\n",
    "        for infile in $mol/seqs/*a.gz \n",
    "        do \n",
    "        for k in 7 12 15\n",
    "        do\n",
    "            out_name=$(basename $infile .faa.gz)\n",
    "            sourmash compute --input-is-protein -k $k --scaled 2000 --track-abundance -o $mol/sigs/${out_name}.sig ${infile}\n",
    "           \n",
    "           #back to relative home dir, overwrite and append this file\n",
    "            cd ..\n",
    "            echo \"$k\" >calcd_k_amino_acids.txt\n",
    "        done\n",
    "        done\n",
    "    done\n",
    "   \n",
    "   \n",
    "done\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "7 12 15\n",
      "21 31 51\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "echo \"7 12 15\" >calcd_k_amino_acids.txt\n",
    "echo \"21 31 51\" >calcd_k_nuclc_acids.txt\n",
    "\n",
    "less calcd_k_amino_acids.txt\n",
    "\n",
    "less calcd_k_nuclc_acids.txt"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
