{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "    #activate environment\n",
    "#conda activate rotation\n",
    "    #install pandas, os, re, argparse, ipfsapi, requests, shutil; e.g.\n",
    "#conda install pandas\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/hehouts/2019-fall-rotation-dib/sm_rotation_proj\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "mkdir -p github_souce_files\n",
    "cd github_souce_files\n",
    "\n",
    "git clone https://github.com/bluegenes/2018-test_datasets.git\n",
    "\n",
    "cp 2018-test_datasets/* .\n",
    "rm -r 2018-test_datasets "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "#This should be better than the below block, but is not tested.\n",
    "#Hypothetically, new organisms can be added willy nilly\n",
    "\n",
    "for org in bacteroides denticola gingivalis\n",
    "do\n",
    "    for mol in genomic protein  rna \n",
    "    do\n",
    "    \n",
    "        #Downloads seqs\n",
    "        python github_souce_files/download_genbank_datasets.py --genbank --protein --rna --subfolder github_souce_files/$org.csv\n",
    "\n",
    "        #This organizes folders\n",
    "        mv github_souce_files/$org.csv github_souce_files/$org/\n",
    "        mkdir -p $org\n",
    "        mv github_souce_files/$org/* $org\n",
    "        rmdir github_souce_files/$org/\n",
    "\n",
    "        cd $org\n",
    "        mkdir $mol/seqs\n",
    "        mv $mol/*.gz $mol/seqs\n",
    "        cd .. \n",
    "    done\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "#REDUNDANT\n",
    "\n",
    "python github_souce_files/download_genbank_datasets.py --genbank --protein --rna --subfolder github_souce_files/bacteroides.csv\n",
    "\n",
    "python github_souce_files/download_genbank_datasets.py --genbank --protein --rna --subfolder github_souce_files/denticola.csv\n",
    "\n",
    "python github_souce_files/download_genbank_datasets.py --genbank --protein --rna --subfolder github_souce_files/gingivalis.csv\n",
    " \n",
    " \n",
    "#This organizes folders\n",
    "mv github_souce_files/bacteroides.csv github_souce_files/bacteroides/\n",
    "mkdir -p bacteroides\n",
    "mv github_souce_files/bacteroides/* bacteroides\n",
    "rmdir github_souce_files/bacteroides/\n",
    "\n",
    "mv github_souce_files/denticola.csv github_souce_files/denticola\n",
    "mkdir denticola\n",
    "mv github_souce_files/denticola/* denticola\n",
    "rmdir github_souce_files/denticola/\n",
    "\n",
    "mv github_souce_files/gingivalis.csv github_souce_files/gingivalis\n",
    "mkdir gingivalis\n",
    "mv github_souce_files/gingivalis/* gingivalis\n",
    "rmdir github_souce_files/gingivalis/\n",
    "\n",
    "\n",
    "\n",
    "#This is to organize seqs\n",
    "for org in bacteroides denticola gingivalis\n",
    "    do\n",
    "    for mol in genomic protein  rna \n",
    "    do\n",
    "        cd $org\n",
    "        mkdir $mol/seqs\n",
    "        mv $mol/*a.gz $mol/seqs\n",
    "        cd .. \n",
    "    done\n",
    "done\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00108 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000007585.1_ASM758v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00191 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000007585.1_ASM758v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000007585.1_ASM758v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00210 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000007585.1_ASM758v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00120 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000010505.1_ASM1050v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00214 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000010505.1_ASM1050v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000010505.1_ASM1050v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00103 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000010505.1_ASM1050v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00124 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000064305.2_ASM6430v2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00134 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000064305.2_ASM6430v2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000064305.2_ASM6430v2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00127 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000064305.2_ASM6430v2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz 38 sequences\r",
      "\u001b[Kcalculated 1 signatures for 39 sequences in genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00263 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz 38 sequences\r",
      "\u001b[Kcalculated 1 signatures for 39 sequences in genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00216 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz 38 sequences\r",
      "\u001b[Kcalculated 1 signatures for 39 sequences in genomic/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00234 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz 60 sequences\r",
      "\u001b[Kcalculated 1 signatures for 61 sequences in genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00280 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000162415.1_ASM16241v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz 60 sequences\r",
      "\u001b[Kcalculated 1 signatures for 61 sequences in genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00187 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000162415.1_ASM16241v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz 60 sequences\r",
      "\u001b[Kcalculated 1 signatures for 61 sequences in genomic/seqs/GCA_000162415.1_ASM16241v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00296 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000162415.1_ASM16241v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz 249 sequences\r",
      "\u001b[Kcalculated 1 signatures for 250 sequences in genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00107 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000174775.1_ASM17477v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz 249 sequences\r",
      "\u001b[Kcalculated 1 signatures for 250 sequences in genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00101 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000174775.1_ASM17477v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz 249 sequences\r",
      "\u001b[Kcalculated 1 signatures for 250 sequences in genomic/seqs/GCA_000174775.1_ASM17477v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00122 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000174775.1_ASM17477v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00139 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000212375.1_ASM21237v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00096 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000212375.1_ASM21237v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000212375.1_ASM21237v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00113 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000212375.1_ASM21237v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz 462 sequences\r",
      "\u001b[Kcalculated 1 signatures for 463 sequences in genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00214 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000217375.2_CLC_glsol081_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz 462 sequences\r",
      "\u001b[Kcalculated 1 signatures for 463 sequences in genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00236 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000217375.2_CLC_glsol081_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz 462 sequences\r",
      "\u001b[Kcalculated 1 signatures for 463 sequences in genomic/seqs/GCA_000217375.2_CLC_glsol081_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00199 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000217375.2_CLC_glsol081_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz 11 sequences\r",
      "\u001b[Kcalculated 1 signatures for 12 sequences in genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00208 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz 11 sequences\r",
      "\u001b[Kcalculated 1 signatures for 12 sequences in genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00356 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz 11 sequences\r",
      "\u001b[Kcalculated 1 signatures for 12 sequences in genomic/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00209 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00212 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00108 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00172 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000380305.1_PgingivalisJCVISC001v1.0_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz 850 sequences\r",
      "\u001b[Kcalculated 1 signatures for 851 sequences in genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00065 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000417845.2_ASM41784v2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz 850 sequences\r",
      "\u001b[Kcalculated 1 signatures for 851 sequences in genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00060 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000417845.2_ASM41784v2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz 850 sequences\r",
      "\u001b[Kcalculated 1 signatures for 851 sequences in genomic/seqs/GCA_000417845.2_ASM41784v2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00055 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000417845.2_ASM41784v2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz 72 sequences\r",
      "\u001b[Kcalculated 1 signatures for 73 sequences in genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00150 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000428105.1_ASM42810v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz 72 sequences\r",
      "\u001b[Kcalculated 1 signatures for 73 sequences in genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00143 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000428105.1_ASM42810v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz 72 sequences\r",
      "\u001b[Kcalculated 1 signatures for 73 sequences in genomic/seqs/GCA_000428105.1_ASM42810v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00133 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000428105.1_ASM42810v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz 20 sequences\r",
      "\u001b[Kcalculated 1 signatures for 21 sequences in genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00137 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000439065.1_ASM43906v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz 20 sequences\r",
      "\u001b[Kcalculated 1 signatures for 21 sequences in genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00123 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000439065.1_ASM43906v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz 20 sequences\r",
      "\u001b[Kcalculated 1 signatures for 21 sequences in genomic/seqs/GCA_000439065.1_ASM43906v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00302 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000439065.1_ASM43906v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz 112 sequences\r",
      "\u001b[Kcalculated 1 signatures for 113 sequences in genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00114 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000467955.1_ASM46795v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz 112 sequences\r",
      "\u001b[Kcalculated 1 signatures for 113 sequences in genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00104 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000467955.1_ASM46795v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz 112 sequences\r",
      "\u001b[Kcalculated 1 signatures for 113 sequences in genomic/seqs/GCA_000467955.1_ASM46795v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00108 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000467955.1_ASM46795v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz 28 sequences\r",
      "\u001b[Kcalculated 1 signatures for 29 sequences in genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00128 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000482365.1_ASM48236v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz 28 sequences\r",
      "\u001b[Kcalculated 1 signatures for 29 sequences in genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00107 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000482365.1_ASM48236v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz 28 sequences\r",
      "\u001b[Kcalculated 1 signatures for 29 sequences in genomic/seqs/GCA_000482365.1_ASM48236v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00138 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000482365.1_ASM48236v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz 116 sequences\r",
      "\u001b[Kcalculated 1 signatures for 117 sequences in genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00150 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000503975.1_SJD2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz 116 sequences\r",
      "\u001b[Kcalculated 1 signatures for 117 sequences in genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00109 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000503975.1_SJD2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz 116 sequences\r",
      "\u001b[Kcalculated 1 signatures for 117 sequences in genomic/seqs/GCA_000503975.1_SJD2_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00160 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000503975.1_SJD2_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz 1290 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1291 sequences in genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00156 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000510525.1_BU063sc679_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz 1290 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1291 sequences in genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00148 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000510525.1_BU063sc679_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz 1290 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1291 sequences in genomic/seqs/GCA_000510525.1_BU063sc679_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00587 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000510525.1_BU063sc679_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz 1262 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1263 sequences in genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00187 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000510545.1_BU063sc13_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz 1262 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1263 sequences in genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00131 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000510545.1_BU063sc13_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz 1262 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1263 sequences in genomic/seqs/GCA_000510545.1_BU063sc13_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00116 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000510545.1_BU063sc13_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz 298 sequences\r",
      "\u001b[Kcalculated 1 signatures for 299 sequences in genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00298 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000583675.1_ASM58367v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz 298 sequences\r",
      "\u001b[Kcalculated 1 signatures for 299 sequences in genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00280 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000583675.1_ASM58367v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz 298 sequences\r",
      "\u001b[Kcalculated 1 signatures for 299 sequences in genomic/seqs/GCA_000583675.1_ASM58367v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00295 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000583675.1_ASM58367v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz 83 sequences\r",
      "\u001b[Kcalculated 1 signatures for 84 sequences in genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00254 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000599245.1_ASM59924v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz 83 sequences\r",
      "\u001b[Kcalculated 1 signatures for 84 sequences in genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00244 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000599245.1_ASM59924v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz 83 sequences\r",
      "\u001b[Kcalculated 1 signatures for 84 sequences in genomic/seqs/GCA_000599245.1_ASM59924v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00240 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000599245.1_ASM59924v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 1 signatures for 65 sequences in genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00254 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000599325.1_ASM59932v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 1 signatures for 65 sequences in genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00278 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000599325.1_ASM59932v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 1 signatures for 65 sequences in genomic/seqs/GCA_000599325.1_ASM59932v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00239 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000599325.1_ASM59932v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz 43 sequences\r",
      "\u001b[Kcalculated 1 signatures for 44 sequences in genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00120 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000614325.1_ASM61432v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz 43 sequences\r",
      "\u001b[Kcalculated 1 signatures for 44 sequences in genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00165 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000614325.1_ASM61432v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz 43 sequences\r",
      "\u001b[Kcalculated 1 signatures for 44 sequences in genomic/seqs/GCA_000614325.1_ASM61432v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00103 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000614325.1_ASM61432v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz 91 sequences\r",
      "\u001b[Kcalculated 1 signatures for 92 sequences in genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00266 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000614865.1_ASM61486v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz 91 sequences\r",
      "\u001b[Kcalculated 1 signatures for 92 sequences in genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00335 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000614865.1_ASM61486v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz 91 sequences\r",
      "\u001b[Kcalculated 1 signatures for 92 sequences in genomic/seqs/GCA_000614865.1_ASM61486v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00257 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000614865.1_ASM61486v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz 104 sequences\r",
      "\u001b[Kcalculated 1 signatures for 105 sequences in genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00283 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000614895.1_ASM61489v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz 104 sequences\r",
      "\u001b[Kcalculated 1 signatures for 105 sequences in genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00264 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000614895.1_ASM61489v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz 104 sequences\r",
      "\u001b[Kcalculated 1 signatures for 105 sequences in genomic/seqs/GCA_000614895.1_ASM61489v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00236 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000614895.1_ASM61489v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00344 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000626635.1_ASM62663v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00244 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000626635.1_ASM62663v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000626635.1_ASM62663v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00239 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000626635.1_ASM62663v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00138 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000739415.1_ASM73941v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00110 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000739415.1_ASM73941v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_000739415.1_ASM73941v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00113 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000739415.1_ASM73941v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz 30 sequences\r",
      "\u001b[Kcalculated 1 signatures for 31 sequences in genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00109 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000765945.1_ASM76594v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz 30 sequences\r",
      "\u001b[Kcalculated 1 signatures for 31 sequences in genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00117 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000765945.1_ASM76594v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz 30 sequences\r",
      "\u001b[Kcalculated 1 signatures for 31 sequences in genomic/seqs/GCA_000765945.1_ASM76594v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00172 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000765945.1_ASM76594v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz 29 sequences\r",
      "\u001b[Kcalculated 1 signatures for 30 sequences in genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00104 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000768975.1_ASM76897v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz 29 sequences\r",
      "\u001b[Kcalculated 1 signatures for 30 sequences in genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00106 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000768975.1_ASM76897v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz 29 sequences\r",
      "\u001b[Kcalculated 1 signatures for 30 sequences in genomic/seqs/GCA_000768975.1_ASM76897v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00090 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000768975.1_ASM76897v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz 46 sequences\r",
      "\u001b[Kcalculated 1 signatures for 47 sequences in genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00126 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000769015.1_ASM76901v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz 46 sequences\r",
      "\u001b[Kcalculated 1 signatures for 47 sequences in genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00133 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000769015.1_ASM76901v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz 46 sequences\r",
      "\u001b[Kcalculated 1 signatures for 47 sequences in genomic/seqs/GCA_000769015.1_ASM76901v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00210 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000769015.1_ASM76901v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz 31 sequences\r",
      "\u001b[Kcalculated 1 signatures for 32 sequences in genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00187 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000769035.1_ASM76903v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz 31 sequences\r",
      "\u001b[Kcalculated 1 signatures for 32 sequences in genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00129 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000769035.1_ASM76903v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz 31 sequences\r",
      "\u001b[Kcalculated 1 signatures for 32 sequences in genomic/seqs/GCA_000769035.1_ASM76903v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00169 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000769035.1_ASM76903v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz 18 sequences\r",
      "\u001b[Kcalculated 1 signatures for 19 sequences in genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00122 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000769135.1_ASM76913v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz 18 sequences\r",
      "\u001b[Kcalculated 1 signatures for 19 sequences in genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00094 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000769135.1_ASM76913v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz 18 sequences\r",
      "\u001b[Kcalculated 1 signatures for 19 sequences in genomic/seqs/GCA_000769135.1_ASM76913v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00124 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000769135.1_ASM76913v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz 335 sequences\r",
      "\u001b[Kcalculated 1 signatures for 336 sequences in genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00118 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000945515.1_Velvet_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz 335 sequences\r",
      "\u001b[Kcalculated 1 signatures for 336 sequences in genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00119 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000945515.1_Velvet_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz 335 sequences\r",
      "\u001b[Kcalculated 1 signatures for 336 sequences in genomic/seqs/GCA_000945515.1_Velvet_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00111 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000945515.1_Velvet_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz 18 sequences\r",
      "\u001b[Kcalculated 1 signatures for 19 sequences in genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00302 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000949475.1_ASM94947v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz 18 sequences\r",
      "\u001b[Kcalculated 1 signatures for 19 sequences in genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00285 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000949475.1_ASM94947v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz 18 sequences\r",
      "\u001b[Kcalculated 1 signatures for 19 sequences in genomic/seqs/GCA_000949475.1_ASM94947v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00236 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000949475.1_ASM94947v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 1 signatures for 33 sequences in genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000993115.1_ASM99311v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 1 signatures for 33 sequences in genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00051 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000993115.1_ASM99311v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 1 signatures for 33 sequences in genomic/seqs/GCA_000993115.1_ASM99311v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00156 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_000993115.1_ASM99311v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz 174 sequences\r",
      "\u001b[Kcalculated 1 signatures for 175 sequences in genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00340 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001049535.1_3731_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz 174 sequences\r",
      "\u001b[Kcalculated 1 signatures for 175 sequences in genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00281 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001049535.1_3731_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz 174 sequences\r",
      "\u001b[Kcalculated 1 signatures for 175 sequences in genomic/seqs/GCA_001049535.1_3731_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00554 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001049535.1_3731_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz 178 sequences\r",
      "\u001b[Kcalculated 1 signatures for 179 sequences in genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00179 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001064135.1_ASM106413v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz 178 sequences\r",
      "\u001b[Kcalculated 1 signatures for 179 sequences in genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00276 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001064135.1_ASM106413v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz 178 sequences\r",
      "\u001b[Kcalculated 1 signatures for 179 sequences in genomic/seqs/GCA_001064135.1_ASM106413v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00133 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001064135.1_ASM106413v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz 195 sequences\r",
      "\u001b[Kcalculated 1 signatures for 196 sequences in genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00226 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001261715.1_HGC4_v3_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz 195 sequences\r",
      "\u001b[Kcalculated 1 signatures for 196 sequences in genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00215 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001261715.1_HGC4_v3_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz 195 sequences\r",
      "\u001b[Kcalculated 1 signatures for 196 sequences in genomic/seqs/GCA_001261715.1_HGC4_v3_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00222 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001261715.1_HGC4_v3_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00125 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001263815.1_ASM126381v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00145 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001263815.1_ASM126381v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001263815.1_ASM126381v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00114 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001263815.1_ASM126381v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00209 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001274615.1_ASM127461v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00124 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001274615.1_ASM127461v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001274615.1_ASM127461v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00162 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001274615.1_ASM127461v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2 sequences in genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00147 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2 sequences in genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00123 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz 1 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2 sequences in genomic/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00132 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001282625.1_Lascolabacillus_massiliensis_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz 19 sequences\r",
      "\u001b[Kcalculated 1 signatures for 20 sequences in genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00209 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001404575.1_13470_2_67_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz 19 sequences\r",
      "\u001b[Kcalculated 1 signatures for 20 sequences in genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00200 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001404575.1_13470_2_67_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz 19 sequences\r",
      "\u001b[Kcalculated 1 signatures for 20 sequences in genomic/seqs/GCA_001404575.1_13470_2_67_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00190 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001404575.1_13470_2_67_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz 80 sequences\r",
      "\u001b[Kcalculated 1 signatures for 81 sequences in genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00231 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001406635.1_13470_2_63_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz 80 sequences\r",
      "\u001b[Kcalculated 1 signatures for 81 sequences in genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00213 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001406635.1_13470_2_63_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz 80 sequences\r",
      "\u001b[Kcalculated 1 signatures for 81 sequences in genomic/seqs/GCA_001406635.1_13470_2_63_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00196 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001406635.1_13470_2_63_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz 108 sequences\r",
      "\u001b[Kcalculated 1 signatures for 109 sequences in genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00107 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001438125.1_ASM143812v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz 108 sequences\r",
      "\u001b[Kcalculated 1 signatures for 109 sequences in genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00285 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001438125.1_ASM143812v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz 108 sequences\r",
      "\u001b[Kcalculated 1 signatures for 109 sequences in genomic/seqs/GCA_001438125.1_ASM143812v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00095 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001438125.1_ASM143812v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz 76 sequences\r",
      "\u001b[Kcalculated 1 signatures for 77 sequences in genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00215 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001438205.1_ASM143820v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz 76 sequences\r",
      "\u001b[Kcalculated 1 signatures for 77 sequences in genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00084 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001438205.1_ASM143820v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz 76 sequences\r",
      "\u001b[Kcalculated 1 signatures for 77 sequences in genomic/seqs/GCA_001438205.1_ASM143820v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00077 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001438205.1_ASM143820v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00224 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001443605.1_ASM144360v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00212 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001443605.1_ASM144360v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001443605.1_ASM144360v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00254 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001443605.1_ASM144360v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz 154 sequences\r",
      "\u001b[Kcalculated 1 signatures for 155 sequences in genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00237 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz 154 sequences\r",
      "\u001b[Kcalculated 1 signatures for 155 sequences in genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00217 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz 154 sequences\r",
      "\u001b[Kcalculated 1 signatures for 155 sequences in genomic/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00189 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz 91 sequences\r",
      "\u001b[Kcalculated 1 signatures for 92 sequences in genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00364 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001512095.1_7521_8_38_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz 91 sequences\r",
      "\u001b[Kcalculated 1 signatures for 92 sequences in genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00246 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001512095.1_7521_8_38_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz 91 sequences\r",
      "\u001b[Kcalculated 1 signatures for 92 sequences in genomic/seqs/GCA_001512095.1_7521_8_38_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00185 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001512095.1_7521_8_38_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz 492 sequences\r",
      "\u001b[Kcalculated 1 signatures for 493 sequences in genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00182 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001602775.1_ASM160277v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz 492 sequences\r",
      "\u001b[Kcalculated 1 signatures for 493 sequences in genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00135 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001602775.1_ASM160277v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz 492 sequences\r",
      "\u001b[Kcalculated 1 signatures for 493 sequences in genomic/seqs/GCA_001602775.1_ASM160277v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00107 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001602775.1_ASM160277v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz 300 sequences\r",
      "\u001b[Kcalculated 1 signatures for 301 sequences in genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00292 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001607025.1_ASM160702v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz 300 sequences\r",
      "\u001b[Kcalculated 1 signatures for 301 sequences in genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.08432 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001607025.1_ASM160702v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz 300 sequences\r",
      "\u001b[Kcalculated 1 signatures for 301 sequences in genomic/seqs/GCA_001607025.1_ASM160702v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00278 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001607025.1_ASM160702v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz 205 sequences\r",
      "\u001b[Kcalculated 1 signatures for 206 sequences in genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00298 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001637845.1_ASM163784v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz 205 sequences\r",
      "\u001b[Kcalculated 1 signatures for 206 sequences in genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00294 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001637845.1_ASM163784v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz 205 sequences\r",
      "\u001b[Kcalculated 1 signatures for 206 sequences in genomic/seqs/GCA_001637845.1_ASM163784v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00279 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001637845.1_ASM163784v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 1 signatures for 52 sequences in genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00332 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001670785.1_RCAD0183_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 1 signatures for 52 sequences in genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00103 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001670785.1_RCAD0183_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 1 signatures for 52 sequences in genomic/seqs/GCA_001670785.1_RCAD0183_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00109 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001670785.1_RCAD0183_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz 122 sequences\r",
      "\u001b[Kcalculated 1 signatures for 123 sequences in genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00116 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001724935.1_ASM172493v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz 122 sequences\r",
      "\u001b[Kcalculated 1 signatures for 123 sequences in genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00071 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001724935.1_ASM172493v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz 122 sequences\r",
      "\u001b[Kcalculated 1 signatures for 123 sequences in genomic/seqs/GCA_001724935.1_ASM172493v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00069 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001724935.1_ASM172493v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz 109 sequences\r",
      "\u001b[Kcalculated 1 signatures for 110 sequences in genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00259 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001768555.1_ASM176855v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz 109 sequences\r",
      "\u001b[Kcalculated 1 signatures for 110 sequences in genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00130 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001768555.1_ASM176855v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz 109 sequences\r",
      "\u001b[Kcalculated 1 signatures for 110 sequences in genomic/seqs/GCA_001768555.1_ASM176855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00166 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001768555.1_ASM176855v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz 102 sequences\r",
      "\u001b[Kcalculated 1 signatures for 103 sequences in genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00108 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001808555.1_ASM180855v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz 102 sequences\r",
      "\u001b[Kcalculated 1 signatures for 103 sequences in genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00093 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001808555.1_ASM180855v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz 102 sequences\r",
      "\u001b[Kcalculated 1 signatures for 103 sequences in genomic/seqs/GCA_001808555.1_ASM180855v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00099 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001808555.1_ASM180855v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz 87 sequences\r",
      "\u001b[Kcalculated 1 signatures for 88 sequences in genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00129 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001873105.1_ASM187310v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz 87 sequences\r",
      "\u001b[Kcalculated 1 signatures for 88 sequences in genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00117 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001873105.1_ASM187310v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz 87 sequences\r",
      "\u001b[Kcalculated 1 signatures for 88 sequences in genomic/seqs/GCA_001873105.1_ASM187310v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00128 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001873105.1_ASM187310v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00154 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001889045.1_ASM188904v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00153 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001889045.1_ASM188904v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in genomic/seqs/GCA_001889045.1_ASM188904v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00223 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001889045.1_ASM188904v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz 190 sequences\r",
      "\u001b[Kcalculated 1 signatures for 191 sequences in genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00110 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001896575.1_ASM189657v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz 190 sequences\r",
      "\u001b[Kcalculated 1 signatures for 191 sequences in genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00101 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001896575.1_ASM189657v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz 190 sequences\r",
      "\u001b[Kcalculated 1 signatures for 191 sequences in genomic/seqs/GCA_001896575.1_ASM189657v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00113 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001896575.1_ASM189657v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz 98 sequences\r",
      "\u001b[Kcalculated 1 signatures for 99 sequences in genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00281 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001897595.1_ASM189759v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz 98 sequences\r",
      "\u001b[Kcalculated 1 signatures for 99 sequences in genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00293 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001897595.1_ASM189759v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz 98 sequences\r",
      "\u001b[Kcalculated 1 signatures for 99 sequences in genomic/seqs/GCA_001897595.1_ASM189759v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00309 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001897595.1_ASM189759v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz 79 sequences\r",
      "\u001b[Kcalculated 1 signatures for 80 sequences in genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00194 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001898165.1_ASM189816v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz 79 sequences\r",
      "\u001b[Kcalculated 1 signatures for 80 sequences in genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00163 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001898165.1_ASM189816v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz 79 sequences\r",
      "\u001b[Kcalculated 1 signatures for 80 sequences in genomic/seqs/GCA_001898165.1_ASM189816v1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00196 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_001898165.1_ASM189816v1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz 23 sequences\r",
      "\u001b[Kcalculated 1 signatures for 24 sequences in genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00249 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900085525.1_12045_7_37_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz 23 sequences\r",
      "\u001b[Kcalculated 1 signatures for 24 sequences in genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00283 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900085525.1_12045_7_37_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz 23 sequences\r",
      "\u001b[Kcalculated 1 signatures for 24 sequences in genomic/seqs/GCA_900085525.1_12045_7_37_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00252 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900085525.1_12045_7_37_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz 67 sequences\r",
      "\u001b[Kcalculated 1 signatures for 68 sequences in genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00039 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900089535.1_A04_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz 67 sequences\r",
      "\u001b[Kcalculated 1 signatures for 68 sequences in genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00039 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900089535.1_A04_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz 67 sequences\r",
      "\u001b[Kcalculated 1 signatures for 68 sequences in genomic/seqs/GCA_900089535.1_A04_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00038 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900089535.1_A04_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz 93 sequences\r",
      "\u001b[Kcalculated 1 signatures for 94 sequences in genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00206 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz 93 sequences\r",
      "\u001b[Kcalculated 1 signatures for 94 sequences in genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00204 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz 93 sequences\r",
      "\u001b[Kcalculated 1 signatures for 94 sequences in genomic/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00190 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz 31 sequences\r",
      "\u001b[Kcalculated 1 signatures for 32 sequences in genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00178 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz 31 sequences\r",
      "\u001b[Kcalculated 1 signatures for 32 sequences in genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00192 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz 31 sequences\r",
      "\u001b[Kcalculated 1 signatures for 32 sequences in genomic/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00159 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 1 signatures for 33 sequences in genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00242 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 1 signatures for 33 sequences in genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00187 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 1 signatures for 33 sequences in genomic/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00189 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz 46 sequences\r",
      "\u001b[Kcalculated 1 signatures for 47 sequences in genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00224 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz 46 sequences\r",
      "\u001b[Kcalculated 1 signatures for 47 sequences in genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00226 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz 46 sequences\r",
      "\u001b[Kcalculated 1 signatures for 47 sequences in genomic/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00211 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00318 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00200 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in genomic/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00193 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz 80 sequences\r",
      "\u001b[Kcalculated 1 signatures for 81 sequences in genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00299 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz 80 sequences\r",
      "\u001b[Kcalculated 1 signatures for 81 sequences in genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00239 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz 80 sequences\r",
      "\u001b[Kcalculated 1 signatures for 81 sequences in genomic/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00289 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 1 signatures for 72 sequences in genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00104 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900157215.1_Strain_3-3_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 1 signatures for 72 sequences in genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00111 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900157215.1_Strain_3-3_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz 71 sequences\r",
      "\u001b[Kcalculated 1 signatures for 72 sequences in genomic/seqs/GCA_900157215.1_Strain_3-3_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00114 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900157215.1_Strain_3-3_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz 55 sequences\r",
      "\u001b[Kcalculated 1 signatures for 56 sequences in genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00115 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900157325.1_3A1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz 55 sequences\r",
      "\u001b[Kcalculated 1 signatures for 56 sequences in genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00101 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900157325.1_3A1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\r",
      "\u001b[K...genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz 55 sequences\r",
      "\u001b[Kcalculated 1 signatures for 56 sequences in genomic/seqs/GCA_900157325.1_3A1_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00102 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to genomic/sigs/GCA_900157325.1_3A1_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 1 signatures for 65 sequences in rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000007585.1_ASM758v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 1 signatures for 65 sequences in rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000007585.1_ASM758v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 1 signatures for 65 sequences in rna/seqs/GCA_000007585.1_ASM758v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000007585.1_ASM758v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 1 signatures for 65 sequences in rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000010505.1_ASM1050v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 1 signatures for 65 sequences in rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000010505.1_ASM1050v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz 64 sequences\r",
      "\u001b[Kcalculated 1 signatures for 65 sequences in rna/seqs/GCA_000010505.1_ASM1050v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000010505.1_ASM1050v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 1 signatures for 67 sequences in rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000064305.2_ASM6430v2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 1 signatures for 67 sequences in rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000064305.2_ASM6430v2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 1 signatures for 67 sequences in rna/seqs/GCA_000064305.2_ASM6430v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000064305.2_ASM6430v2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz 88 sequences\r",
      "\u001b[Kcalculated 1 signatures for 89 sequences in rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz 88 sequences\r",
      "\u001b[Kcalculated 1 signatures for 89 sequences in rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz 88 sequences\r",
      "\u001b[Kcalculated 1 signatures for 89 sequences in rna/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz 43 sequences\r",
      "\u001b[Kcalculated 1 signatures for 44 sequences in rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000162415.1_ASM16241v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz 43 sequences\r",
      "\u001b[Kcalculated 1 signatures for 44 sequences in rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000162415.1_ASM16241v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz 43 sequences\r",
      "\u001b[Kcalculated 1 signatures for 44 sequences in rna/seqs/GCA_000162415.1_ASM16241v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000162415.1_ASM16241v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 1 signatures for 52 sequences in rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000174775.1_ASM17477v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 1 signatures for 52 sequences in rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000174775.1_ASM17477v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 1 signatures for 52 sequences in rna/seqs/GCA_000174775.1_ASM17477v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000174775.1_ASM17477v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz 54 sequences\r",
      "\u001b[Kcalculated 1 signatures for 55 sequences in rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000212375.1_ASM21237v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz 54 sequences\r",
      "\u001b[Kcalculated 1 signatures for 55 sequences in rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000212375.1_ASM21237v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz 54 sequences\r",
      "\u001b[Kcalculated 1 signatures for 55 sequences in rna/seqs/GCA_000212375.1_ASM21237v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000212375.1_ASM21237v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000217375.2_CLC_glsol081_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000217375.2_CLC_glsol081_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000217375.2_CLC_glsol081_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000217375.2_CLC_glsol081_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000217375.2_CLC_glsol081_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000217375.2_CLC_glsol081_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000217375.2_CLC_glsol081_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000217375.2_CLC_glsol081_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000217375.2_CLC_glsol081_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz 99 sequences\r",
      "\u001b[Kcalculated 1 signatures for 100 sequences in rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz 99 sequences\r",
      "\u001b[Kcalculated 1 signatures for 100 sequences in rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz 99 sequences\r",
      "\u001b[Kcalculated 1 signatures for 100 sequences in rna/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000380305.1_PgingivalisJCVISC001v1.0_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz 40 sequences\r",
      "\u001b[Kcalculated 1 signatures for 41 sequences in rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000417845.2_ASM41784v2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz 40 sequences\r",
      "\u001b[Kcalculated 1 signatures for 41 sequences in rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000417845.2_ASM41784v2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz 40 sequences\r",
      "\u001b[Kcalculated 1 signatures for 41 sequences in rna/seqs/GCA_000417845.2_ASM41784v2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000417845.2_ASM41784v2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000428105.1_ASM42810v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000428105.1_ASM42810v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000428105.1_ASM42810v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000428105.1_ASM42810v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000428105.1_ASM42810v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000428105.1_ASM42810v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000428105.1_ASM42810v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000428105.1_ASM42810v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000428105.1_ASM42810v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000439065.1_ASM43906v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000439065.1_ASM43906v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000439065.1_ASM43906v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000439065.1_ASM43906v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000439065.1_ASM43906v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000439065.1_ASM43906v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000439065.1_ASM43906v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000439065.1_ASM43906v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000439065.1_ASM43906v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz 49 sequences\r",
      "\u001b[Kcalculated 1 signatures for 50 sequences in rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000467955.1_ASM46795v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz 49 sequences\r",
      "\u001b[Kcalculated 1 signatures for 50 sequences in rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000467955.1_ASM46795v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz 49 sequences\r",
      "\u001b[Kcalculated 1 signatures for 50 sequences in rna/seqs/GCA_000467955.1_ASM46795v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000467955.1_ASM46795v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000482365.1_ASM48236v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000482365.1_ASM48236v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000482365.1_ASM48236v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000482365.1_ASM48236v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000482365.1_ASM48236v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000482365.1_ASM48236v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000482365.1_ASM48236v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000482365.1_ASM48236v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000482365.1_ASM48236v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000503975.1_SJD2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000503975.1_SJD2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000503975.1_SJD2_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000503975.1_SJD2_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000510525.1_BU063sc679_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00181 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000510525.1_BU063sc679_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_000510525.1_BU063sc679_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00139 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000510525.1_BU063sc679_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz 54 sequences\r",
      "\u001b[Kcalculated 1 signatures for 55 sequences in rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000510545.1_BU063sc13_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz 54 sequences\r",
      "\u001b[Kcalculated 1 signatures for 55 sequences in rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000510545.1_BU063sc13_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz 54 sequences\r",
      "\u001b[Kcalculated 1 signatures for 55 sequences in rna/seqs/GCA_000510545.1_BU063sc13_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000510545.1_BU063sc13_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz 65 sequences\r",
      "\u001b[Kcalculated 1 signatures for 66 sequences in rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000583675.1_ASM58367v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz 65 sequences\r",
      "\u001b[Kcalculated 1 signatures for 66 sequences in rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000583675.1_ASM58367v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz 65 sequences\r",
      "\u001b[Kcalculated 1 signatures for 66 sequences in rna/seqs/GCA_000583675.1_ASM58367v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000583675.1_ASM58367v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz 89 sequences\r",
      "\u001b[Kcalculated 1 signatures for 90 sequences in rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000599245.1_ASM59924v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz 89 sequences\r",
      "\u001b[Kcalculated 1 signatures for 90 sequences in rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000599245.1_ASM59924v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz 89 sequences\r",
      "\u001b[Kcalculated 1 signatures for 90 sequences in rna/seqs/GCA_000599245.1_ASM59924v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000599245.1_ASM59924v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz 78 sequences\r",
      "\u001b[Kcalculated 1 signatures for 79 sequences in rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000599325.1_ASM59932v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz 78 sequences\r",
      "\u001b[Kcalculated 1 signatures for 79 sequences in rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000599325.1_ASM59932v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz 78 sequences\r",
      "\u001b[Kcalculated 1 signatures for 79 sequences in rna/seqs/GCA_000599325.1_ASM59932v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000599325.1_ASM59932v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000614325.1_ASM61432v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000614325.1_ASM61432v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000614325.1_ASM61432v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000614325.1_ASM61432v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000614325.1_ASM61432v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000614325.1_ASM61432v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000614325.1_ASM61432v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000614325.1_ASM61432v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000614325.1_ASM61432v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000614865.1_ASM61486v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000614865.1_ASM61486v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000614865.1_ASM61486v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000614865.1_ASM61486v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000614865.1_ASM61486v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000614865.1_ASM61486v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000614865.1_ASM61486v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000614865.1_ASM61486v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000614865.1_ASM61486v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000614895.1_ASM61489v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000614895.1_ASM61489v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000614895.1_ASM61489v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000614895.1_ASM61489v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000614895.1_ASM61489v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000614895.1_ASM61489v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000614895.1_ASM61489v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000614895.1_ASM61489v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000614895.1_ASM61489v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 1 signatures for 52 sequences in rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000626635.1_ASM62663v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 1 signatures for 52 sequences in rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000626635.1_ASM62663v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz 51 sequences\r",
      "\u001b[Kcalculated 1 signatures for 52 sequences in rna/seqs/GCA_000626635.1_ASM62663v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000626635.1_ASM62663v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz 65 sequences\r",
      "\u001b[Kcalculated 1 signatures for 66 sequences in rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000739415.1_ASM73941v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz 65 sequences\r",
      "\u001b[Kcalculated 1 signatures for 66 sequences in rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000739415.1_ASM73941v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz 65 sequences\r",
      "\u001b[Kcalculated 1 signatures for 66 sequences in rna/seqs/GCA_000739415.1_ASM73941v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000739415.1_ASM73941v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 1 signatures for 54 sequences in rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000765945.1_ASM76594v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 1 signatures for 54 sequences in rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000765945.1_ASM76594v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 1 signatures for 54 sequences in rna/seqs/GCA_000765945.1_ASM76594v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000765945.1_ASM76594v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 1 signatures for 54 sequences in rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000768975.1_ASM76897v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 1 signatures for 54 sequences in rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000768975.1_ASM76897v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 1 signatures for 54 sequences in rna/seqs/GCA_000768975.1_ASM76897v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000768975.1_ASM76897v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000769015.1_ASM76901v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000769015.1_ASM76901v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_000769015.1_ASM76901v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000769015.1_ASM76901v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000769035.1_ASM76903v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000769035.1_ASM76903v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000769035.1_ASM76903v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000769035.1_ASM76903v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz 47 sequences\r",
      "\u001b[Kcalculated 1 signatures for 48 sequences in rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000769135.1_ASM76913v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz 47 sequences\r",
      "\u001b[Kcalculated 1 signatures for 48 sequences in rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00024 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000769135.1_ASM76913v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz 47 sequences\r",
      "\u001b[Kcalculated 1 signatures for 48 sequences in rna/seqs/GCA_000769135.1_ASM76913v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00026 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000769135.1_ASM76913v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000945515.1_Velvet_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000945515.1_Velvet_assembly_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000945515.1_Velvet_assembly_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000945515.1_Velvet_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000945515.1_Velvet_assembly_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000945515.1_Velvet_assembly_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000945515.1_Velvet_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000945515.1_Velvet_assembly_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_000945515.1_Velvet_assembly_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000949475.1_ASM94947v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000949475.1_ASM94947v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_000949475.1_ASM94947v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000949475.1_ASM94947v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000993115.1_ASM99311v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000993115.1_ASM99311v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz 0 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1 sequences in rna/seqs/GCA_000993115.1_ASM99311v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_000993115.1_ASM99311v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001049535.1_3731_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001049535.1_3731_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001049535.1_3731_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001049535.1_3731_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001049535.1_3731_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001049535.1_3731_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001049535.1_3731_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001049535.1_3731_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001049535.1_3731_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001064135.1_ASM106413v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001064135.1_ASM106413v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001064135.1_ASM106413v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001064135.1_ASM106413v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001064135.1_ASM106413v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001064135.1_ASM106413v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001064135.1_ASM106413v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001064135.1_ASM106413v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001064135.1_ASM106413v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001261715.1_HGC4_v3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001261715.1_HGC4_v3_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001261715.1_HGC4_v3_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001261715.1_HGC4_v3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001261715.1_HGC4_v3_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001261715.1_HGC4_v3_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001261715.1_HGC4_v3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001261715.1_HGC4_v3_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001261715.1_HGC4_v3_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 1 signatures for 67 sequences in rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001263815.1_ASM126381v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 1 signatures for 67 sequences in rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001263815.1_ASM126381v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 1 signatures for 67 sequences in rna/seqs/GCA_001263815.1_ASM126381v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001263815.1_ASM126381v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 1 signatures for 67 sequences in rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001274615.1_ASM127461v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 1 signatures for 67 sequences in rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001274615.1_ASM127461v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz 66 sequences\r",
      "\u001b[Kcalculated 1 signatures for 67 sequences in rna/seqs/GCA_001274615.1_ASM127461v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001274615.1_ASM127461v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz 68 sequences\r",
      "\u001b[Kcalculated 1 signatures for 69 sequences in rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001404575.1_13470_2_67_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz 68 sequences\r",
      "\u001b[Kcalculated 1 signatures for 69 sequences in rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001404575.1_13470_2_67_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz 68 sequences\r",
      "\u001b[Kcalculated 1 signatures for 69 sequences in rna/seqs/GCA_001404575.1_13470_2_67_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001404575.1_13470_2_67_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz 56 sequences\r",
      "\u001b[Kcalculated 1 signatures for 57 sequences in rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001406635.1_13470_2_63_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz 56 sequences\r",
      "\u001b[Kcalculated 1 signatures for 57 sequences in rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001406635.1_13470_2_63_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz 56 sequences\r",
      "\u001b[Kcalculated 1 signatures for 57 sequences in rna/seqs/GCA_001406635.1_13470_2_63_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001406635.1_13470_2_63_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz 30 sequences\r",
      "\u001b[Kcalculated 1 signatures for 31 sequences in rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001438125.1_ASM143812v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz 30 sequences\r",
      "\u001b[Kcalculated 1 signatures for 31 sequences in rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001438125.1_ASM143812v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz 30 sequences\r",
      "\u001b[Kcalculated 1 signatures for 31 sequences in rna/seqs/GCA_001438125.1_ASM143812v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001438125.1_ASM143812v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz 36 sequences\r",
      "\u001b[Kcalculated 1 signatures for 37 sequences in rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001438205.1_ASM143820v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz 36 sequences\r",
      "\u001b[Kcalculated 1 signatures for 37 sequences in rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001438205.1_ASM143820v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz 36 sequences\r",
      "\u001b[Kcalculated 1 signatures for 37 sequences in rna/seqs/GCA_001438205.1_ASM143820v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001438205.1_ASM143820v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz 57 sequences\r",
      "\u001b[Kcalculated 1 signatures for 58 sequences in rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001443605.1_ASM144360v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz 57 sequences\r",
      "\u001b[Kcalculated 1 signatures for 58 sequences in rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001443605.1_ASM144360v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz 57 sequences\r",
      "\u001b[Kcalculated 1 signatures for 58 sequences in rna/seqs/GCA_001443605.1_ASM144360v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001443605.1_ASM144360v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz 95 sequences\r",
      "\u001b[Kcalculated 1 signatures for 96 sequences in rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz 95 sequences\r",
      "\u001b[Kcalculated 1 signatures for 96 sequences in rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz 95 sequences\r",
      "\u001b[Kcalculated 1 signatures for 96 sequences in rna/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001512095.1_7521_8_38_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001512095.1_7521_8_38_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001512095.1_7521_8_38_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001512095.1_7521_8_38_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001512095.1_7521_8_38_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001512095.1_7521_8_38_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001512095.1_7521_8_38_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001512095.1_7521_8_38_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001512095.1_7521_8_38_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001602775.1_ASM160277v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001602775.1_ASM160277v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001602775.1_ASM160277v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001602775.1_ASM160277v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001602775.1_ASM160277v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001602775.1_ASM160277v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001602775.1_ASM160277v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001602775.1_ASM160277v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001602775.1_ASM160277v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz 102 sequences\r",
      "\u001b[Kcalculated 1 signatures for 103 sequences in rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001607025.1_ASM160702v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz 102 sequences\r",
      "\u001b[Kcalculated 1 signatures for 103 sequences in rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001607025.1_ASM160702v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz 102 sequences\r",
      "\u001b[Kcalculated 1 signatures for 103 sequences in rna/seqs/GCA_001607025.1_ASM160702v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001607025.1_ASM160702v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001637845.1_ASM163784v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001637845.1_ASM163784v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001637845.1_ASM163784v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001637845.1_ASM163784v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001637845.1_ASM163784v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001637845.1_ASM163784v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001637845.1_ASM163784v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001637845.1_ASM163784v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001637845.1_ASM163784v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz 44 sequences\r",
      "\u001b[Kcalculated 1 signatures for 45 sequences in rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001670785.1_RCAD0183_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz 44 sequences\r",
      "\u001b[Kcalculated 1 signatures for 45 sequences in rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001670785.1_RCAD0183_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz 44 sequences\r",
      "\u001b[Kcalculated 1 signatures for 45 sequences in rna/seqs/GCA_001670785.1_RCAD0183_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001670785.1_RCAD0183_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz 5 sequences\r",
      "\u001b[Kcalculated 1 signatures for 6 sequences in rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001724935.1_ASM172493v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz 5 sequences\r",
      "\u001b[Kcalculated 1 signatures for 6 sequences in rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001724935.1_ASM172493v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz 5 sequences\r",
      "\u001b[Kcalculated 1 signatures for 6 sequences in rna/seqs/GCA_001724935.1_ASM172493v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001724935.1_ASM172493v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz 33 sequences\r",
      "\u001b[Kcalculated 1 signatures for 34 sequences in rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001768555.1_ASM176855v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz 33 sequences\r",
      "\u001b[Kcalculated 1 signatures for 34 sequences in rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001768555.1_ASM176855v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz 33 sequences\r",
      "\u001b[Kcalculated 1 signatures for 34 sequences in rna/seqs/GCA_001768555.1_ASM176855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001768555.1_ASM176855v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001808555.1_ASM180855v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001808555.1_ASM180855v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_001808555.1_ASM180855v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001808555.1_ASM180855v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 1 signatures for 33 sequences in rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001873105.1_ASM187310v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 1 signatures for 33 sequences in rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001873105.1_ASM187310v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz 32 sequences\r",
      "\u001b[Kcalculated 1 signatures for 33 sequences in rna/seqs/GCA_001873105.1_ASM187310v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001873105.1_ASM187310v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001889045.1_ASM188904v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001889045.1_ASM188904v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_001889045.1_ASM188904v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001889045.1_ASM188904v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001896575.1_ASM189657v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001896575.1_ASM189657v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001896575.1_ASM189657v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001896575.1_ASM189657v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001896575.1_ASM189657v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001896575.1_ASM189657v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001896575.1_ASM189657v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001896575.1_ASM189657v1_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_001896575.1_ASM189657v1_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz 60 sequences\r",
      "\u001b[Kcalculated 1 signatures for 61 sequences in rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001897595.1_ASM189759v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz 60 sequences\r",
      "\u001b[Kcalculated 1 signatures for 61 sequences in rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001897595.1_ASM189759v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz 60 sequences\r",
      "\u001b[Kcalculated 1 signatures for 61 sequences in rna/seqs/GCA_001897595.1_ASM189759v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001897595.1_ASM189759v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz 46 sequences\r",
      "\u001b[Kcalculated 1 signatures for 47 sequences in rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001898165.1_ASM189816v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz 46 sequences\r",
      "\u001b[Kcalculated 1 signatures for 47 sequences in rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001898165.1_ASM189816v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz 46 sequences\r",
      "\u001b[Kcalculated 1 signatures for 47 sequences in rna/seqs/GCA_001898165.1_ASM189816v1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_001898165.1_ASM189816v1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz 70 sequences\r",
      "\u001b[Kcalculated 1 signatures for 71 sequences in rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900085525.1_12045_7_37_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz 70 sequences\r",
      "\u001b[Kcalculated 1 signatures for 71 sequences in rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900085525.1_12045_7_37_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz 70 sequences\r",
      "\u001b[Kcalculated 1 signatures for 71 sequences in rna/seqs/GCA_900085525.1_12045_7_37_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900085525.1_12045_7_37_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900089535.1_A04_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900089535.1_A04_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_900089535.1_A04_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900089535.1_A04_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900089535.1_A04_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_900089535.1_A04_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900089535.1_A04_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900089535.1_A04_rna_from_genomic.fna.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'rna/seqs/GCA_900089535.1_A04_rna_from_genomic.fna.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz 70 sequences\r",
      "\u001b[Kcalculated 1 signatures for 71 sequences in rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz 70 sequences\r",
      "\u001b[Kcalculated 1 signatures for 71 sequences in rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz 70 sequences\r",
      "\u001b[Kcalculated 1 signatures for 71 sequences in rna/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz 59 sequences\r",
      "\u001b[Kcalculated 1 signatures for 60 sequences in rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz 59 sequences\r",
      "\u001b[Kcalculated 1 signatures for 60 sequences in rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz 59 sequences\r",
      "\u001b[Kcalculated 1 signatures for 60 sequences in rna/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz 131 sequences\r",
      "\u001b[Kcalculated 1 signatures for 132 sequences in rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz 131 sequences\r",
      "\u001b[Kcalculated 1 signatures for 132 sequences in rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz 131 sequences\r",
      "\u001b[Kcalculated 1 signatures for 132 sequences in rna/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz 56 sequences\r",
      "\u001b[Kcalculated 1 signatures for 57 sequences in rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz 56 sequences\r",
      "\u001b[Kcalculated 1 signatures for 57 sequences in rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz 56 sequences\r",
      "\u001b[Kcalculated 1 signatures for 57 sequences in rna/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz 52 sequences\r",
      "\u001b[Kcalculated 1 signatures for 53 sequences in rna/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 1 signatures for 54 sequences in rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 1 signatures for 54 sequences in rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz 53 sequences\r",
      "\u001b[Kcalculated 1 signatures for 54 sequences in rna/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00009 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900157215.1_Strain_3-3_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900157215.1_Strain_3-3_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_900157215.1_Strain_3-3_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900157215.1_Strain_3-3_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [21]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900157325.1_3A1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [31]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900157325.1_3A1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [51]\n",
      "\r",
      "\u001b[KComputing only nucleotide (and not protein) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[K...rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz 50 sequences\r",
      "\u001b[Kcalculated 1 signatures for 51 sequences in rna/seqs/GCA_900157325.1_3A1_rna_from_genomic.fna.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to rna/sigs/GCA_900157325.1_3A1_rna_from_genomic.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz 1908 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1909 sequences in protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000007585.1_ASM758v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz 1908 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1909 sequences in protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000007585.1_ASM758v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz 1908 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1909 sequences in protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00039 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000007585.1_ASM758v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz 2089 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2090 sequences in protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000010505.1_ASM1050v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz 2089 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2090 sequences in protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000010505.1_ASM1050v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz 2089 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2090 sequences in protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000010505.1_ASM1050v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz 2445 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2446 sequences in protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000064305.2_ASM6430v2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz 2445 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2446 sequences in protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000064305.2_ASM6430v2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz 2445 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2446 sequences in protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000064305.2_ASM6430v2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz 4705 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4706 sequences in protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz 4705 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4706 sequences in protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00023 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz 4705 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4706 sequences in protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00063 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz 2925 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2926 sequences in protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000162415.1_ASM16241v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz 2925 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2926 sequences in protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000162415.1_ASM16241v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz 2925 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2926 sequences in protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00081 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000162415.1_ASM16241v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz 1976 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1977 sequences in protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000174775.1_ASM17477v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz 1976 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1977 sequences in protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000174775.1_ASM17477v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz 1976 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1977 sequences in protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000174775.1_ASM17477v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz 1698 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1699 sequences in protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000212375.1_ASM21237v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz 1698 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1699 sequences in protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000212375.1_ASM21237v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz 1698 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1699 sequences in protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000212375.1_ASM21237v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz 4040 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4041 sequences in protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz 4040 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4041 sequences in protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz 4040 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4041 sequences in protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00071 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz 2353 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2354 sequences in protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz 2353 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2354 sequences in protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz 2353 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2354 sequences in protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz 3208 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3209 sequences in protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000417845.2_ASM41784v2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz 3208 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3209 sequences in protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000417845.2_ASM41784v2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz 3208 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3209 sequences in protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000417845.2_ASM41784v2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2236 sequences in protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000467955.1_ASM46795v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2236 sequences in protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000467955.1_ASM46795v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2236 sequences in protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000467955.1_ASM46795v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz 2019 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2020 sequences in protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000503975.1_SJD2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz 2019 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2020 sequences in protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000503975.1_SJD2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz 2019 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2020 sequences in protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000503975.1_SJD2_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz 2934 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2935 sequences in protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000510525.1_BU063sc679_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz 2934 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2935 sequences in protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000510525.1_BU063sc679_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz 2934 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2935 sequences in protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00038 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000510525.1_BU063sc679_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz 2370 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2371 sequences in protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00026 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000510545.1_BU063sc13_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz 2370 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2371 sequences in protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000510545.1_BU063sc13_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz 2370 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2371 sequences in protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000510545.1_BU063sc13_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz 4662 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4663 sequences in protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000583675.1_ASM58367v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz 4662 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4663 sequences in protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000583675.1_ASM58367v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz 4662 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4663 sequences in protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00218 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000583675.1_ASM58367v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz 4667 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4668 sequences in protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000599245.1_ASM59924v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz 4667 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4668 sequences in protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000599245.1_ASM59924v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz 4667 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4668 sequences in protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00119 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000599245.1_ASM59924v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz 4637 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4638 sequences in protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000599325.1_ASM59932v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz 4637 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4638 sequences in protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000599325.1_ASM59932v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz 4637 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4638 sequences in protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000599325.1_ASM59932v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz 3970 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3971 sequences in protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz 3970 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3971 sequences in protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz 3970 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3971 sequences in protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00059 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz 1957 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1958 sequences in protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000739415.1_ASM73941v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz 1957 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1958 sequences in protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000739415.1_ASM73941v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz 1957 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1958 sequences in protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000739415.1_ASM73941v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz 1903 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1904 sequences in protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000765945.1_ASM76594v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz 1903 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1904 sequences in protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00025 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000765945.1_ASM76594v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz 1903 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1904 sequences in protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00076 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000765945.1_ASM76594v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz 1606 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1607 sequences in protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000768975.1_ASM76897v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz 1606 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1607 sequences in protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000768975.1_ASM76897v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz 1606 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1607 sequences in protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00031 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000768975.1_ASM76897v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz 1839 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1840 sequences in protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00030 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000769015.1_ASM76901v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz 1839 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1840 sequences in protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000769015.1_ASM76901v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz 1839 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1840 sequences in protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00078 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000769015.1_ASM76901v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz 1842 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1843 sequences in protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000769035.1_ASM76903v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz 1842 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1843 sequences in protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000769035.1_ASM76903v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz 1842 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1843 sequences in protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000769035.1_ASM76903v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz 1432 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1433 sequences in protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000769135.1_ASM76913v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz 1432 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1433 sequences in protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000769135.1_ASM76913v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz 1432 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1433 sequences in protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00028 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000769135.1_ASM76913v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz 4125 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4126 sequences in protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000949475.1_ASM94947v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz 4125 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4126 sequences in protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000949475.1_ASM94947v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz 4125 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4126 sequences in protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00089 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000949475.1_ASM94947v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz 977 sequences\r",
      "\u001b[Kcalculated 1 signatures for 978 sequences in protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000993115.1_ASM99311v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz 977 sequences\r",
      "\u001b[Kcalculated 1 signatures for 978 sequences in protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000993115.1_ASM99311v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz 977 sequences\r",
      "\u001b[Kcalculated 1 signatures for 978 sequences in protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_000993115.1_ASM99311v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz 2003 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2004 sequences in protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001263815.1_ASM126381v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz 2003 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2004 sequences in protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001263815.1_ASM126381v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz 2003 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2004 sequences in protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001263815.1_ASM126381v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz 2001 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2002 sequences in protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001274615.1_ASM127461v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz 2001 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2002 sequences in protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00026 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001274615.1_ASM127461v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz 2001 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2002 sequences in protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00780 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001274615.1_ASM127461v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz 3544 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3545 sequences in protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001404575.1_13470_2_67_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz 3544 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3545 sequences in protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00031 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001404575.1_13470_2_67_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz 3544 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3545 sequences in protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00089 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001404575.1_13470_2_67_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz 3812 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3813 sequences in protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001406635.1_13470_2_63_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz 3812 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3813 sequences in protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001406635.1_13470_2_63_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz 3812 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3813 sequences in protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00065 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001406635.1_13470_2_63_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz 1661 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1662 sequences in protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001438125.1_ASM143812v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz 1661 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1662 sequences in protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00023 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001438125.1_ASM143812v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz 1661 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1662 sequences in protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00049 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001438125.1_ASM143812v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz 1494 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1495 sequences in protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001438205.1_ASM143820v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz 1494 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1495 sequences in protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001438205.1_ASM143820v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz 1494 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1495 sequences in protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00102 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001438205.1_ASM143820v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz 3864 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3865 sequences in protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz 3864 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3865 sequences in protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz 3864 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3865 sequences in protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00060 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz 4588 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4589 sequences in protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz 4588 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4589 sequences in protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz 4588 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4589 sequences in protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00057 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz 5572 sequences\r",
      "\u001b[Kcalculated 1 signatures for 5573 sequences in protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001607025.1_ASM160702v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz 5572 sequences\r",
      "\u001b[Kcalculated 1 signatures for 5573 sequences in protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001607025.1_ASM160702v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz 5572 sequences\r",
      "\u001b[Kcalculated 1 signatures for 5573 sequences in protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001607025.1_ASM160702v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz 2016 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2017 sequences in protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001670785.1_RCAD0183_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz 2016 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2017 sequences in protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001670785.1_RCAD0183_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz 2016 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2017 sequences in protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00039 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001670785.1_RCAD0183_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz 897 sequences\r",
      "\u001b[Kcalculated 1 signatures for 898 sequences in protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001724935.1_ASM172493v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz 897 sequences\r",
      "\u001b[Kcalculated 1 signatures for 898 sequences in protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001724935.1_ASM172493v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz 897 sequences\r",
      "\u001b[Kcalculated 1 signatures for 898 sequences in protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00028 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001724935.1_ASM172493v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz 1958 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1959 sequences in protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001768555.1_ASM176855v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz 1958 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1959 sequences in protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001768555.1_ASM176855v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz 1958 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1959 sequences in protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001768555.1_ASM176855v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz 1614 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1615 sequences in protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001808555.1_ASM180855v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz 1614 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1615 sequences in protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001808555.1_ASM180855v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz 1614 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1615 sequences in protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001808555.1_ASM180855v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz 2281 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2282 sequences in protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001873105.1_ASM187310v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz 2281 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2282 sequences in protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001873105.1_ASM187310v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz 2281 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2282 sequences in protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001873105.1_ASM187310v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz 2660 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2661 sequences in protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001889045.1_ASM188904v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz 2660 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2661 sequences in protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001889045.1_ASM188904v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz 2660 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2661 sequences in protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00068 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001889045.1_ASM188904v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz 4207 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4208 sequences in protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001897595.1_ASM189759v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz 4207 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4208 sequences in protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001897595.1_ASM189759v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz 4207 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4208 sequences in protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00064 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001897595.1_ASM189759v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz 2941 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2942 sequences in protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001898165.1_ASM189816v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz 2941 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2942 sequences in protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001898165.1_ASM189816v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz 2941 sequences\r",
      "\u001b[Kcalculated 1 signatures for 2942 sequences in protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00043 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_001898165.1_ASM189816v1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz 4960 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4961 sequences in protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900085525.1_12045_7_37_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz 4960 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4961 sequences in protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900085525.1_12045_7_37_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz 4960 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4961 sequences in protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00059 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900085525.1_12045_7_37_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900089535.1_A04_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900089535.1_A04_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 567, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900089535.1_A04_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz 3333 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3334 sequences in protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz 3333 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3334 sequences in protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz 3333 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3334 sequences in protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00060 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz 3559 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3560 sequences in protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz 3559 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3560 sequences in protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz 3559 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3560 sequences in protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00058 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz 3924 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3925 sequences in protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz 3924 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3925 sequences in protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00027 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz 3924 sequences\r",
      "\u001b[Kcalculated 1 signatures for 3925 sequences in protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00182 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz 4004 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4005 sequences in protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz 4004 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4005 sequences in protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz 4004 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4005 sequences in protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00062 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz 4014 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4015 sequences in protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz 4014 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4015 sequences in protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz 4014 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4015 sequences in protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00059 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz 4108 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4109 sequences in protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00010 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz 4108 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4109 sequences in protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz 4108 sequences\r",
      "\u001b[Kcalculated 1 signatures for 4109 sequences in protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00059 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz 1778 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1779 sequences in protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900157215.1_Strain_3-3_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz 1778 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1779 sequences in protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00024 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900157215.1_Strain_3-3_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz 1778 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1779 sequences in protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00060 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900157215.1_Strain_3-3_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157325.1_3A1_protein.faa.gz 1845 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1846 sequences in protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900157325.1_3A1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157325.1_3A1_protein.faa.gz 1845 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1846 sequences in protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900157325.1_3A1_protein.sig. Note: signature license is CC0.\r",
      "\r",
      "\u001b[K== This is sourmash version 2.2.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157325.1_3A1_protein.faa.gz 1845 sequences\r",
      "\u001b[Kcalculated 1 signatures for 1846 sequences in protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00033 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/GCA_900157325.1_3A1_protein.sig. Note: signature license is CC0.\r"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "#Calc signatures\n",
    "\n",
    "\n",
    "for org in gingivalis\n",
    "\n",
    "#bacteroides denticola \n",
    "do\n",
    "    for mol in genomic rna \n",
    "    do\n",
    "        cd $org\n",
    "        mkdir -p $mol/sigs\n",
    "        for infile in $mol/seqs/*a.gz \n",
    "          do \n",
    "            for k in 21 31 51\n",
    "              do\n",
    "\n",
    "                out_name=$(basename $infile .fna.gz)\n",
    "                sourmash compute -k $k --scaled 2000 --track-abundance -o $mol/sigs/${out_name}.sig ${infile}\n",
    "\n",
    "                #back to relative home dir, overwrite and append this file\n",
    "\n",
    "             done\n",
    "          done\n",
    "        cd ..\n",
    "    done\n",
    "    \n",
    "    \n",
    "    for mol in protein\n",
    "    do\n",
    "\n",
    "        cd $org\n",
    "        mkdir -p $mol/sigs\n",
    "        for infile in $mol/seqs/*a.gz \n",
    "        do \n",
    "        for k in 7 12 15\n",
    "        do\n",
    "            out_name=$(basename $infile .faa.gz)\n",
    "            sourmash compute --input-is-protein -k $k --scaled 2000 --track-abundance -o $mol/sigs/${out_name}.sig ${infile}\n",
    "\n",
    "        done\n",
    "        done\n",
    "        cd ..\n",
    "    done\n",
    "   \n",
    "   \n",
    "done\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz 1908 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1909 sequences in protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000007585.1_ASM758v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz 1908 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1909 sequences in protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00078 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000007585.1_ASM758v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz 1908 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1909 sequences in protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000007585.1_ASM758v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz 1908 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1909 sequences in protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000007585.1_ASM758v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz 1908 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1909 sequences in protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00033 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000007585.1_ASM758v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz 1908 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1909 sequences in protein/seqs/GCA_000007585.1_ASM758v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00033 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000007585.1_ASM758v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz 2089 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2090 sequences in protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000010505.1_ASM1050v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz 2089 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2090 sequences in protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000010505.1_ASM1050v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz 2089 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2090 sequences in protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000010505.1_ASM1050v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz 2089 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2090 sequences in protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000010505.1_ASM1050v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz 2089 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2090 sequences in protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000010505.1_ASM1050v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz 2089 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2090 sequences in protein/seqs/GCA_000010505.1_ASM1050v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000010505.1_ASM1050v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz 2445 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2446 sequences in protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00029 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000064305.2_ASM6430v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz 2445 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2446 sequences in protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000064305.2_ASM6430v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz 2445 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2446 sequences in protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000064305.2_ASM6430v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz 2445 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2446 sequences in protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000064305.2_ASM6430v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz 2445 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2446 sequences in protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00073 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000064305.2_ASM6430v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz 2445 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2446 sequences in protein/seqs/GCA_000064305.2_ASM6430v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00038 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000064305.2_ASM6430v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz 4705 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4706 sequences in protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz 4705 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4706 sequences in protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz 4705 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4706 sequences in protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz 4705 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4706 sequences in protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz 4705 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4706 sequences in protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00064 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz 4705 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4706 sequences in protein/seqs/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00073 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000158515.2_Bacteroides_sp_4_3_47FAA_V2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz 2925 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2926 sequences in protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000162415.1_ASM16241v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz 2925 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2926 sequences in protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000162415.1_ASM16241v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz 2925 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2926 sequences in protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00038 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000162415.1_ASM16241v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz 2925 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2926 sequences in protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000162415.1_ASM16241v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz 2925 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2926 sequences in protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00045 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000162415.1_ASM16241v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz 2925 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2926 sequences in protein/seqs/GCA_000162415.1_ASM16241v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00069 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000162415.1_ASM16241v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz 1976 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1977 sequences in protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000174775.1_ASM17477v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz 1976 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1977 sequences in protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000174775.1_ASM17477v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz 1976 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1977 sequences in protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000174775.1_ASM17477v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz 1976 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1977 sequences in protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000174775.1_ASM17477v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz 1976 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1977 sequences in protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000174775.1_ASM17477v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz 1976 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1977 sequences in protein/seqs/GCA_000174775.1_ASM17477v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000174775.1_ASM17477v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz 1698 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1699 sequences in protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000212375.1_ASM21237v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz 1698 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1699 sequences in protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000212375.1_ASM21237v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz 1698 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1699 sequences in protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00027 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000212375.1_ASM21237v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz 1698 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1699 sequences in protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00025 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000212375.1_ASM21237v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz 1698 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1699 sequences in protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00051 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000212375.1_ASM21237v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz 1698 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1699 sequences in protein/seqs/GCA_000212375.1_ASM21237v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000212375.1_ASM21237v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000217375.2_CLC_glsol081_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz 4040 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4041 sequences in protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz 4040 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4041 sequences in protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz 4040 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4041 sequences in protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00023 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz 4040 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4041 sequences in protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00026 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz 4040 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4041 sequences in protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00064 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz 4040 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4041 sequences in protein/seqs/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00062 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000273295.1_Bact_vulg_CL09T03C04_V1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz 2353 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2354 sequences in protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz 2353 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2354 sequences in protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz 2353 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2354 sequences in protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz 2353 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2354 sequences in protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00031 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz 2353 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2354 sequences in protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00056 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz 2353 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2354 sequences in protein/seqs/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000380305.1_PgingivalisJCVISC001v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz 3208 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3209 sequences in protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000417845.2_ASM41784v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz 3208 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3209 sequences in protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000417845.2_ASM41784v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz 3208 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3209 sequences in protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000417845.2_ASM41784v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz 3208 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3209 sequences in protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000417845.2_ASM41784v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz 3208 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3209 sequences in protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00024 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000417845.2_ASM41784v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz 3208 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3209 sequences in protein/seqs/GCA_000417845.2_ASM41784v2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000417845.2_ASM41784v2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000428105.1_ASM42810v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000439065.1_ASM43906v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2236 sequences in protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000467955.1_ASM46795v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2236 sequences in protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000467955.1_ASM46795v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2236 sequences in protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000467955.1_ASM46795v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2236 sequences in protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00025 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000467955.1_ASM46795v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2236 sequences in protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000467955.1_ASM46795v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz 2235 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2236 sequences in protein/seqs/GCA_000467955.1_ASM46795v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000467955.1_ASM46795v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000482365.1_ASM48236v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz 2019 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2020 sequences in protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000503975.1_SJD2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz 2019 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2020 sequences in protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000503975.1_SJD2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz 2019 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2020 sequences in protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000503975.1_SJD2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz 2019 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2020 sequences in protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000503975.1_SJD2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz 2019 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2020 sequences in protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000503975.1_SJD2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz 2019 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2020 sequences in protein/seqs/GCA_000503975.1_SJD2_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000503975.1_SJD2_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz 2934 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2935 sequences in protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000510525.1_BU063sc679_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz 2934 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2935 sequences in protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000510525.1_BU063sc679_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz 2934 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2935 sequences in protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000510525.1_BU063sc679_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz 2934 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2935 sequences in protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00024 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000510525.1_BU063sc679_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz 2934 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2935 sequences in protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00042 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000510525.1_BU063sc679_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz 2934 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2935 sequences in protein/seqs/GCA_000510525.1_BU063sc679_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00042 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000510525.1_BU063sc679_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz 2370 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2371 sequences in protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000510545.1_BU063sc13_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz 2370 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2371 sequences in protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000510545.1_BU063sc13_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz 2370 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2371 sequences in protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000510545.1_BU063sc13_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz 2370 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2371 sequences in protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000510545.1_BU063sc13_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz 2370 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2371 sequences in protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000510545.1_BU063sc13_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz 2370 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2371 sequences in protein/seqs/GCA_000510545.1_BU063sc13_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00033 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000510545.1_BU063sc13_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz 4662 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4663 sequences in protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000583675.1_ASM58367v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz 4662 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4663 sequences in protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000583675.1_ASM58367v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz 4662 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4663 sequences in protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000583675.1_ASM58367v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz 4662 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4663 sequences in protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00029 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000583675.1_ASM58367v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz 4662 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4663 sequences in protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00065 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000583675.1_ASM58367v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz 4662 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4663 sequences in protein/seqs/GCA_000583675.1_ASM58367v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00066 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000583675.1_ASM58367v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz 4667 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4668 sequences in protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000599245.1_ASM59924v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz 4667 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4668 sequences in protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000599245.1_ASM59924v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz 4667 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4668 sequences in protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000599245.1_ASM59924v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz 4667 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4668 sequences in protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000599245.1_ASM59924v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz 4667 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4668 sequences in protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00062 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000599245.1_ASM59924v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz 4667 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4668 sequences in protein/seqs/GCA_000599245.1_ASM59924v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00064 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000599245.1_ASM59924v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz 4637 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4638 sequences in protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000599325.1_ASM59932v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz 4637 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4638 sequences in protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000599325.1_ASM59932v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz 4637 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4638 sequences in protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000599325.1_ASM59932v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz 4637 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4638 sequences in protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000599325.1_ASM59932v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz 4637 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4638 sequences in protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00068 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000599325.1_ASM59932v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz 4637 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4638 sequences in protein/seqs/GCA_000599325.1_ASM59932v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000599325.1_ASM59932v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614325.1_ASM61432v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614865.1_ASM61486v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000614895.1_ASM61489v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz 3970 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3971 sequences in protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz 3970 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3971 sequences in protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz 3970 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3971 sequences in protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz 3970 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3971 sequences in protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz 3970 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3971 sequences in protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00071 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz 3970 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3971 sequences in protein/seqs/GCA_000626635.1_ASM62663v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00062 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000626635.1_ASM62663v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz 1957 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1958 sequences in protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000739415.1_ASM73941v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz 1957 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1958 sequences in protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000739415.1_ASM73941v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz 1957 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1958 sequences in protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000739415.1_ASM73941v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz 1957 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1958 sequences in protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00027 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000739415.1_ASM73941v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz 1957 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1958 sequences in protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000739415.1_ASM73941v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz 1957 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1958 sequences in protein/seqs/GCA_000739415.1_ASM73941v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000739415.1_ASM73941v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz 1903 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1904 sequences in protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000765945.1_ASM76594v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz 1903 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1904 sequences in protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000765945.1_ASM76594v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz 1903 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1904 sequences in protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000765945.1_ASM76594v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz 1903 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1904 sequences in protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000765945.1_ASM76594v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz 1903 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1904 sequences in protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000765945.1_ASM76594v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz 1903 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1904 sequences in protein/seqs/GCA_000765945.1_ASM76594v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000765945.1_ASM76594v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz 1606 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1607 sequences in protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000768975.1_ASM76897v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz 1606 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1607 sequences in protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000768975.1_ASM76897v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz 1606 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1607 sequences in protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00025 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000768975.1_ASM76897v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz 1606 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1607 sequences in protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000768975.1_ASM76897v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz 1606 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1607 sequences in protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000768975.1_ASM76897v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz 1606 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1607 sequences in protein/seqs/GCA_000768975.1_ASM76897v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00033 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000768975.1_ASM76897v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz 1839 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1840 sequences in protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000769015.1_ASM76901v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz 1839 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1840 sequences in protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000769015.1_ASM76901v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz 1839 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1840 sequences in protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000769015.1_ASM76901v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz 1839 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1840 sequences in protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000769015.1_ASM76901v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz 1839 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1840 sequences in protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000769015.1_ASM76901v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz 1839 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1840 sequences in protein/seqs/GCA_000769015.1_ASM76901v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000769015.1_ASM76901v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz 1842 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1843 sequences in protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000769035.1_ASM76903v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz 1842 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1843 sequences in protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000769035.1_ASM76903v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz 1842 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1843 sequences in protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000769035.1_ASM76903v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz 1842 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1843 sequences in protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000769035.1_ASM76903v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz 1842 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1843 sequences in protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00039 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000769035.1_ASM76903v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz 1842 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1843 sequences in protein/seqs/GCA_000769035.1_ASM76903v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000769035.1_ASM76903v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz 1432 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1433 sequences in protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000769135.1_ASM76913v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz 1432 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1433 sequences in protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000769135.1_ASM76913v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz 1432 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1433 sequences in protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00027 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000769135.1_ASM76913v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz 1432 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1433 sequences in protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000769135.1_ASM76913v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz 1432 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1433 sequences in protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000769135.1_ASM76913v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz 1432 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1433 sequences in protein/seqs/GCA_000769135.1_ASM76913v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00030 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000769135.1_ASM76913v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_000945515.1_Velvet_assembly_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz 4125 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4126 sequences in protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000949475.1_ASM94947v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz 4125 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4126 sequences in protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000949475.1_ASM94947v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz 4125 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4126 sequences in protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000949475.1_ASM94947v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz 4125 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4126 sequences in protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000949475.1_ASM94947v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz 4125 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4126 sequences in protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00072 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000949475.1_ASM94947v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz 4125 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4126 sequences in protein/seqs/GCA_000949475.1_ASM94947v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00065 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000949475.1_ASM94947v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz 977 sequences\r",
      "\u001b[Kcalculated 2 signatures for 978 sequences in protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000993115.1_ASM99311v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz 977 sequences\r",
      "\u001b[Kcalculated 2 signatures for 978 sequences in protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000993115.1_ASM99311v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz 977 sequences\r",
      "\u001b[Kcalculated 2 signatures for 978 sequences in protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00016 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000993115.1_ASM99311v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz 977 sequences\r",
      "\u001b[Kcalculated 2 signatures for 978 sequences in protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000993115.1_ASM99311v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz 977 sequences\r",
      "\u001b[Kcalculated 2 signatures for 978 sequences in protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_000993115.1_ASM99311v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz 977 sequences\r",
      "\u001b[Kcalculated 2 signatures for 978 sequences in protein/seqs/GCA_000993115.1_ASM99311v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_000993115.1_ASM99311v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001049535.1_3731_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001049535.1_3731_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001064135.1_ASM106413v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001261715.1_HGC4_v3_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz 2003 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2004 sequences in protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00026 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001263815.1_ASM126381v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz 2003 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2004 sequences in protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001263815.1_ASM126381v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz 2003 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2004 sequences in protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001263815.1_ASM126381v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz 2003 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2004 sequences in protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001263815.1_ASM126381v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz 2003 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2004 sequences in protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00044 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001263815.1_ASM126381v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz 2003 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2004 sequences in protein/seqs/GCA_001263815.1_ASM126381v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001263815.1_ASM126381v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz 2001 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2002 sequences in protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001274615.1_ASM127461v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz 2001 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2002 sequences in protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001274615.1_ASM127461v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz 2001 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2002 sequences in protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001274615.1_ASM127461v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz 2001 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2002 sequences in protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001274615.1_ASM127461v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz 2001 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2002 sequences in protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001274615.1_ASM127461v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz 2001 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2002 sequences in protein/seqs/GCA_001274615.1_ASM127461v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00040 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001274615.1_ASM127461v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001282625.1_Lascolabacillus_massiliensis_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz 3544 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3545 sequences in protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001404575.1_13470_2_67_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz 3544 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3545 sequences in protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001404575.1_13470_2_67_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz 3544 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3545 sequences in protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001404575.1_13470_2_67_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz 3544 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3545 sequences in protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001404575.1_13470_2_67_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz 3544 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3545 sequences in protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00058 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001404575.1_13470_2_67_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz 3544 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3545 sequences in protein/seqs/GCA_001404575.1_13470_2_67_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00060 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001404575.1_13470_2_67_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz 3812 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3813 sequences in protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001406635.1_13470_2_63_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz 3812 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3813 sequences in protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001406635.1_13470_2_63_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz 3812 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3813 sequences in protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001406635.1_13470_2_63_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz 3812 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3813 sequences in protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00030 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001406635.1_13470_2_63_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz 3812 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3813 sequences in protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00063 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001406635.1_13470_2_63_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz 3812 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3813 sequences in protein/seqs/GCA_001406635.1_13470_2_63_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00085 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001406635.1_13470_2_63_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz 1661 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1662 sequences in protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001438125.1_ASM143812v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz 1661 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1662 sequences in protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001438125.1_ASM143812v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz 1661 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1662 sequences in protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001438125.1_ASM143812v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz 1661 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1662 sequences in protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001438125.1_ASM143812v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz 1661 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1662 sequences in protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001438125.1_ASM143812v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz 1661 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1662 sequences in protein/seqs/GCA_001438125.1_ASM143812v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00049 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001438125.1_ASM143812v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz 1494 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1495 sequences in protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001438205.1_ASM143820v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz 1494 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1495 sequences in protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001438205.1_ASM143820v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz 1494 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1495 sequences in protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001438205.1_ASM143820v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz 1494 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1495 sequences in protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00031 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001438205.1_ASM143820v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz 1494 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1495 sequences in protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00043 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001438205.1_ASM143820v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz 1494 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1495 sequences in protein/seqs/GCA_001438205.1_ASM143820v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001438205.1_ASM143820v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz 3864 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3865 sequences in protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz 3864 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3865 sequences in protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz 3864 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3865 sequences in protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz 3864 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3865 sequences in protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz 3864 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3865 sequences in protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00059 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz 3864 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3865 sequences in protein/seqs/GCA_001443605.1_ASM144360v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00058 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001443605.1_ASM144360v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz 4588 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4589 sequences in protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz 4588 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4589 sequences in protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz 4588 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4589 sequences in protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz 4588 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4589 sequences in protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00023 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz 4588 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4589 sequences in protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz 4588 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4589 sequences in protein/seqs/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00058 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001480535.1_Salmonella_enterica_CVM_N38910-SQ_v1.0_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001512095.1_7521_8_38_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001602775.1_ASM160277v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz 5572 sequences\r",
      "\u001b[Kcalculated 2 signatures for 5573 sequences in protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001607025.1_ASM160702v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz 5572 sequences\r",
      "\u001b[Kcalculated 2 signatures for 5573 sequences in protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001607025.1_ASM160702v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz 5572 sequences\r",
      "\u001b[Kcalculated 2 signatures for 5573 sequences in protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001607025.1_ASM160702v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz 5572 sequences\r",
      "\u001b[Kcalculated 2 signatures for 5573 sequences in protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00078 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001607025.1_ASM160702v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz 5572 sequences\r",
      "\u001b[Kcalculated 2 signatures for 5573 sequences in protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00064 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001607025.1_ASM160702v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz 5572 sequences\r",
      "\u001b[Kcalculated 2 signatures for 5573 sequences in protein/seqs/GCA_001607025.1_ASM160702v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00112 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001607025.1_ASM160702v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001637845.1_ASM163784v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz 2016 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2017 sequences in protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001670785.1_RCAD0183_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz 2016 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2017 sequences in protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001670785.1_RCAD0183_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz 2016 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2017 sequences in protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001670785.1_RCAD0183_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz 2016 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2017 sequences in protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001670785.1_RCAD0183_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz 2016 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2017 sequences in protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001670785.1_RCAD0183_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz 2016 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2017 sequences in protein/seqs/GCA_001670785.1_RCAD0183_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001670785.1_RCAD0183_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz 897 sequences\r",
      "\u001b[Kcalculated 2 signatures for 898 sequences in protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001724935.1_ASM172493v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz 897 sequences\r",
      "\u001b[Kcalculated 2 signatures for 898 sequences in protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001724935.1_ASM172493v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz 897 sequences\r",
      "\u001b[Kcalculated 2 signatures for 898 sequences in protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001724935.1_ASM172493v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz 897 sequences\r",
      "\u001b[Kcalculated 2 signatures for 898 sequences in protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001724935.1_ASM172493v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz 897 sequences\r",
      "\u001b[Kcalculated 2 signatures for 898 sequences in protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00033 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001724935.1_ASM172493v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz 897 sequences\r",
      "\u001b[Kcalculated 2 signatures for 898 sequences in protein/seqs/GCA_001724935.1_ASM172493v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00030 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001724935.1_ASM172493v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz 1958 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1959 sequences in protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001768555.1_ASM176855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz 1958 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1959 sequences in protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001768555.1_ASM176855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz 1958 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1959 sequences in protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001768555.1_ASM176855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz 1958 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1959 sequences in protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001768555.1_ASM176855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz 1958 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1959 sequences in protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001768555.1_ASM176855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz 1958 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1959 sequences in protein/seqs/GCA_001768555.1_ASM176855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00040 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001768555.1_ASM176855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz 1614 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1615 sequences in protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001808555.1_ASM180855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz 1614 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1615 sequences in protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001808555.1_ASM180855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz 1614 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1615 sequences in protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001808555.1_ASM180855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz 1614 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1615 sequences in protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001808555.1_ASM180855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz 1614 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1615 sequences in protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00037 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001808555.1_ASM180855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz 1614 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1615 sequences in protein/seqs/GCA_001808555.1_ASM180855v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00033 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001808555.1_ASM180855v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz 2281 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2282 sequences in protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001873105.1_ASM187310v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz 2281 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2282 sequences in protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001873105.1_ASM187310v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz 2281 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2282 sequences in protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001873105.1_ASM187310v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz 2281 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2282 sequences in protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001873105.1_ASM187310v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz 2281 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2282 sequences in protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00039 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001873105.1_ASM187310v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz 2281 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2282 sequences in protein/seqs/GCA_001873105.1_ASM187310v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00046 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001873105.1_ASM187310v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz 2660 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2661 sequences in protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001889045.1_ASM188904v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz 2660 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2661 sequences in protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001889045.1_ASM188904v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz 2660 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2661 sequences in protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001889045.1_ASM188904v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz 2660 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2661 sequences in protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001889045.1_ASM188904v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz 2660 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2661 sequences in protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00041 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001889045.1_ASM188904v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz 2660 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2661 sequences in protein/seqs/GCA_001889045.1_ASM188904v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00041 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001889045.1_ASM188904v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_001896575.1_ASM189657v1_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz 4207 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4208 sequences in protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001897595.1_ASM189759v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz 4207 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4208 sequences in protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001897595.1_ASM189759v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz 4207 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4208 sequences in protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00035 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001897595.1_ASM189759v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz 4207 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4208 sequences in protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00027 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001897595.1_ASM189759v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz 4207 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4208 sequences in protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00067 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001897595.1_ASM189759v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz 4207 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4208 sequences in protein/seqs/GCA_001897595.1_ASM189759v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00068 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001897595.1_ASM189759v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz 2941 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2942 sequences in protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001898165.1_ASM189816v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz 2941 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2942 sequences in protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001898165.1_ASM189816v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz 2941 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2942 sequences in protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001898165.1_ASM189816v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz 2941 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2942 sequences in protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001898165.1_ASM189816v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz 2941 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2942 sequences in protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00044 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_001898165.1_ASM189816v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz 2941 sequences\r",
      "\u001b[Kcalculated 2 signatures for 2942 sequences in protein/seqs/GCA_001898165.1_ASM189816v1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00049 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_001898165.1_ASM189816v1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz 4960 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4961 sequences in protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900085525.1_12045_7_37_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz 4960 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4961 sequences in protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900085525.1_12045_7_37_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz 4960 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4961 sequences in protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900085525.1_12045_7_37_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz 4960 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4961 sequences in protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00031 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900085525.1_12045_7_37_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz 4960 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4961 sequences in protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00066 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900085525.1_12045_7_37_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz 4960 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4961 sequences in protein/seqs/GCA_900085525.1_12045_7_37_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900085525.1_12045_7_37_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900089535.1_A04_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900089535.1_A04_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900089535.1_A04_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900089535.1_A04_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900089535.1_A04_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900089535.1_A04_protein.faa.gz\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/bin/sourmash\", line 11, in <module>\n",
      "    sys.exit(main())\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/__main__.py\", line 83, in main\n",
      "    cmd(sys.argv[2:])\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/sourmash/command_compute.py\", line 335, in compute\n",
      "    for n, record in enumerate(screed.open(filename)):\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 39, in __init__\n",
      "    self.iter_fn = self.open_reader(filename, *args, **kwargs)\n",
      "  File \"/home/hehouts/miniconda3/envs/rotation/lib/python3.7/site-packages/screed/openscreed.py\", line 95, in open_reader\n",
      "    raise ValueError(\"unknown file format for '%s'\" % filename)\n",
      "ValueError: unknown file format for 'protein/seqs/GCA_900089535.1_A04_protein.faa.gz'\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz 3333 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3334 sequences in protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz 3333 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3334 sequences in protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00032 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz 3333 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3334 sequences in protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz 3333 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3334 sequences in protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz 3333 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3334 sequences in protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00078 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz 3333 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3334 sequences in protein/seqs/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00095 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900107785.1_IMG-taxon_2654588140_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz 3559 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3560 sequences in protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz 3559 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3560 sequences in protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz 3559 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3560 sequences in protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00034 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz 3559 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3560 sequences in protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz 3559 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3560 sequences in protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00061 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz 3559 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3560 sequences in protein/seqs/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00058 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900112105.1_IMG-taxon_2622736505_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz 3924 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3925 sequences in protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz 3924 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3925 sequences in protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz 3924 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3925 sequences in protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz 3924 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3925 sequences in protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00024 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz 3924 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3925 sequences in protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00063 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz 3924 sequences\r",
      "\u001b[Kcalculated 2 signatures for 3925 sequences in protein/seqs/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00066 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900112395.1_IMG-taxon_2634166310_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz 4004 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4005 sequences in protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz 4004 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4005 sequences in protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00013 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz 4004 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4005 sequences in protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz 4004 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4005 sequences in protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz 4004 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4005 sequences in protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00064 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz 4004 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4005 sequences in protein/seqs/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00068 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900114365.1_IMG-taxon_2651870357_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz 4014 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4015 sequences in protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz 4014 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4015 sequences in protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00023 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz 4014 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4015 sequences in protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz 4014 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4015 sequences in protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00022 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz 4014 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4015 sequences in protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00059 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz 4014 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4015 sequences in protein/seqs/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00055 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900128985.1_IMG-taxon_2695420931_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz 4108 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4109 sequences in protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00014 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz 4108 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4109 sequences in protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00012 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz 4108 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4109 sequences in protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz 4108 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4109 sequences in protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00030 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz 4108 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4109 sequences in protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00062 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz 4108 sequences\r",
      "\u001b[Kcalculated 2 signatures for 4109 sequences in protein/seqs/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00117 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900141875.1_IMG-taxon_2700988694_annotated_assembly_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz 1778 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1779 sequences in protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00018 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900157215.1_Strain_3-3_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz 1778 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1779 sequences in protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00011 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900157215.1_Strain_3-3_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz 1778 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1779 sequences in protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00020 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900157215.1_Strain_3-3_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz 1778 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1779 sequences in protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900157215.1_Strain_3-3_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz 1778 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1779 sequences in protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00082 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900157215.1_Strain_3-3_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz 1778 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1779 sequences in protein/seqs/GCA_900157215.1_Strain_3-3_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00052 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900157215.1_Strain_3-3_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157325.1_3A1_protein.faa.gz 1845 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1846 sequences in protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00015 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900157325.1_3A1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [7]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157325.1_3A1_protein.faa.gz 1845 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1846 sequences in protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00017 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900157325.1_3A1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157325.1_3A1_protein.faa.gz 1845 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1846 sequences in protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00019 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900157325.1_3A1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [12]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157325.1_3A1_protein.faa.gz 1845 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1846 sequences in protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00021 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900157325.1_3A1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157325.1_3A1_protein.faa.gz 1845 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1846 sequences in protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00058 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/dayhoff/GCA_900157325.1_3A1_protein.sig. Note: signature license is CC0.\n",
      "\r",
      "\u001b[K== This is sourmash version 2.3.0. ==\n",
      "\r",
      "\u001b[K== Please cite Brown and Irber (2016), doi:10.21105/joss.00027. ==\n",
      "\n",
      "\r",
      "\u001b[KWARNING: input is protein, turning off nucleotide hashing\n",
      "\r",
      "\u001b[Ksetting num_hashes to 0 because --scaled is set\n",
      "\r",
      "\u001b[Kcomputing signatures for files: protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[KComputing signature for ksizes: [15]\n",
      "\r",
      "\u001b[KComputing only protein (and not nucleotide) signatures.\n",
      "\r",
      "\u001b[KComputing a total of 1 signature(s).\n",
      "\r",
      "\u001b[KTracking abundance of input k-mers.\n",
      "\r",
      "\u001b[K... reading sequences from protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[K...protein/seqs/GCA_900157325.1_3A1_protein.faa.gz 1845 sequences\r",
      "\u001b[Kcalculated 2 signatures for 1846 sequences in protein/seqs/GCA_900157325.1_3A1_protein.faa.gz\n",
      "\r",
      "\u001b[Ktime taken to save signatures is 0.00036 seconds\r",
      "\r",
      "\u001b[Ksaved signature(s) to protein/sigs/hp/GCA_900157325.1_3A1_protein.sig. Note: signature license is CC0.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "\n",
    "##For dayhoff and HP\n",
    "#Calc signatures\n",
    "\n",
    "\n",
    "for org in gingivalis\n",
    "#bacteroides denticola \n",
    "do\n",
    "    for mol in protein\n",
    "    do\n",
    "\n",
    "        cd $org\n",
    "        mkdir -p $mol/sigs/dayhoff\n",
    "        mkdir -p $mol/sigs/hp\n",
    "        \n",
    "        for infile in $mol/seqs/*a.gz \n",
    "        do \n",
    "            for k in 7 12 15\n",
    "            do\n",
    "                out_name=$(basename $infile .faa.gz)\n",
    "\n",
    "                #Dayhoff\n",
    "                sourmash compute --input-is-protein --dayhoff -k $k --scaled 2000 --track-abundance -o $mol/sigs/dayhoff/${out_name}.sig ${infile}\n",
    "\n",
    "                #HP\n",
    "                sourmash compute --input-is-protein --hp -k $k --scaled 2000 --track-abundance -o $mol/sigs/hp/${out_name}.sig ${infile}\n",
    "\n",
    "            done\n",
    "        done\n",
    "        cd ..\n",
    "    done\n",
    "   \n",
    "   \n",
    "done\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "usage: sourmash [--protein] [--no-protein] [--dayhoff] [--no-dayhoff] [--hp]\r\n",
      "                [--no-hp] [--dna] [--no-dna] [-q] [--input-is-protein]\r\n",
      "                [-k KSIZES] [-n NUM_HASHES] [--check-sequence] [-f]\r\n",
      "                [-o OUTPUT] [--singleton] [--merge MERGED] [--name-from-first]\r\n",
      "                [--input-is-10x] [--count-valid-reads COUNT_VALID_READS]\r\n",
      "                [--write-barcode-meta-csv WRITE_BARCODE_META_CSV]\r\n",
      "                [-p PROCESSES] [--save-fastas SAVE_FASTAS]\r\n",
      "                [--line-count LINE_COUNT] [--track-abundance]\r\n",
      "                [--scaled SCALED] [--seed SEED] [--randomize]\r\n",
      "                [--license LICENSE]\r\n",
      "                [--rename-10x-barcodes RENAME_10X_BARCODES]\r\n",
      "                [--barcodes-file BARCODES_FILE]\r\n",
      "                filenames [filenames ...]\r\n",
      "sourmash: error: the following arguments are required: filenames\r\n"
     ]
    }
   ],
   "source": [
    "!sourmash compute -h "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
